<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html>
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>annotationInfo — annotationInfo • gemmaAPI</title>

<!-- jquery -->
<script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha384-nrOSfDHtoPMzJHjVTdCopGqIqeYETSXhZDFyniQ8ZHcVy08QesyHcnOUpMpqnmWq" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>

<!-- Font Awesome icons -->
<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js" integrity="sha384-cV+rhyOuRHc9Ub/91rihWcGmMmCXDeksTtCihMupQHSsi8GIIRDG0ThDc3HGQFJ3" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../jquery.sticky-kit.min.js"></script>
<script src="../pkgdown.js"></script>
  
  
<meta property="og:title" content="annotationInfo — annotationInfo" />

<meta property="og:description" content="annotationInfo" />
<meta name="twitter:card" content="summary" />
<!-- mathjax -->
<script src='https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->


  </head>

  <body>
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="../index.html">gemmaAPI</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
      
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      
      </header>

      <div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>annotationInfo</h1>
    </div>

    
    <p>annotationInfo</p>
    

    <pre class="usage"><span class='fu'>annotationInfo</span>(<span class='no'>annotation</span>, <span class='kw'>request</span> <span class='kw'>=</span> <span class='kw'>NULL</span>, <span class='kw'>file</span> <span class='kw'>=</span> <span class='kw'>NULL</span>, <span class='kw'>return</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>,
  <span class='kw'>overwrite</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>, <span class='kw'>memoised</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>)</pre>
    
    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a> Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>annotation</th>
      <td><p>Character. Either plain text, ontology term ID or an ontology term URI.</p></td>
    </tr>
    <tr>
      <th>request</th>
      <td><p>Character. If null retreeives the annotation object. Otherwise</p><ul>
<li><p><code>datasets</code>: Does a search for datasets containing annotations matching the given string</p></li>
</ul></td>
    </tr>
    <tr>
      <th>file</th>
      <td><p>Character. File path. If provided, response will be saved to file</p></td>
    </tr>
    <tr>
      <th>return</th>
      <td><p>Logical. If the response should be returned. Set to false when
you only want to save a file</p></td>
    </tr>
    <tr>
      <th>overwrite</th>
      <td><p>Logical. If TRUE, existing files will be overwritten. If FALSE
a warning will be thrown and no action is taken.</p></td>
    </tr>
    <tr>
      <th>memoised</th>
      <td><p>Logical. If TRUE a memoised version of the function will be
used which is faster for repeated requests. Use <code><a href='forgetGemmaMemoised.html'>forgetGemmaMemoised</a></code>
to clear memory.</p></td>
    </tr>
    </table>
    
    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A list</p>
    

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='fu'>annotationInfo</span>(<span class='st'>'http://purl.obolibrary.org/obo/OBI_0000105'</span>)</div><div class='output co'>#&gt; $transplantation
#&gt; $transplantation$value
#&gt; [1] "transplantation"
#&gt; 
#&gt; $transplantation$valueUri
#&gt; [1] "http://purl.obolibrary.org/obo/OBI_0000105"
#&gt; 
#&gt; $transplantation$category
#&gt; [1] "medical procedure"
#&gt; 
#&gt; $transplantation$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0002571"
#&gt; 
#&gt; </div><div class='input'><span class='fu'>annotationInfo</span>(<span class='st'>'transplantation'</span>)</div><div class='output co'>#&gt; $transplantation
#&gt; $transplantation$value
#&gt; [1] "transplantation"
#&gt; 
#&gt; $transplantation$valueUri
#&gt; [1] "http://purl.obolibrary.org/obo/OBI_0000105"
#&gt; 
#&gt; $transplantation$category
#&gt; [1] "treatment"
#&gt; 
#&gt; $transplantation$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000727"
#&gt; 
#&gt; 
#&gt; $xenotransplantation
#&gt; $xenotransplantation$value
#&gt; [1] "xenotransplantation"
#&gt; 
#&gt; $xenotransplantation$valueUri
#&gt; [1] "http://purl.obolibrary.org/obo/OBI_0000043"
#&gt; 
#&gt; $xenotransplantation$category
#&gt; [1] "growth condition"
#&gt; 
#&gt; $xenotransplantation$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000523"
#&gt; 
#&gt; 
#&gt; $allotransplantation
#&gt; $allotransplantation$value
#&gt; [1] "allotransplantation"
#&gt; 
#&gt; $allotransplantation$valueUri
#&gt; [1] "http://purl.obolibrary.org/obo/OBI_0000057"
#&gt; 
#&gt; $allotransplantation$category
#&gt; [1] "medical procedure"
#&gt; 
#&gt; $allotransplantation$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0002571"
#&gt; 
#&gt; 
#&gt; $`patient with myelodysplastic syndrome refractory anemia with excess of blasts type 2 in remission after transplantation with allogeneic hematopoietic stem cells`
#&gt; $`patient with myelodysplastic syndrome refractory anemia with excess of blasts type 2 in remission after transplantation with allogeneic hematopoietic stem cells`$value
#&gt; [1] "patient with myelodysplastic syndrome refractory anemia with excess of blasts type 2 in remission after transplantation with allogeneic hematopoietic stem cells"
#&gt; 
#&gt; $`patient with myelodysplastic syndrome refractory anemia with excess of blasts type 2 in remission after transplantation with allogeneic hematopoietic stem cells`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`patient with myelodysplastic syndrome refractory anemia with excess of blasts type 2 in remission after transplantation with allogeneic hematopoietic stem cells`$category
#&gt; [1] "disease"
#&gt; 
#&gt; $`patient with myelodysplastic syndrome refractory anemia with excess of blasts type 2 in remission after transplantation with allogeneic hematopoietic stem cells`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000408"
#&gt; 
#&gt; 
#&gt; $`patient with chronic myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`
#&gt; $`patient with chronic myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`$value
#&gt; [1] "patient with chronic myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells"
#&gt; 
#&gt; $`patient with chronic myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`patient with chronic myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`$category
#&gt; [1] "disease"
#&gt; 
#&gt; $`patient with chronic myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000408"
#&gt; 
#&gt; 
#&gt; $`12month protocol bx (post-renal transplantation)`
#&gt; $`12month protocol bx (post-renal transplantation)`$value
#&gt; [1] "12month protocol bx (post-renal transplantation)"
#&gt; 
#&gt; $`12month protocol bx (post-renal transplantation)`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`12month protocol bx (post-renal transplantation)`$category
#&gt; NULL
#&gt; 
#&gt; $`12month protocol bx (post-renal transplantation)`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`heart transplantation`
#&gt; $`heart transplantation`$value
#&gt; [1] "heart transplantation"
#&gt; 
#&gt; $`heart transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`heart transplantation`$category
#&gt; [1] "treatment"
#&gt; 
#&gt; $`heart transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000727"
#&gt; 
#&gt; 
#&gt; $`liver transplantation`
#&gt; $`liver transplantation`$value
#&gt; [1] "liver transplantation"
#&gt; 
#&gt; $`liver transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`liver transplantation`$category
#&gt; [1] "treatment"
#&gt; 
#&gt; $`liver transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000727"
#&gt; 
#&gt; 
#&gt; $`hematopoietic cell transplantation`
#&gt; $`hematopoietic cell transplantation`$value
#&gt; [1] "hematopoietic cell transplantation"
#&gt; 
#&gt; $`hematopoietic cell transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`hematopoietic cell transplantation`$category
#&gt; [1] "treatment"
#&gt; 
#&gt; $`hematopoietic cell transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000727"
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 39`
#&gt; $`rejection time (days after transplantation): 39`$value
#&gt; [1] "rejection time (days after transplantation): 39"
#&gt; 
#&gt; $`rejection time (days after transplantation): 39`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 39`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 39`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`patient with acute myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`
#&gt; $`patient with acute myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`$value
#&gt; [1] "patient with acute myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells"
#&gt; 
#&gt; $`patient with acute myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`patient with acute myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`$category
#&gt; [1] "disease"
#&gt; 
#&gt; $`patient with acute myeloid leukemia in remission after transplantation with allogeneic hematopoietic stem cells`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000408"
#&gt; 
#&gt; 
#&gt; $`Donor dendritic cells from medulla of renal allograft (3 days post-transplantation)`
#&gt; $`Donor dendritic cells from medulla of renal allograft (3 days post-transplantation)`$value
#&gt; [1] "Donor dendritic cells from medulla of renal allograft (3 days post-transplantation)"
#&gt; 
#&gt; $`Donor dendritic cells from medulla of renal allograft (3 days post-transplantation)`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`Donor dendritic cells from medulla of renal allograft (3 days post-transplantation)`$category
#&gt; [1] "BioSource"
#&gt; 
#&gt; $`Donor dendritic cells from medulla of renal allograft (3 days post-transplantation)`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 3`
#&gt; $`rejection time (days after transplantation): 3`$value
#&gt; [1] "rejection time (days after transplantation): 3"
#&gt; 
#&gt; $`rejection time (days after transplantation): 3`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 3`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 3`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 73`
#&gt; $`rejection time (days after transplantation): 73`$value
#&gt; [1] "rejection time (days after transplantation): 73"
#&gt; 
#&gt; $`rejection time (days after transplantation): 73`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 73`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 73`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`time after transplantation: ?2 months`
#&gt; $`time after transplantation: ?2 months`$value
#&gt; [1] "time after transplantation: ?2 months"
#&gt; 
#&gt; $`time after transplantation: ?2 months`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`time after transplantation: ?2 months`$category
#&gt; NULL
#&gt; 
#&gt; $`time after transplantation: ?2 months`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 35`
#&gt; $`rejection time (days after transplantation): 35`$value
#&gt; [1] "rejection time (days after transplantation): 35"
#&gt; 
#&gt; $`rejection time (days after transplantation): 35`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 35`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 35`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`advanced heart failure-DCM-serum-at LVAD explantation for heart transplantation`
#&gt; $`advanced heart failure-DCM-serum-at LVAD explantation for heart transplantation`$value
#&gt; [1] "advanced heart failure-DCM-serum-at LVAD explantation for heart transplantation"
#&gt; 
#&gt; $`advanced heart failure-DCM-serum-at LVAD explantation for heart transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`advanced heart failure-DCM-serum-at LVAD explantation for heart transplantation`$category
#&gt; [1] "BioSource"
#&gt; 
#&gt; $`advanced heart failure-DCM-serum-at LVAD explantation for heart transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`advanced heart failure-ICM-plasma-at LVAD explantation for heart transplantation`
#&gt; $`advanced heart failure-ICM-plasma-at LVAD explantation for heart transplantation`$value
#&gt; [1] "advanced heart failure-ICM-plasma-at LVAD explantation for heart transplantation"
#&gt; 
#&gt; $`advanced heart failure-ICM-plasma-at LVAD explantation for heart transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`advanced heart failure-ICM-plasma-at LVAD explantation for heart transplantation`$category
#&gt; [1] "BioSource"
#&gt; 
#&gt; $`advanced heart failure-ICM-plasma-at LVAD explantation for heart transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`advanced heart failure-ICM-serum-at LVAD explantation for heart transplantation`
#&gt; $`advanced heart failure-ICM-serum-at LVAD explantation for heart transplantation`$value
#&gt; [1] "advanced heart failure-ICM-serum-at LVAD explantation for heart transplantation"
#&gt; 
#&gt; $`advanced heart failure-ICM-serum-at LVAD explantation for heart transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`advanced heart failure-ICM-serum-at LVAD explantation for heart transplantation`$category
#&gt; [1] "BioSource"
#&gt; 
#&gt; $`advanced heart failure-ICM-serum-at LVAD explantation for heart transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`gxbdataset_biirinternal: P3 TRANSPLANTATION-BUMC (26) 11Jun08 Liver Transplant for Modules`
#&gt; $`gxbdataset_biirinternal: P3 TRANSPLANTATION-BUMC (26) 11Jun08 Liver Transplant for Modules`$value
#&gt; [1] "gxbdataset_biirinternal: P3 TRANSPLANTATION-BUMC (26) 11Jun08 Liver Transplant for Modules"
#&gt; 
#&gt; $`gxbdataset_biirinternal: P3 TRANSPLANTATION-BUMC (26) 11Jun08 Liver Transplant for Modules`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`gxbdataset_biirinternal: P3 TRANSPLANTATION-BUMC (26) 11Jun08 Liver Transplant for Modules`$category
#&gt; NULL
#&gt; 
#&gt; $`gxbdataset_biirinternal: P3 TRANSPLANTATION-BUMC (26) 11Jun08 Liver Transplant for Modules`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 7`
#&gt; $`rejection time (days after transplantation): 7`$value
#&gt; [1] "rejection time (days after transplantation): 7"
#&gt; 
#&gt; $`rejection time (days after transplantation): 7`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 7`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 7`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 8`
#&gt; $`rejection time (days after transplantation): 8`$value
#&gt; [1] "rejection time (days after transplantation): 8"
#&gt; 
#&gt; $`rejection time (days after transplantation): 8`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 8`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 8`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 9`
#&gt; $`rejection time (days after transplantation): 9`$value
#&gt; [1] "rejection time (days after transplantation): 9"
#&gt; 
#&gt; $`rejection time (days after transplantation): 9`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 9`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 9`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`Host dendritic cells from cortex of renal allograft (3 days post-transplantation)`
#&gt; $`Host dendritic cells from cortex of renal allograft (3 days post-transplantation)`$value
#&gt; [1] "Host dendritic cells from cortex of renal allograft (3 days post-transplantation)"
#&gt; 
#&gt; $`Host dendritic cells from cortex of renal allograft (3 days post-transplantation)`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`Host dendritic cells from cortex of renal allograft (3 days post-transplantation)`$category
#&gt; [1] "BioSource"
#&gt; 
#&gt; $`Host dendritic cells from cortex of renal allograft (3 days post-transplantation)`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 69`
#&gt; $`rejection time (days after transplantation): 69`$value
#&gt; [1] "rejection time (days after transplantation): 69"
#&gt; 
#&gt; $`rejection time (days after transplantation): 69`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 69`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 69`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`deterioration of graft function within 2 years after transplantation: yes`
#&gt; $`deterioration of graft function within 2 years after transplantation: yes`$value
#&gt; [1] "deterioration of graft function within 2 years after transplantation: yes"
#&gt; 
#&gt; $`deterioration of graft function within 2 years after transplantation: yes`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`deterioration of graft function within 2 years after transplantation: yes`$category
#&gt; NULL
#&gt; 
#&gt; $`deterioration of graft function within 2 years after transplantation: yes`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 28`
#&gt; $`rejection time (days after transplantation): 28`$value
#&gt; [1] "rejection time (days after transplantation): 28"
#&gt; 
#&gt; $`rejection time (days after transplantation): 28`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 28`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 28`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 61`
#&gt; $`rejection time (days after transplantation): 61`$value
#&gt; [1] "rejection time (days after transplantation): 61"
#&gt; 
#&gt; $`rejection time (days after transplantation): 61`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 61`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 61`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 22`
#&gt; $`rejection time (days after transplantation): 22`$value
#&gt; [1] "rejection time (days after transplantation): 22"
#&gt; 
#&gt; $`rejection time (days after transplantation): 22`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 22`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 22`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`advanced heart failure-DCM-plasma-at LVAD explantation for heart transplantation`
#&gt; $`advanced heart failure-DCM-plasma-at LVAD explantation for heart transplantation`$value
#&gt; [1] "advanced heart failure-DCM-plasma-at LVAD explantation for heart transplantation"
#&gt; 
#&gt; $`advanced heart failure-DCM-plasma-at LVAD explantation for heart transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`advanced heart failure-DCM-plasma-at LVAD explantation for heart transplantation`$category
#&gt; [1] "BioSource"
#&gt; 
#&gt; $`advanced heart failure-DCM-plasma-at LVAD explantation for heart transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`time after transplantation: 3 months`
#&gt; $`time after transplantation: 3 months`$value
#&gt; [1] "time after transplantation: 3 months"
#&gt; 
#&gt; $`time after transplantation: 3 months`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`time after transplantation: 3 months`$category
#&gt; NULL
#&gt; 
#&gt; $`time after transplantation: 3 months`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`Hematopoietic progenitor cells (HPC) were enriched within the CD34+ cell fraction (selection by magnetic beads and subsequent flow cytometric separation) from G-CSF mobilized blood from healthy donors for allogeneic blood stem cell transplantation.`
#&gt; $`Hematopoietic progenitor cells (HPC) were enriched within the CD34+ cell fraction (selection by magnetic beads and subsequent flow cytometric separation) from G-CSF mobilized blood from healthy donors for allogeneic blood stem cell transplantation.`$value
#&gt; [1] "Hematopoietic progenitor cells (HPC) were enriched within the CD34+ cell fraction (selection by magnetic beads and subsequent flow cytometric separation) from G-CSF mobilized blood from healthy donors for allogeneic blood stem cell transplantation."
#&gt; 
#&gt; $`Hematopoietic progenitor cells (HPC) were enriched within the CD34+ cell fraction (selection by magnetic beads and subsequent flow cytometric separation) from G-CSF mobilized blood from healthy donors for allogeneic blood stem cell transplantation.`$...
#&gt; NULL
#&gt; 
#&gt; $`Hematopoietic progenitor cells (HPC) were enriched within the CD34+ cell fraction (selection by magnetic beads and subsequent flow cytometric separation) from G-CSF mobilized blood from healthy donors for allogeneic blood stem cell transplantation.`$...
#&gt; NULL
#&gt; 
#&gt; $`Hematopoietic progenitor cells (HPC) were enriched within the CD34+ cell fraction (selection by magnetic beads and subsequent flow cytometric separation) from G-CSF mobilized blood from healthy donors for allogeneic blood stem cell transplantation.`$...
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`before transplantation`
#&gt; $`before transplantation`$value
#&gt; [1] "before transplantation"
#&gt; 
#&gt; $`before transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`before transplantation`$category
#&gt; [1] "treatment"
#&gt; 
#&gt; $`before transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000727"
#&gt; 
#&gt; 
#&gt; $`after transplantation`
#&gt; $`after transplantation`$value
#&gt; [1] "after transplantation"
#&gt; 
#&gt; $`after transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`after transplantation`$category
#&gt; [1] "treatment"
#&gt; 
#&gt; $`after transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000727"
#&gt; 
#&gt; 
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with anti-IL6R antibody immediately after transplantation`
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with anti-IL6R antibody immediately after transplantation`$value
#&gt; [1] "kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with anti-IL6R antibody immediately after transplantation"
#&gt; 
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with anti-IL6R antibody immediately after transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with anti-IL6R antibody immediately after transplantation`$category
#&gt; [1] "organism part"
#&gt; 
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with anti-IL6R antibody immediately after transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 15`
#&gt; $`rejection time (days after transplantation): 15`$value
#&gt; [1] "rejection time (days after transplantation): 15"
#&gt; 
#&gt; $`rejection time (days after transplantation): 15`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 15`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 15`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`explanted heart tissue at time of cardiac transplantation`
#&gt; $`explanted heart tissue at time of cardiac transplantation`$value
#&gt; [1] "explanted heart tissue at time of cardiac transplantation"
#&gt; 
#&gt; $`explanted heart tissue at time of cardiac transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`explanted heart tissue at time of cardiac transplantation`$category
#&gt; [1] "BioSource"
#&gt; 
#&gt; $`explanted heart tissue at time of cardiac transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 16`
#&gt; $`rejection time (days after transplantation): 16`$value
#&gt; [1] "rejection time (days after transplantation): 16"
#&gt; 
#&gt; $`rejection time (days after transplantation): 16`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 16`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 16`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 18`
#&gt; $`rejection time (days after transplantation): 18`$value
#&gt; [1] "rejection time (days after transplantation): 18"
#&gt; 
#&gt; $`rejection time (days after transplantation): 18`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 18`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 18`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 53`
#&gt; $`rejection time (days after transplantation): 53`$value
#&gt; [1] "rejection time (days after transplantation): 53"
#&gt; 
#&gt; $`rejection time (days after transplantation): 53`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 53`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 53`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 99`
#&gt; $`rejection time (days after transplantation): 99`$value
#&gt; [1] "rejection time (days after transplantation): 99"
#&gt; 
#&gt; $`rejection time (days after transplantation): 99`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 99`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 99`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`kidney transplantation`
#&gt; $`kidney transplantation`$value
#&gt; [1] "kidney transplantation"
#&gt; 
#&gt; $`kidney transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`kidney transplantation`$category
#&gt; [1] "treatment"
#&gt; 
#&gt; $`kidney transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000727"
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 10`
#&gt; $`rejection time (days after transplantation): 10`$value
#&gt; [1] "rejection time (days after transplantation): 10"
#&gt; 
#&gt; $`rejection time (days after transplantation): 10`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 10`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 10`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 13`
#&gt; $`rejection time (days after transplantation): 13`$value
#&gt; [1] "rejection time (days after transplantation): 13"
#&gt; 
#&gt; $`rejection time (days after transplantation): 13`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 13`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 13`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`rejection time (days after transplantation): 12`
#&gt; $`rejection time (days after transplantation): 12`$value
#&gt; [1] "rejection time (days after transplantation): 12"
#&gt; 
#&gt; $`rejection time (days after transplantation): 12`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 12`$category
#&gt; NULL
#&gt; 
#&gt; $`rejection time (days after transplantation): 12`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`Host dendritic cells from medulla of renal allograft (3 days post-transplantation)`
#&gt; $`Host dendritic cells from medulla of renal allograft (3 days post-transplantation)`$value
#&gt; [1] "Host dendritic cells from medulla of renal allograft (3 days post-transplantation)"
#&gt; 
#&gt; $`Host dendritic cells from medulla of renal allograft (3 days post-transplantation)`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`Host dendritic cells from medulla of renal allograft (3 days post-transplantation)`$category
#&gt; [1] "BioSource"
#&gt; 
#&gt; $`Host dendritic cells from medulla of renal allograft (3 days post-transplantation)`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`microbiota transplantation`
#&gt; $`microbiota transplantation`$value
#&gt; [1] "microbiota transplantation"
#&gt; 
#&gt; $`microbiota transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`microbiota transplantation`$category
#&gt; [1] "treatment"
#&gt; 
#&gt; $`microbiota transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000727"
#&gt; 
#&gt; 
#&gt; $`Donor dendritic cells from cortex of renal allograft (3 days post-transplantation)`
#&gt; $`Donor dendritic cells from cortex of renal allograft (3 days post-transplantation)`$value
#&gt; [1] "Donor dendritic cells from cortex of renal allograft (3 days post-transplantation)"
#&gt; 
#&gt; $`Donor dendritic cells from cortex of renal allograft (3 days post-transplantation)`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`Donor dendritic cells from cortex of renal allograft (3 days post-transplantation)`$category
#&gt; [1] "BioSource"
#&gt; 
#&gt; $`Donor dendritic cells from cortex of renal allograft (3 days post-transplantation)`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`patient with myelofibrose in remission after transplantation with allogeneic hematopoietic stem cells`
#&gt; $`patient with myelofibrose in remission after transplantation with allogeneic hematopoietic stem cells`$value
#&gt; [1] "patient with myelofibrose in remission after transplantation with allogeneic hematopoietic stem cells"
#&gt; 
#&gt; $`patient with myelofibrose in remission after transplantation with allogeneic hematopoietic stem cells`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`patient with myelofibrose in remission after transplantation with allogeneic hematopoietic stem cells`$category
#&gt; [1] "disease"
#&gt; 
#&gt; $`patient with myelofibrose in remission after transplantation with allogeneic hematopoietic stem cells`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000408"
#&gt; 
#&gt; 
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with prednisolone immediately after transplantation`
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with prednisolone immediately after transplantation`$value
#&gt; [1] "kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with prednisolone immediately after transplantation"
#&gt; 
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with prednisolone immediately after transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with prednisolone immediately after transplantation`$category
#&gt; [1] "organism part"
#&gt; 
#&gt; $`kidney, transplanted from Dark Agouti rats into Lewis rats, high weight difference between donor and recipient, treatment with prednisolone immediately after transplantation`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000635"
#&gt; 
#&gt; 
#&gt; $`Surgery due to:\t\tlung transplantation`
#&gt; $`Surgery due to:\t\tlung transplantation`$value
#&gt; [1] "Surgery due to:\t\tlung transplantation"
#&gt; 
#&gt; $`Surgery due to:\t\tlung transplantation`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`Surgery due to:\t\tlung transplantation`$category
#&gt; NULL
#&gt; 
#&gt; $`Surgery due to:\t\tlung transplantation`$categoryUri
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $`patient with chronic myeloid leukemia in remission prior to transplantation with allogeneic hematopoietic stem cells`
#&gt; $`patient with chronic myeloid leukemia in remission prior to transplantation with allogeneic hematopoietic stem cells`$value
#&gt; [1] "patient with chronic myeloid leukemia in remission prior to transplantation with allogeneic hematopoietic stem cells"
#&gt; 
#&gt; $`patient with chronic myeloid leukemia in remission prior to transplantation with allogeneic hematopoietic stem cells`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`patient with chronic myeloid leukemia in remission prior to transplantation with allogeneic hematopoietic stem cells`$category
#&gt; [1] "disease"
#&gt; 
#&gt; $`patient with chronic myeloid leukemia in remission prior to transplantation with allogeneic hematopoietic stem cells`$categoryUri
#&gt; [1] "http://www.ebi.ac.uk/efo/EFO_0000408"
#&gt; 
#&gt; 
#&gt; $`deterioration of graft function within 2 years after transplantation: no`
#&gt; $`deterioration of graft function within 2 years after transplantation: no`$value
#&gt; [1] "deterioration of graft function within 2 years after transplantation: no"
#&gt; 
#&gt; $`deterioration of graft function within 2 years after transplantation: no`$valueUri
#&gt; NULL
#&gt; 
#&gt; $`deterioration of graft function within 2 years after transplantation: no`$category
#&gt; NULL
#&gt; 
#&gt; $`deterioration of graft function within 2 years after transplantation: no`$categoryUri
#&gt; NULL
#&gt; 
#&gt; </div><div class='input'><span class='fu'>annotationInfo</span>(<span class='st'>'http://purl.obolibrary.org/obo/OBI_0000105'</span>, <span class='kw'>request</span> <span class='kw'>=</span> <span class='st'>'datasets'</span>)</div><div class='output co'>#&gt; $GSE47199
#&gt; $GSE47199$shortName
#&gt; [1] "GSE47199"
#&gt; 
#&gt; $GSE47199$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE47199$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$accession
#&gt; [1] "GSE47199"
#&gt; 
#&gt; $GSE47199$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE47199$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE47199$experimentalDesign
#&gt; [1] 5707
#&gt; 
#&gt; $GSE47199$batchConfound
#&gt; [1] "Factor  'organism part' may be confounded with batches; p=1.8e-09"
#&gt; 
#&gt; $GSE47199$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC1); p=1.4e-05"
#&gt; 
#&gt; $GSE47199$bioMaterialCount
#&gt; [1] 57
#&gt; 
#&gt; $GSE47199$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE47199$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE47199$geeq
#&gt; $GSE47199$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.00114467
#&gt; 
#&gt; $GSE47199$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE47199$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE47199$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$qScoreOutliers
#&gt; [1] -1
#&gt; 
#&gt; $GSE47199$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE47199$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9558614
#&gt; 
#&gt; $GSE47199$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9654504
#&gt; 
#&gt; $GSE47199$geeq$publicQualityScore
#&gt; [1] -0.1477928
#&gt; 
#&gt; $GSE47199$geeq$publicSuitabilityScore
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE47199$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE47199$geeq$id
#&gt; [1] 2015
#&gt; 
#&gt; 
#&gt; $GSE47199$processedExpressionVectorCount
#&gt; [1] 33252
#&gt; 
#&gt; $GSE47199$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE47199$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE47199$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE47199$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$bioAssayCount
#&gt; [1] 57
#&gt; 
#&gt; $GSE47199$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE47199$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE47199$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$description
#&gt; [1] " We study the global gene expression profiles of BKV viremia and nephropathy patients using microarrays in order to better understand the immunologic response to polyomavirus BK (BKV). BKV has become increasingly prevalent since the introduction of more potent immunosuppressive agents. It has been shown that as many as 30% of renal transplant recipients develop asymptomatic viral shedding in the urine shortly after transplant, 10-20% have viremia, and as many as 1-10% can go on to develop overt nephropathy (BKVN) that might lead to graft loss. To date, the genomics of BKV viremia and BKVN have not been investigated thoroughly by microarray.\nLast Updated (by provider): May 23 2013\n\nContributers:  Enver Akalin Pilib Ó Broin\n"
#&gt; 
#&gt; $GSE47199$source
#&gt; NULL
#&gt; 
#&gt; $GSE47199$name
#&gt; [1] "Expression data from blood and biopsies of BKV viremia and nephropathy transplant patients"
#&gt; 
#&gt; $GSE47199$id
#&gt; [1] 6430
#&gt; 
#&gt; $GSE47199$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$lastUpdated
#&gt; [1] 1.519872e+12
#&gt; 
#&gt; $GSE47199$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE47199$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE47199$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE47199$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE47199$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE47199$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE24223
#&gt; $GSE24223$shortName
#&gt; [1] "GSE24223"
#&gt; 
#&gt; $GSE24223$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE24223$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$accession
#&gt; [1] "GSE24223"
#&gt; 
#&gt; $GSE24223$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE24223$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE24223$experimentalDesign
#&gt; [1] 3458
#&gt; 
#&gt; $GSE24223$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE24223$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE24223$bioMaterialCount
#&gt; [1] 179
#&gt; 
#&gt; $GSE24223$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE24223$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE24223$geeq
#&gt; $GSE24223$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.003893934
#&gt; 
#&gt; $GSE24223$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE24223$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE24223$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE24223$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE24223$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE24223$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.8763534
#&gt; 
#&gt; $GSE24223$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.8911214
#&gt; 
#&gt; $GSE24223$geeq$publicQualityScore
#&gt; [1] 0.1273031
#&gt; 
#&gt; $GSE24223$geeq$publicSuitabilityScore
#&gt; [1] 0.75
#&gt; 
#&gt; $GSE24223$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE24223$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE24223$geeq$id
#&gt; [1] 5467
#&gt; 
#&gt; 
#&gt; $GSE24223$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE24223$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE24223$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE24223$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE24223$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$bioAssayCount
#&gt; [1] 179
#&gt; 
#&gt; $GSE24223$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE24223$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE24223$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$description
#&gt; [1] " In this study we employed transcriptome  mRNA profiling of whole blood and purified CD4, CD8 T cells, B cells and monocytes in tandem with high-throughput flow cytometry in 10 kidney transplant patients sampled serially pre-transplant, 1, 2, 4, 8 and 12 weeks. We then mechanistically deconvoluted the early post-transplant immune response. The flow cytometry data confirms depletion of specific cell subsets in response to ATG  induction and immunosuppression with sustained decreases in CD4 as well as CD8 cell subsets. A series of T cell activation markers were expressed from Pre-Tx to 12 weeks indicating the evolution of immunity including expansion of CD45RO+CD62L- effector memory cells. Serial whole blood transcript monitoring demonstrated over 2000 differentially expressed genes, with over 80 percent down-regulated Post-Tx. However, cell subset analysis revealed a unique spectrum of subset-specific gene expression with time-dependent changes, with contrasting significant Post-Tx gene upregulation. Our results provide a unique view of the complex evolution of immune/inflammatory molecular networks marking the early post transplant immune response. A critical finding is that analysis of the constituent blood cell subsets provides an entirely new level of detail revealing the nature of this process, effectively deconvoluting the changes that are otherwise lost in the noise of cellular complexity of whole blood. \nLast Updated (by provider): Aug 18 2011\n\nContributers:  Sunil M Kurian\n"
#&gt; 
#&gt; $GSE24223$source
#&gt; NULL
#&gt; 
#&gt; $GSE24223$name
#&gt; [1] "Deconvoluting Early Post-Transplant Immunity Using Purified Cell Subsets Reveals Functional Networks Not Evident by Whole Blood Analysis"
#&gt; 
#&gt; $GSE24223$id
#&gt; [1] 3195
#&gt; 
#&gt; $GSE24223$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$lastUpdated
#&gt; [1] 1.519869e+12
#&gt; 
#&gt; $GSE24223$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE24223$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE24223$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE24223$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE24223$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE24223$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE14328
#&gt; $GSE14328$shortName
#&gt; [1] "GSE14328"
#&gt; 
#&gt; $GSE14328$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE14328$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE14328$accession
#&gt; [1] "GSE14328"
#&gt; 
#&gt; $GSE14328$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE14328$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE14328$experimentalDesign
#&gt; [1] 2372
#&gt; 
#&gt; $GSE14328$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE14328$batchEffect
#&gt; [1] "Data has been batch-corrected"
#&gt; 
#&gt; $GSE14328$bioMaterialCount
#&gt; [1] 36
#&gt; 
#&gt; $GSE14328$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE14328$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE14328$geeq
#&gt; $GSE14328$geeq$batchCorrected
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE14328$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0003628688
#&gt; 
#&gt; $GSE14328$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE14328$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE14328$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE14328$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE14328$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE14328$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9535394
#&gt; 
#&gt; $GSE14328$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9579753
#&gt; 
#&gt; $GSE14328$geeq$publicQualityScore
#&gt; [1] 0.9939965
#&gt; 
#&gt; $GSE14328$geeq$publicSuitabilityScore
#&gt; [1] 0.6625
#&gt; 
#&gt; $GSE14328$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$geeq$id
#&gt; [1] 1211
#&gt; 
#&gt; 
#&gt; $GSE14328$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE14328$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE14328$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE14328$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE14328$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE14328$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE14328$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE14328$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE14328$bioAssayCount
#&gt; [1] 36
#&gt; 
#&gt; $GSE14328$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE14328$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE14328$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE14328$description
#&gt; [1] " We integrated three transplant rejection microarray studies examining gene expression in samples from pediatric renal, adult renal, and adult heart transplants. We performed one study ourselves and retrieved two others from the NCBI Gene Expression Omnibus (GEO)(GSE4470 and GSE1563). We identified 45 genes that were upregulated in common in acute rejection. Half were involved in one immune-related pathway. Among ten proteins we tested by serum ELISA, three successfully distinguished acute rejection from stable transplants. These were CXCL9, PECAM1, and CD44, with areas under the receiver operating characteristic curves of 0.844, 0.802, and 0.738, respectively. Immunohistochemistry showed that the PECAM1 protein was increased in acute rejection in renal, liver and heart transplants versus normal tissues. Our results show that integrating publicly-available gene expression data sets is a fast, powerful, and cost-effective way to identify serum-detectable diagnostic biomarkers.\nLast Updated (by provider): Dec 15 2010\n\nContributers:  Neeraja Kambham Atul Butte Bryan Klassen Minnie Sarwal Tara Sigdel Sue Hsieh Li Li Rong Chen\n"
#&gt; 
#&gt; $GSE14328$source
#&gt; NULL
#&gt; 
#&gt; $GSE14328$name
#&gt; [1] "Three non-invasive protein biomarkers for solid-organ transplant rejection found through integrative genomics"
#&gt; 
#&gt; $GSE14328$id
#&gt; [1] 2105
#&gt; 
#&gt; $GSE14328$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE14328$lastUpdated
#&gt; [1] 1.519925e+12
#&gt; 
#&gt; $GSE14328$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE14328$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE14328$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE14328$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE14328$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE14328$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE50058
#&gt; $GSE50058$shortName
#&gt; [1] "GSE50058"
#&gt; 
#&gt; $GSE50058$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE50058$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$accession
#&gt; [1] "GSE50058"
#&gt; 
#&gt; $GSE50058$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE50058$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE50058$experimentalDesign
#&gt; [1] 6274
#&gt; 
#&gt; $GSE50058$batchConfound
#&gt; [1] "Factor  'disease' may be confounded with batches; p=3.1e-05"
#&gt; 
#&gt; $GSE50058$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC1); p=1.1e-08"
#&gt; 
#&gt; $GSE50058$bioMaterialCount
#&gt; [1] 101
#&gt; 
#&gt; $GSE50058$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE50058$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE50058$geeq
#&gt; $GSE50058$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.008131342
#&gt; 
#&gt; $GSE50058$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE50058$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE50058$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.8485738
#&gt; 
#&gt; $GSE50058$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.8622591
#&gt; 
#&gt; $GSE50058$geeq$publicQualityScore
#&gt; [1] 0.4088942
#&gt; 
#&gt; $GSE50058$geeq$publicSuitabilityScore
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE50058$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE50058$geeq$id
#&gt; [1] 4052
#&gt; 
#&gt; 
#&gt; $GSE50058$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE50058$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE50058$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE50058$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE50058$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$bioAssayCount
#&gt; [1] 101
#&gt; 
#&gt; $GSE50058$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE50058$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE50058$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$description
#&gt; [1] " Using meta-analysis of eight independent transplant datasets (236 graft biopsy samples) from four organs, we identified a common rejection module (CRM) consisting of 11 genes that were significantly overexpressed in acute rejection (AR) across all transplanted organs. The CRM genes could diagnose AR with high specificity and sensitivity in three additional independent cohorts (794 samples). In another two independent cohorts (151 renal transplant biopsies), the CRM genes correlated with the extent of graft injury and predicted future injury to a graft using protocol  biopsies. Inferred drug mechanisms from the literature suggested that two FDA-approved drugs (atorvastatin and dasatinib), approved for non-transplant indications, could regulate specific CRM genes and reduce the number of graft infiltrating cells during acute rejection. We treated mice with HLA-mismatched murine cardiac transplant with atorvastatin and dasatinib and showed reduction of the CRM genes, significant reduction of graft infiltrating cells, and extended graft survival. We further validated the beneficial effect of atorvastatina on graft survival by retrospective analysis of electronic medical records of a single-center cohort of 2,515 renal transplant patients. In conclusion, we identified a CRM in transplantation that provides new opportunities for diagnosis, drug repositioning and rational drug design. The eight transplant data sets used for discovery include  GSE4315, GSE2596, GSE4470, GSE9377,  GSE1563, GSE9493, GSE13440, GSE6095. The two additional independent cohorts of 151 transplant biopsies are GSE25902 and GSE1563. Not all samples from all studies were used. Only the samples marked as either AR or STA in each data set were used for analysis.\nLast Updated (by provider): Oct 04 2013\n\nContributers:  Purvesh Khatri\n"
#&gt; 
#&gt; $GSE50058$source
#&gt; NULL
#&gt; 
#&gt; $GSE50058$name
#&gt; [1] "A common rejection module for acute rejection in multiple organs identifies novel therapeutics."
#&gt; 
#&gt; $GSE50058$id
#&gt; [1] 7060
#&gt; 
#&gt; $GSE50058$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$lastUpdated
#&gt; [1] 1.519893e+12
#&gt; 
#&gt; $GSE50058$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE50058$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE50058$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE50058$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE50058$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50058$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE37838
#&gt; $GSE37838$shortName
#&gt; [1] "GSE37838"
#&gt; 
#&gt; $GSE37838$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE37838$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE37838$accession
#&gt; [1] "GSE37838"
#&gt; 
#&gt; $GSE37838$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE37838$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE37838$experimentalDesign
#&gt; [1] 5393
#&gt; 
#&gt; $GSE37838$batchConfound
#&gt; [1] "Factor  'CellType' may be confounded with batches; p=0.00088"
#&gt; 
#&gt; $GSE37838$batchEffect
#&gt; [1] "Data has been batch-corrected"
#&gt; 
#&gt; $GSE37838$bioMaterialCount
#&gt; [1] 78
#&gt; 
#&gt; $GSE37838$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE37838$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE37838$geeq
#&gt; $GSE37838$geeq$batchCorrected
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE37838$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 1.973804e-05
#&gt; 
#&gt; $GSE37838$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE37838$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE37838$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE37838$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE37838$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9844451
#&gt; 
#&gt; $GSE37838$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9851267
#&gt; 
#&gt; $GSE37838$geeq$publicQualityScore
#&gt; [1] 0.5693038
#&gt; 
#&gt; $GSE37838$geeq$publicSuitabilityScore
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE37838$geeq$id
#&gt; [1] 1033
#&gt; 
#&gt; 
#&gt; $GSE37838$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE37838$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE37838$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE37838$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE37838$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE37838$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE37838$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE37838$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE37838$bioAssayCount
#&gt; [1] 78
#&gt; 
#&gt; $GSE37838$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE37838$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE37838$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE37838$description
#&gt; [1] " In deceased donor kidney transplantation, acute kidney injury (AKI) prioir to surgery is a major determinant of delayed graft function (DGF), but AKI is histologically silent and difficult to assess. We hypothesized that a molecular measurement of AKI would add power to conventional risk assessments to predict the early poor allograft function at first week post transplantation.\nLast Updated (by provider): Feb 26 2013\n\nContributers:  Philip F Halloran Chatchai Kreepala Konrad S Famulski\n"
#&gt; 
#&gt; $GSE37838$source
#&gt; NULL
#&gt; 
#&gt; $GSE37838$name
#&gt; [1] "Comparing three phenotypes: Comparing molecular assessment of implantation biopsy"
#&gt; 
#&gt; $GSE37838$id
#&gt; [1] 6067
#&gt; 
#&gt; $GSE37838$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE37838$lastUpdated
#&gt; [1] 1.519862e+12
#&gt; 
#&gt; $GSE37838$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE37838$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE37838$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE37838$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE37838$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE37838$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE1563
#&gt; $GSE1563$shortName
#&gt; [1] "GSE1563"
#&gt; 
#&gt; $GSE1563$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE1563$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$accession
#&gt; [1] "GSE1563"
#&gt; 
#&gt; $GSE1563$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE1563$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE1563$experimentalDesign
#&gt; [1] 2582
#&gt; 
#&gt; $GSE1563$batchConfound
#&gt; [1] "Factor  'disease' may be confounded with batches; p=7.8e-05, Factor  'treatment' may be confounded with batches; p=0.0022, Factor  'tissue' may be confounded with batches; p=2.7e-09"
#&gt; 
#&gt; $GSE1563$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC3); p=1.0e-08"
#&gt; 
#&gt; $GSE1563$bioMaterialCount
#&gt; [1] 62
#&gt; 
#&gt; $GSE1563$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE1563$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE1563$geeq
#&gt; $GSE1563$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.002148932
#&gt; 
#&gt; $GSE1563$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE1563$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE1563$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 0
#&gt; 
#&gt; $GSE1563$geeq$sScoreAvgPlatformSize
#&gt; [1] -0.5
#&gt; 
#&gt; $GSE1563$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$geeq$qScoreOutliers
#&gt; [1] -1
#&gt; 
#&gt; $GSE1563$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE1563$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9304672
#&gt; 
#&gt; $GSE1563$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.944799
#&gt; 
#&gt; $GSE1563$geeq$publicQualityScore
#&gt; [1] -0.007885861
#&gt; 
#&gt; $GSE1563$geeq$publicSuitabilityScore
#&gt; [1] 0.6875
#&gt; 
#&gt; $GSE1563$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE1563$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE1563$geeq$id
#&gt; [1] 2180
#&gt; 
#&gt; 
#&gt; $GSE1563$processedExpressionVectorCount
#&gt; [1] 12625
#&gt; 
#&gt; $GSE1563$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE1563$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE1563$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE1563$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$bioAssayCount
#&gt; [1] 62
#&gt; 
#&gt; $GSE1563$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE1563$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE1563$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$description
#&gt; [1] " We used DNA microarrays (HG-U95Av2 GeneChips) to determine gene expression profiles for kidney biopsies and peripheral blood lymphocytes (PBLs) in transplant patients. Sample classes include kidney biopsies and PBLs from patients with 1) healthy normal donor kidneys, 2) well-functioning transplants with no clinical evidence of rejection, 3) kidneys undergoing acute rejection, and 4) transplants with renal dysfunction without rejection. Nomenclature for samples is as follows: 1) all sample names include either BX or PBL to indicate that they were derived from biopsies or PBLs respectively, 2) C indicates samples from healthy normal donors, 3) TX indicates samples from patients with well-functioning transplants with no clinical evidence of rejection, 3) AR indicates samples from transplant patients with kidneys undergoing acute rejection, 4) NR indicates samples from transplant patients with renal dysfunction without rejection. Abbreviations used to describe patient samples include the following: BX - Biopsy; PBL- Peripheral Blood Lymphocytes; CsA -Cyclosporine; MMF - Mycophenolate Mofetil; P - Prednisone; FK - Tacrolimus;  SRL - Sirolimus; CAD -Cadaveric;  LD - Live Donor; Scr - Serum Creatinine; ATN - Acute Tubular Necrosis CNI - Calcineurin Inhibitor; FSGS - Focal Segmental Glomerulosclerosis several array data sets did not pass quality control and were not analyzed. These include AR1PBL, NR4BX, and NR6PBL\nLast Updated (by provider): May 29 2005\n\nContributers:  Daniel J Cook Steven R Head Starlette M Sharp Jie Zhang Steven A Kay Daniel R Salomon Thomas C Whisenant Timothy Gilmartin Jeffrey D Chismar Sunil M Kurian Stuart M Flechner Tony Mondala Steve Horvath John R Walker\n\nIncludes GDS724.\n Update date: Sep 21 2004.\n Dataset description GDS724: Analysis of kidney transplant rejection by expression profiling of kidney biopsies and peripheral blood lymphocytes from patients. Results identify expression profiles unique to rejection, dysfunction without rejection, and well-functioning transplants..\n"
#&gt; 
#&gt; $GSE1563$source
#&gt; NULL
#&gt; 
#&gt; $GSE1563$name
#&gt; [1] "Kidney Transplant Rejection and Tissue Injury by Gene Profiling of Biopsies and Peripheral Blood Lymphocytes"
#&gt; 
#&gt; $GSE1563$id
#&gt; [1] 2314
#&gt; 
#&gt; $GSE1563$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$lastUpdated
#&gt; [1] 1.519934e+12
#&gt; 
#&gt; $GSE1563$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE1563$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE1563$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE1563$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE1563$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1563$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE8021
#&gt; $GSE8021$shortName
#&gt; [1] "GSE8021"
#&gt; 
#&gt; $GSE8021$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE8021$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE8021$accession
#&gt; [1] "GSE8021"
#&gt; 
#&gt; $GSE8021$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE8021$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE8021$experimentalDesign
#&gt; [1] 726
#&gt; 
#&gt; $GSE8021$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE8021$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC3); p=1.8e-05"
#&gt; 
#&gt; $GSE8021$bioMaterialCount
#&gt; [1] 50
#&gt; 
#&gt; $GSE8021$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE8021$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE8021$geeq
#&gt; $GSE8021$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE8021$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0
#&gt; 
#&gt; $GSE8021$geeq$noVectors
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE8021$geeq$corrMatIssues
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE8021$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$geeq$sScoreAvgPlatformPopularity
#&gt; [1] -1
#&gt; 
#&gt; $GSE8021$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE8021$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE8021$geeq$sScoreMissingValues
#&gt; [1] -1
#&gt; 
#&gt; $GSE8021$geeq$qScoreOutliers
#&gt; [1] -1
#&gt; 
#&gt; $GSE8021$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0
#&gt; 
#&gt; $GSE8021$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0
#&gt; 
#&gt; $GSE8021$geeq$publicQualityScore
#&gt; [1] 0.2857143
#&gt; 
#&gt; $GSE8021$geeq$publicSuitabilityScore
#&gt; [1] 0.125
#&gt; 
#&gt; $GSE8021$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE8021$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$geeq$id
#&gt; [1] 1972
#&gt; 
#&gt; 
#&gt; $GSE8021$processedExpressionVectorCount
#&gt; [1] 17635
#&gt; 
#&gt; $GSE8021$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE8021$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE8021$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE8021$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE8021$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE8021$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE8021$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE8021$bioAssayCount
#&gt; [1] 50
#&gt; 
#&gt; $GSE8021$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE8021$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE8021$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE8021$description
#&gt; [1] " Expression profile of human donor lungs that have developed primary graft dysfunction (PGD) after lung transplantation and those that have not. In this study, we attempt to further our understanding of PGD by observing the changes in gene expression across donor lungs that developed PGD versus those that did not.\r\nLast Updated (by provider): Jun 06 2007\r\n\r\nContributers:  Sekhar Dharmarajan Monika Ray\r\n"
#&gt; 
#&gt; $GSE8021$source
#&gt; [1] ""
#&gt; 
#&gt; $GSE8021$name
#&gt; [1] "Expression data from human donor lung biopsies"
#&gt; 
#&gt; $GSE8021$id
#&gt; [1] 619
#&gt; 
#&gt; $GSE8021$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE8021$lastUpdated
#&gt; [1] 1.519875e+12
#&gt; 
#&gt; $GSE8021$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE8021$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE8021$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE8021$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE8021$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE8021$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE22459
#&gt; $GSE22459$shortName
#&gt; [1] "GSE22459"
#&gt; 
#&gt; $GSE22459$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE22459$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$accession
#&gt; [1] "GSE22459"
#&gt; 
#&gt; $GSE22459$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE22459$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE22459$experimentalDesign
#&gt; [1] 3378
#&gt; 
#&gt; $GSE22459$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE22459$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE22459$bioMaterialCount
#&gt; [1] 65
#&gt; 
#&gt; $GSE22459$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE22459$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE22459$geeq
#&gt; $GSE22459$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.001311401
#&gt; 
#&gt; $GSE22459$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE22459$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE22459$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE22459$geeq$sScoreMissingValues
#&gt; [1] -1
#&gt; 
#&gt; $GSE22459$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE22459$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9423929
#&gt; 
#&gt; $GSE22459$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9544383
#&gt; 
#&gt; $GSE22459$geeq$publicQualityScore
#&gt; [1] 0.4220626
#&gt; 
#&gt; $GSE22459$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE22459$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE22459$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE22459$geeq$id
#&gt; [1] 5099
#&gt; 
#&gt; 
#&gt; $GSE22459$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE22459$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE22459$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE22459$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE22459$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$bioAssayCount
#&gt; [1] 65
#&gt; 
#&gt; $GSE22459$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE22459$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE22459$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$description
#&gt; [1] " We previously observed reduced graft survival for kidney transplants having interstitial fibrosis with subclinical inflammation, but not fibrosis alone, on 1-year protocol biopsy. The current study aimed to determine whether fibrosis with inflammation at 1 year is associated with renal functional decline in a low-risk transplant cohort and to characterize the nature of the inflammation. Subjects were living-donor, tacrolimus/mycophenolate-treated transplant recipients without overt risk factors for reduced graft survival (n=151). Transplants with normal histology (n=86) or fibrosis alone (n=45) on 1-year protocol biopsy had stable renal function between 1 and 5 years, while those having fibrosis with inflammation (n=20) had declining glomerular filtration rate and reduced graft survival. Immunohistochemistry confirmed increased interstitial T-cells and macrophages/dendritic cells in the fibrosis with inflammation group. Gene expression was performed on a subset of biopsies in each group and demonstrated increased expression of transcripts related to innate and cognate immunity in transplants having fibrosis with inflammation. Pathway- and pathological process-specific analyses of microarray profiles revealed that, in fibrosis with inflammation, over-expressed transcripts were enriched for potentially damaging immunological activities including Toll-like receptor signaling, antigen presentation/dendritic cell maturation, interferon gamma-inducible response, cytotoxic T lymphocyte-associated and acute rejection-associated genes. Thus, fibrosis with inflammation in 1-year protocol biopsies is associated with reduced graft survival and function and with a rejection-like gene expression signature even in recipients with no clinical risk for inferior outcome. Early interventions aimed at altering rejection-like inflammation may favor improved long-term KTx survival.\nLast Updated (by provider): Aug 18 2011\n\nContributers:  Fernando G Cosio Lynn D Cornell Matthew D Griffin Walter D Park Mark D Stegall\n"
#&gt; 
#&gt; $GSE22459$source
#&gt; NULL
#&gt; 
#&gt; $GSE22459$name
#&gt; [1] "Fibrosis with Inflammation at One Year Predicts Transplant Functional Decline"
#&gt; 
#&gt; $GSE22459$id
#&gt; [1] 3115
#&gt; 
#&gt; $GSE22459$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$lastUpdated
#&gt; [1] 1.520134e+12
#&gt; 
#&gt; $GSE22459$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE22459$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE22459$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE22459$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE22459$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22459$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE42742
#&gt; $GSE42742$shortName
#&gt; [1] "GSE42742"
#&gt; 
#&gt; $GSE42742$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE42742$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$accession
#&gt; [1] "GSE42742"
#&gt; 
#&gt; $GSE42742$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE42742$taxon
#&gt; [1] "mouse"
#&gt; 
#&gt; $GSE42742$experimentalDesign
#&gt; [1] 7132
#&gt; 
#&gt; $GSE42742$batchConfound
#&gt; [1] "Factor 'genotype' may be confounded with batches; p=3.8e-09"
#&gt; 
#&gt; $GSE42742$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC2); p=7.0e-07"
#&gt; 
#&gt; $GSE42742$bioMaterialCount
#&gt; [1] 56
#&gt; 
#&gt; $GSE42742$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE42742$taxonId
#&gt; [1] 2
#&gt; 
#&gt; $GSE42742$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE42742$geeq
#&gt; $GSE42742$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0007356624
#&gt; 
#&gt; $GSE42742$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE42742$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE42742$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE42742$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.940714
#&gt; 
#&gt; $GSE42742$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9458795
#&gt; 
#&gt; $GSE42742$geeq$publicQualityScore
#&gt; [1] 0.2779828
#&gt; 
#&gt; $GSE42742$geeq$publicSuitabilityScore
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE42742$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE42742$geeq$id
#&gt; [1] 7304
#&gt; 
#&gt; 
#&gt; $GSE42742$processedExpressionVectorCount
#&gt; [1] 45101
#&gt; 
#&gt; $GSE42742$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE42742$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE42742$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE42742$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$bioAssayCount
#&gt; [1] 56
#&gt; 
#&gt; $GSE42742$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE42742$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE42742$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$description
#&gt; [1] " We developed a mouse model that captures radiation effects on host biology by transplanting unirradiated Trp53 null mammary tissue to sham or irradiated hosts. Gene expression profiles of tumors that arose in irradiated mice are distinct from those that arose in naïve hosts. Host irradiation induces a metaprofile consisting of gene modules representing stem cells, cell motility, macrophages and autophagy. Human orthologs of the host irradiation metaprofile discriminated between radiation-preceded and sporadic human thyroid cancers. An irradiated host centroid was strongly associated with estrogen receptor negative breast cancer. When applied to sporadic human breast cancers, the irradiated host metaprofile strongly associated with basal-like and claudin-low breast cancer intrinsic subtypes. Comparing host irradiation in the context of TGF? levels showed that inflammation was robustly associated with claudin-low tumors.  The association of the irradiated host metaprofiles with estrogen receptor negative status and claudin-low subtype suggests that host processes similar to those induced by radiation underlie sporadic cancers.\nLast Updated (by provider): Sep 23 2013\n\nContributers:  David H Nguyen Mary H Barcellos-Hoff Wei Zhao Charles M Perou Erik Fredlund Jian-Hua Mao Allan Balmain\n"
#&gt; 
#&gt; $GSE42742$source
#&gt; NULL
#&gt; 
#&gt; $GSE42742$name
#&gt; [1] "Murine microenvironment metaprofiles associate with human cancer etiology and intrinsic subtypes"
#&gt; 
#&gt; $GSE42742$id
#&gt; [1] 7946
#&gt; 
#&gt; $GSE42742$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$lastUpdated
#&gt; [1] 1.519932e+12
#&gt; 
#&gt; $GSE42742$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE42742$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE42742$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE42742$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE42742$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE42742$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE15480
#&gt; $GSE15480$shortName
#&gt; [1] "GSE15480"
#&gt; 
#&gt; $GSE15480$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE15480$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE15480$accession
#&gt; [1] "GSE15480"
#&gt; 
#&gt; $GSE15480$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE15480$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE15480$experimentalDesign
#&gt; [1] 4424
#&gt; 
#&gt; $GSE15480$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE15480$batchEffect
#&gt; [1] "Data has been batch-corrected"
#&gt; 
#&gt; $GSE15480$bioMaterialCount
#&gt; [1] 24
#&gt; 
#&gt; $GSE15480$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE15480$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE15480$geeq
#&gt; $GSE15480$geeq$batchCorrected
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE15480$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 2.917751e-05
#&gt; 
#&gt; $GSE15480$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE15480$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE15480$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE15480$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE15480$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE15480$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE15480$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9790369
#&gt; 
#&gt; $GSE15480$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9800342
#&gt; 
#&gt; $GSE15480$geeq$publicQualityScore
#&gt; [1] 0.8542906
#&gt; 
#&gt; $GSE15480$geeq$publicSuitabilityScore
#&gt; [1] 0.6625
#&gt; 
#&gt; $GSE15480$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$geeq$id
#&gt; [1] 2741
#&gt; 
#&gt; 
#&gt; $GSE15480$processedExpressionVectorCount
#&gt; [1] 33252
#&gt; 
#&gt; $GSE15480$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE15480$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE15480$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE15480$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE15480$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE15480$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE15480$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE15480$bioAssayCount
#&gt; [1] 24
#&gt; 
#&gt; $GSE15480$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE15480$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE15480$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE15480$description
#&gt; [1] " The clear benefits of ischemic preconditioning (IPC) in reducing ischemia reperfusion injury (IRI) remain indistinct in human liver transplantation. To further understand the mechanistic aspects of IPC in human deceased donor liver transplantation (DDLT), we performed microarray analyses to determine global gene expression profiles associated with IPC administration. Donor and recipient characteristics in both groups were comparable. Clinical data from our study subset and larger trial were similar.  IPC increased expression of 10 transcripts at either time point with roles in: antioxidant defenses, immunological response, lipid biosynthesis, and xenobiotic metabolism.  IPC decreased the expression of 1 cell development related transcript.  Conclusions: 1) IPC in DDLT increased the expression of antioxidant transcripts similar to studies in animal IPC, anesthetic, and remote IPC.  2) IPC increased expression of lipogenic transcripts, which may be relevant to the clinically observed increased IRI in our IPC group. 3) Our microarray findings support our clinical observations and are compatible with the varied outcomes of hepatic IPC studies in human liver transplantation.\nLast Updated (by provider): May 25 2012\n\nContributers:  Baburao Koneru Patricia Soteropoulus Helen Fernandes George Dikdan Ali Raza\n"
#&gt; 
#&gt; $GSE15480$source
#&gt; NULL
#&gt; 
#&gt; $GSE15480$name
#&gt; [1] "Global Gene Expression in Deceased Donor Liver Transplantation with Ischemic Preconditioning"
#&gt; 
#&gt; $GSE15480$id
#&gt; [1] 4868
#&gt; 
#&gt; $GSE15480$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE15480$lastUpdated
#&gt; [1] 1.519906e+12
#&gt; 
#&gt; $GSE15480$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE15480$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE15480$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE15480$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE15480$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE15480$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE7392
#&gt; $GSE7392$shortName
#&gt; [1] "GSE7392"
#&gt; 
#&gt; $GSE7392$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE7392$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE7392$accession
#&gt; [1] "GSE7392"
#&gt; 
#&gt; $GSE7392$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE7392$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE7392$experimentalDesign
#&gt; [1] 7413
#&gt; 
#&gt; $GSE7392$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE7392$batchEffect
#&gt; [1] "Data has been batch-corrected"
#&gt; 
#&gt; $GSE7392$bioMaterialCount
#&gt; [1] 30
#&gt; 
#&gt; $GSE7392$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE7392$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE7392$geeq
#&gt; $GSE7392$geeq$batchCorrected
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE7392$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.723132e-05
#&gt; 
#&gt; $GSE7392$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE7392$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE7392$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE7392$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE7392$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE7392$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9794048
#&gt; 
#&gt; $GSE7392$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9803522
#&gt; 
#&gt; $GSE7392$geeq$publicQualityScore
#&gt; [1] 0.854336
#&gt; 
#&gt; $GSE7392$geeq$publicSuitabilityScore
#&gt; [1] 0.9125
#&gt; 
#&gt; $GSE7392$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$geeq$id
#&gt; [1] 7710
#&gt; 
#&gt; 
#&gt; $GSE7392$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE7392$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE7392$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE7392$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE7392$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE7392$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE7392$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE7392$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE7392$bioAssayCount
#&gt; [1] 30
#&gt; 
#&gt; $GSE7392$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE7392$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE7392$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE7392$description
#&gt; [1] " Introduction.  Factors contributing to kidney transplant fibrosis remain incompletely understood—particularly in the absence of acute complications. Methods.   Baseline and one-year surveillance biopsies from 15 uncomplicated living donor kidney transplants were subjected to microarray and quantitative RT-PCR (qRT-PCR) analyses in order to examine changes in gene expression patterns over time. Biopsy pairs were purposefully selected from allografts with no history of acute complications and were divided into those that were histologically normal (n = 7) and those that had developed subclinical interstitial fibrosis (n = 8) at 1 year. Results.   Compared to the paired baseline specimens, expression levels of 3578 probesets were found altered in all the one-year biopsies studied. A large proportion of the upregulated genes in this transplant-associated profile were functionally linked with inflammation, immunity or response to injury. These included components of inflammation-related signaling pathways (integrin, interferon and TLR) as well as individual mediators of inflammatory and immune responses. An additional 2884 probesets demonstrated altered expression in fibrotic grafts only at 1 year. The gene products in this fibrosis-associated profile were also predominantly linked with inflammation and immune function, suggesting exaggerated inflammatory activity within the fibrotic grafts. qRT-PCR analyses confirmed the predicted expression patterns for selected transcripts from the microarray profiles. Conclusions.   Transcriptional profiles of histologically normal living donor renal allografts indicate that there is ongoing injury response and inflammation at 1 year compared to the immediate post-transplant period. Subclinical development of interstitial fibrosis during the first post-transplant year is associated with additional upregulation of inflammation-related genes.\nLast Updated (by provider): Dec 06 2013\n\nContributers:  Matthew Griffin Walter D Park Fernando Cosio Joseph P Grande Mark D Stegall\n"
#&gt; 
#&gt; $GSE7392$source
#&gt; NULL
#&gt; 
#&gt; $GSE7392$name
#&gt; [1] "Molecular Evidence of Injury and Inflammation in Normal and Fibrotic Renal Allografts One Year Post-Transplant"
#&gt; 
#&gt; $GSE7392$id
#&gt; [1] 8231
#&gt; 
#&gt; $GSE7392$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE7392$lastUpdated
#&gt; [1] 1.519944e+12
#&gt; 
#&gt; $GSE7392$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE7392$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE7392$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE7392$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE7392$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE7392$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE5967
#&gt; $GSE5967$shortName
#&gt; [1] "GSE5967"
#&gt; 
#&gt; $GSE5967$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE5967$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5967$accession
#&gt; [1] "GSE5967"
#&gt; 
#&gt; $GSE5967$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE5967$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE5967$experimentalDesign
#&gt; [1] 7374
#&gt; 
#&gt; $GSE5967$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE5967$batchEffect
#&gt; [1] "Data has been batch-corrected"
#&gt; 
#&gt; $GSE5967$bioMaterialCount
#&gt; [1] 21
#&gt; 
#&gt; $GSE5967$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE5967$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE5967$geeq
#&gt; $GSE5967$geeq$batchCorrected
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE5967$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0003208338
#&gt; 
#&gt; $GSE5967$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5967$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE5967$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE5967$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE5967$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE5967$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE5967$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9658853
#&gt; 
#&gt; $GSE5967$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9711445
#&gt; 
#&gt; $GSE5967$geeq$publicQualityScore
#&gt; [1] 0.8530206
#&gt; 
#&gt; $GSE5967$geeq$publicSuitabilityScore
#&gt; [1] 0.7875
#&gt; 
#&gt; $GSE5967$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$geeq$id
#&gt; [1] 7521
#&gt; 
#&gt; 
#&gt; $GSE5967$processedExpressionVectorCount
#&gt; [1] 22283
#&gt; 
#&gt; $GSE5967$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE5967$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5967$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE5967$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5967$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5967$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE5967$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5967$bioAssayCount
#&gt; [1] 21
#&gt; 
#&gt; $GSE5967$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE5967$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE5967$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5967$description
#&gt; [1] " BACKGROUND: Assessment of gene expression in peripheral blood may provide a noninvasive screening test for allograft rejection. We hypothesized that changes in peripheral blood expression profiles would correlate with biopsy-proven rejection and would resolve after treatment of rejection episodes. METHODS AND RESULTS: We performed a case-control study nested within a cohort of 189 cardiac transplant patients who had blood samples obtained during endomyocardial biopsy (EMB). Using Affymetrix HU133A microarrays, we analyzed whole-blood expression profiles from 3 groups: (1) control samples with negative EMB (n=7); (2) samples obtained during rejection (at least International Society for Heart and Lung Transplantation grade 3A; n=7); and (3) samples obtained after rejection, after treatment and normalization of the EMB (n=7). We identified 91 transcripts differentially expressed in rejection compared with control (false discovery rate &lt;0.10). In postrejection samples, 98% of transcripts returned toward control levels, displaying an intermediate expression profile for patients with treated rejection (P&lt;0.0001). Cluster analysis of the 40 transcripts with &gt;25% change in expression levels during rejection demonstrated good discrimination between control and rejection samples and verified the intermediate expression profile of postrejection samples. Quantitative real-time polymerase chain reaction confirmed significant differential expression for the predictive markers CFLAR and SOD2 (UniGene ID No. 355724 and No. 384944). CONCLUSIONS: These data demonstrate that peripheral blood expression profiles correlate with biopsy-proven allograft rejection. Intermediate expression profiles of treated rejection suggest persistent immune activation despite normalization of the EMB. If validated in larger studies, expression profiling may prove to be a more sensitive screening test for allograft rejection than EMB.\nLast Updated (by provider): Aug 29 2013\n\nContributers:  Jonathan A Epstein Philip A Horwitz Emily J Tsai Thomas P Cappola Mary E Putt\n\nIncludes GDS2386.\n Update date: Feb 14 2013.\n Dataset description GDS2386: Analysis of peripheral blood from cardiac transplant patients with allograft rejection and from the same patients after resolution of rejection with augmented immunosuppression. Results provide insight into potential use of peripheral blood profiling for detection of cardiac allograft rejection..\n"
#&gt; 
#&gt; $GSE5967$source
#&gt; NULL
#&gt; 
#&gt; $GSE5967$name
#&gt; [1] "Detection of cardiac allograft rejection and response to immunosuppressive therapy with peripheral blood gene expression"
#&gt; 
#&gt; $GSE5967$id
#&gt; [1] 8192
#&gt; 
#&gt; $GSE5967$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5967$lastUpdated
#&gt; [1] 1.519942e+12
#&gt; 
#&gt; $GSE5967$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE5967$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE5967$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE5967$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE5967$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5967$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE30718
#&gt; $GSE30718$shortName
#&gt; [1] "GSE30718"
#&gt; 
#&gt; $GSE30718$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE30718$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$accession
#&gt; [1] "GSE30718"
#&gt; 
#&gt; $GSE30718$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE30718$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE30718$experimentalDesign
#&gt; [1] 4667
#&gt; 
#&gt; $GSE30718$batchConfound
#&gt; [1] "Factor  'Treatment' may be confounded with batches; p=0.00020"
#&gt; 
#&gt; $GSE30718$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC1); p=0.0055"
#&gt; 
#&gt; $GSE30718$bioMaterialCount
#&gt; [1] 47
#&gt; 
#&gt; $GSE30718$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE30718$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE30718$geeq
#&gt; $GSE30718$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.001337505
#&gt; 
#&gt; $GSE30718$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE30718$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE30718$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE30718$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$geeq$qScoreOutliers
#&gt; [1] -1
#&gt; 
#&gt; $GSE30718$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE30718$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9446366
#&gt; 
#&gt; $GSE30718$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9556942
#&gt; 
#&gt; $GSE30718$geeq$publicQualityScore
#&gt; [1] 0.1365277
#&gt; 
#&gt; $GSE30718$geeq$publicSuitabilityScore
#&gt; [1] 0.9125
#&gt; 
#&gt; $GSE30718$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE30718$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE30718$geeq$id
#&gt; [1] 3752
#&gt; 
#&gt; 
#&gt; $GSE30718$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE30718$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE30718$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE30718$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE30718$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$bioAssayCount
#&gt; [1] 47
#&gt; 
#&gt; $GSE30718$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE30718$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE30718$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$description
#&gt; [1] " Microarray analysis of human kidneys with acute kidney injury (AKI) has been limited because such kidneys are seldom biopsied. However, all kidney transplants experience AKI, and early kidney transplants without rejection are an excellent model for human AKI: they are screened to exclude chronic kidney disease, frequently biopsied, and have extensive follow-up. We used histopathology and microarrays to compare indication biopsies from 28 transplants with AKI to 11 pristine protocol biopsies of stable transplants. Kidneys with AKI showed increased expression of 394 injury-repair response associated transcripts, including many known epithelial injury molecules (e.g. ITGB6, LCN2), tissue remodeling molecules (e.g. VCAN), and inflammation molecules (S100A8, ITGB3). Many other genes also predict the phenotype, depending on statistical filtering rules, including AKI biomarkers as HAVCR1 and IL18. Most mouse orthologs of the top injury-repair transcripts were increased in published mouse AKI models. Pathway analysis of the injury-repair transcripts revealed similarities to cancer, development, and cell movement. The injury-repair transcript score AKI kidneys correlated with reduced function, future recovery, brain death, and need for dialysis, but not future graft loss. In contrast, histologic features of \"acute tubular injury\" did not correlate with function or with the molecular changes. Thus the injury-repair associated transcripts represent a massive coordinate injury-repair response of kidney parenchyma to AKI, similar to mouse AKI models, and provide an objective measure for assessing the severity of AKI in kidney biopsies and validation for the use of many AKI biomarkers.\nLast Updated (by provider): May 09 2012\n\nContributers:  Philip F Halloran Declan G de Freitas Konrad S Famulski\n"
#&gt; 
#&gt; $GSE30718$source
#&gt; NULL
#&gt; 
#&gt; $GSE30718$name
#&gt; [1] "Molecular Phenotypes of Acute Kidney Injury in Kidney Transplants"
#&gt; 
#&gt; $GSE30718$id
#&gt; [1] 5312
#&gt; 
#&gt; $GSE30718$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$lastUpdated
#&gt; [1] 1.519915e+12
#&gt; 
#&gt; $GSE30718$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE30718$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE30718$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE30718$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE30718$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30718$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE50084
#&gt; $GSE50084$shortName
#&gt; [1] "GSE50084"
#&gt; 
#&gt; $GSE50084$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE50084$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$accession
#&gt; [1] "GSE50084"
#&gt; 
#&gt; $GSE50084$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE50084$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE50084$experimentalDesign
#&gt; [1] 6110
#&gt; 
#&gt; $GSE50084$batchConfound
#&gt; [1] "Factor  'organism part' may be confounded with batches; p=0.0, Factor  'disease' may be confounded with batches; p=1.2e-05"
#&gt; 
#&gt; $GSE50084$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC1); p=3.2e-14"
#&gt; 
#&gt; $GSE50084$bioMaterialCount
#&gt; [1] 115
#&gt; 
#&gt; $GSE50084$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE50084$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE50084$geeq
#&gt; $GSE50084$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0004252722
#&gt; 
#&gt; $GSE50084$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE50084$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE50084$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$qScoreOutliers
#&gt; [1] -1
#&gt; 
#&gt; $GSE50084$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9566969
#&gt; 
#&gt; $GSE50084$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9626142
#&gt; 
#&gt; $GSE50084$geeq$publicQualityScore
#&gt; [1] 0.1375163
#&gt; 
#&gt; $GSE50084$geeq$publicSuitabilityScore
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE50084$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE50084$geeq$id
#&gt; [1] 3470
#&gt; 
#&gt; 
#&gt; $GSE50084$processedExpressionVectorCount
#&gt; [1] 33252
#&gt; 
#&gt; $GSE50084$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE50084$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE50084$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE50084$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$bioAssayCount
#&gt; [1] 115
#&gt; 
#&gt; $GSE50084$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE50084$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE50084$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$description
#&gt; [1] " The presence of Donor-Specific anti-HLA Antibodies (DSA) is associated with an increased risk of both acute and chronic antibody-mediated rejection (AMR) in kidney allografts.  AMR has remained challenging in kidney transplantation and is the major cause of late allograft loss. However, not all patients with DSA develop AMR, leading to the question of whether this represents accommodation, if other protective mechanisms exist or if this is actually a state of pre-rejection. Clinical and histological features, and gene expression profiles of kidney biopsy and blood samples of donor-specific antibody (DSA)+ patients without rejection were compared to antibody-mediated rejection (AMR) patients to elucidate the mechanisms involved in prevention of AMR. Of the 71 DSA+ patients, 46 had diagnosis of AMR and 25 did not show rejection. 50 DSA- patients without rejection were used as control. A subgroup of patients with available biopsy (n=61) and blood samples (n=54) were analyzed by microarrays.  Both, DSA+/AMR+ and DSA+/AMR- biopsies showed increased expression of gene transcripts associated with cytotoxic T, natural killer cells, macrophages, interferon-gamma and rejection compared to DSA- biopsies. Regulatory T cell transcripts were up-regulated in DSA+/AMR+ and B cell transcripts in DSA+/AMR- biopsies. Whole blood gene expression analysis showed increased immune activity in only DSA+/AMR+ patients. There were no differentially expressed tolerant genes studied (n=14) in the blood or biopsy specimens of DSA+/AMR- patients. During a median 36 months follow-up, 4 DSA+/AMR- patients developed AMR, 12 continued to have DSAs but 9 lost DSAs. Gene expression profiles did not predict the development of AMR or persistence of DSAs. These results indicate increased immune activity in DSA+/AMR- biopsies despite lack of histologic findings of rejection.\nLast Updated (by provider): Aug 24 2013\n\nContributers:  Enver Akalin Pilib Ó Broin\n"
#&gt; 
#&gt; $GSE50084$source
#&gt; NULL
#&gt; 
#&gt; $GSE50084$name
#&gt; [1] "Expression data from blood and biopsies of Donor-Specific Antibody positive patients"
#&gt; 
#&gt; $GSE50084$id
#&gt; [1] 6881
#&gt; 
#&gt; $GSE50084$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$lastUpdated
#&gt; [1] 1.519887e+12
#&gt; 
#&gt; $GSE50084$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE50084$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE50084$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE50084$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE50084$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE50084$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE11166
#&gt; $GSE11166$shortName
#&gt; [1] "GSE11166"
#&gt; 
#&gt; $GSE11166$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE11166$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$accession
#&gt; [1] "GSE11166"
#&gt; 
#&gt; $GSE11166$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE11166$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE11166$experimentalDesign
#&gt; [1] 2504
#&gt; 
#&gt; $GSE11166$batchConfound
#&gt; [1] "Factor 'SamplingTimePoint' may be confounded with batches; p=0.00028, Factor 'experimental block' may be confounded with batches; p=0.0060, Factor 'Phenotype' may be confounded with batches; p=0.00052"
#&gt; 
#&gt; $GSE11166$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC1); p=0.0011"
#&gt; 
#&gt; $GSE11166$bioMaterialCount
#&gt; [1] 95
#&gt; 
#&gt; $GSE11166$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE11166$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE11166$geeq
#&gt; $GSE11166$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0004434611
#&gt; 
#&gt; $GSE11166$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE11166$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE11166$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE11166$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9502656
#&gt; 
#&gt; $GSE11166$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9542799
#&gt; 
#&gt; $GSE11166$geeq$publicQualityScore
#&gt; [1] 0.2791828
#&gt; 
#&gt; $GSE11166$geeq$publicSuitabilityScore
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE11166$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE11166$geeq$id
#&gt; [1] 1826
#&gt; 
#&gt; 
#&gt; $GSE11166$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE11166$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE11166$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE11166$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE11166$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$bioAssayCount
#&gt; [1] 95
#&gt; 
#&gt; $GSE11166$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE11166$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE11166$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$description
#&gt; [1] " The biopsy samples obtained at implantation segregated in 2 distinct groups according to donor origin, with a cluster of 319 unique identified genes higher expressed in DD compared to LD kidneys, and 329 genes lower expressed (false discovery rate &lt;5%). Using pathway analysis software a significant local renal overrepresentation of complement genes in DD implantation biopsies was identified. Complement gene expression in DD kidneys related both to donor death and cold ischemia duration, and was associated with a slower onset of renal allograft function. In post-transplantation protocol biopsies, there was a continued overexpression of complement genes, regardless of donor source. The local renal complement gene expression variability in post-transplantation biopsies correlated with renal graft function.  This study demonstrates a significant and clinically relevant local overexpression of complement genes in DD kidneys at engraftment and continuous functionally important regulation of complement gene expression after transplantation, regardless of donor source. Targeted therapy interfering with complement activation is an attractive therapeutic target that deserves further investigation in solid organ transplantation.\nLast Updated (by provider): Dec 15 2010\n\nContributers:  Atul J Butte Poonam Sansanwal Neeraja Kambham Evelyne Lerut Lihua Ying Szu-chuan Hsieh Minnie M Sarwal Oscar Salvatierra Tara Sigdel Li Li Maarten Naesens\n"
#&gt; 
#&gt; $GSE11166$source
#&gt; NULL
#&gt; 
#&gt; $GSE11166$name
#&gt; [1] "Local regulation and clinical impact of complement gene expression in deceased and living donor kidney allografts"
#&gt; 
#&gt; $GSE11166$id
#&gt; [1] 2236
#&gt; 
#&gt; $GSE11166$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$lastUpdated
#&gt; [1] 1.519931e+12
#&gt; 
#&gt; $GSE11166$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE11166$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE11166$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE11166$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE11166$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE11166$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE22707
#&gt; $GSE22707$shortName
#&gt; [1] "GSE22707"
#&gt; 
#&gt; $GSE22707$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE22707$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$accession
#&gt; [1] "GSE22707"
#&gt; 
#&gt; $GSE22707$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE22707$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE22707$experimentalDesign
#&gt; [1] 4502
#&gt; 
#&gt; $GSE22707$batchConfound
#&gt; [1] "Factor 'treatment' may be confounded with batches; p=2.1e-10, Factor 'disease staging' may be confounded with batches; p=3.1e-12"
#&gt; 
#&gt; $GSE22707$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC1); p=0.0013"
#&gt; 
#&gt; $GSE22707$bioMaterialCount
#&gt; [1] 72
#&gt; 
#&gt; $GSE22707$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE22707$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE22707$geeq
#&gt; $GSE22707$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0001185251
#&gt; 
#&gt; $GSE22707$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE22707$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE22707$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9722885
#&gt; 
#&gt; $GSE22707$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9749441
#&gt; 
#&gt; $GSE22707$geeq$publicQualityScore
#&gt; [1] 0.5678492
#&gt; 
#&gt; $GSE22707$geeq$publicSuitabilityScore
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE22707$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE22707$geeq$id
#&gt; [1] 3052
#&gt; 
#&gt; 
#&gt; $GSE22707$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE22707$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE22707$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE22707$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE22707$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$bioAssayCount
#&gt; [1] 72
#&gt; 
#&gt; $GSE22707$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE22707$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE22707$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$description
#&gt; [1] " Transplant recipients spontaneously accepting their grafts in the absence of immunosuppression demonstrate the feasibility of attaining allograft tolerance in humans. Previous studies have identified blood transcriptional and cell phenotypic markers specific for either liver or kidney tolerant recipients, but the two settings have not been directly compared yet employing the same platforms. To identify potential similarities in immune parameters between recipients tolerant to different organs, we analyzed blood samples from tolerant and non-tolerant liver and kidney recipients employing whole genome expression microarrays. Tolerant and non-tolerant liver and kidney recipients differed in their peripheral blood expression patterns, but no significant overlap was observed between the two datasets. This was confirmed at the functional level by employing gene set enrichment analysis.The lack of obvious similarities in immune parameters associated with liver and kidney tolerant recipients implies the involvement of different mechanisms in the two settings and argues against the existence of a common immunological constant of spontaneous operational tolerance in clinical transplantation.\nLast Updated (by provider): Apr 12 2012\n\nContributers:  A Pallier J J Lozano M Lopez J P Soulillou A Rimola M C Londoño M Martinez-Llordella R Danger A Sanchez-Fueyo S Brouard\n"
#&gt; 
#&gt; $GSE22707$source
#&gt; NULL
#&gt; 
#&gt; $GSE22707$name
#&gt; [1] "Absence of significant overlap in transcriptional patterns between operationally  tolerant liver and kidney recipients"
#&gt; 
#&gt; $GSE22707$id
#&gt; [1] 5147
#&gt; 
#&gt; $GSE22707$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$lastUpdated
#&gt; [1] 1.519909e+12
#&gt; 
#&gt; $GSE22707$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE22707$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE22707$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE22707$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE22707$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE22707$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE10419
#&gt; $GSE10419$shortName
#&gt; [1] "GSE10419"
#&gt; 
#&gt; $GSE10419$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE10419$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$accession
#&gt; [1] "GSE10419"
#&gt; 
#&gt; $GSE10419$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE10419$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE10419$experimentalDesign
#&gt; [1] 2369
#&gt; 
#&gt; $GSE10419$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE10419$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE10419$bioMaterialCount
#&gt; [1] 30
#&gt; 
#&gt; $GSE10419$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE10419$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE10419$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE10419$geeq
#&gt; $GSE10419$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.04717663
#&gt; 
#&gt; $GSE10419$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE10419$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE10419$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE10419$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE10419$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE10419$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 0
#&gt; 
#&gt; $GSE10419$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE10419$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE10419$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE10419$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE10419$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE10419$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE10419$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE10419$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE10419$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.175833
#&gt; 
#&gt; $GSE10419$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.1662536
#&gt; 
#&gt; $GSE10419$geeq$publicQualityScore
#&gt; [1] 0.3094648
#&gt; 
#&gt; $GSE10419$geeq$publicSuitabilityScore
#&gt; [1] 0.1625
#&gt; 
#&gt; $GSE10419$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE10419$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE10419$geeq$id
#&gt; [1] 1194
#&gt; 
#&gt; 
#&gt; $GSE10419$processedExpressionVectorCount
#&gt; [1] 22153
#&gt; 
#&gt; $GSE10419$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE10419$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE10419$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE10419$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$bioAssayCount
#&gt; [1] 30
#&gt; 
#&gt; $GSE10419$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE10419$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE10419$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$description
#&gt; [1] " Because of world-wide shortage of renal grafts, kidney transplantation (KTx) from donors after cardiac death (DCD) is an alternative way to KTx from brain-dead donors. Although the prognosis of DCD KTx is gradually improving, the graft often undergoes delayed graft function (DGF) rendering the control of DGF essential for post-KTx patient care. To know the etiology of the DGF, we performed genome-wide gene expression profiling using renal biopsy samples performed at 1 hour after KTx from DCD and compared the data with those of KTx from living donors (LD). A total of 526 genes were differentially expressed between them. Genes involved in acute inflammation were activated, while metabolic pathways were consistently down-regulated in DCD. All of these findings imply inferior performance of the DCD grafts relative to LD grafts. We identified several genes of which expression levels were correlated well with parameters indicating short- and long-term prognosis of the DCD patients. In addition, we identified several genes encoding secretory proteins that might reflect the performance of the graft and be potent non-invasive biomarkers. Our data provide good source for candidates of biomarker that are potentially useful for control of DGF.\nLast Updated (by provider): Feb 07 2008\n"
#&gt; 
#&gt; $GSE10419$source
#&gt; NULL
#&gt; 
#&gt; $GSE10419$name
#&gt; [1] "Global expression profiles in one hour biopsy of human kidney transplantation from donors after cardiac death"
#&gt; 
#&gt; $GSE10419$id
#&gt; [1] 2102
#&gt; 
#&gt; $GSE10419$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$lastUpdated
#&gt; [1] 1.519925e+12
#&gt; 
#&gt; $GSE10419$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE10419$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE10419$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE10419$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE10419$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE10419$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE1145
#&gt; $GSE1145$shortName
#&gt; [1] "GSE1145"
#&gt; 
#&gt; $GSE1145$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE1145$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$accession
#&gt; [1] "GSE1145"
#&gt; 
#&gt; $GSE1145$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE1145$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE1145$experimentalDesign
#&gt; [1] 149
#&gt; 
#&gt; $GSE1145$batchConfound
#&gt; [1] "Factor 'diseaseState' may be confounded with batches; p=3.0e-12"
#&gt; 
#&gt; $GSE1145$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC2); p=4.6e-06"
#&gt; 
#&gt; $GSE1145$bioMaterialCount
#&gt; [1] 37
#&gt; 
#&gt; $GSE1145$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE1145$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE1145$geeq
#&gt; $GSE1145$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0002315413
#&gt; 
#&gt; $GSE1145$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE1145$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE1145$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE1145$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE1145$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE1145$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9691787
#&gt; 
#&gt; $GSE1145$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9730902
#&gt; 
#&gt; $GSE1145$geeq$publicQualityScore
#&gt; [1] 0.4247272
#&gt; 
#&gt; $GSE1145$geeq$publicSuitabilityScore
#&gt; [1] 0.4125
#&gt; 
#&gt; $GSE1145$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE1145$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE1145$geeq$id
#&gt; [1] 728
#&gt; 
#&gt; 
#&gt; $GSE1145$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE1145$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE1145$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE1145$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE1145$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$bioAssayCount
#&gt; [1] 37
#&gt; 
#&gt; $GSE1145$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE1145$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE1145$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$description
#&gt; [1] " To establish changes in cardiac transcription profiles brought about by heart failure we collected myocardial samples from patients undergoing cardiac transplantation whose failure arises from different etiologies (e.g. idiopathic dilated cardiomyopathy, ischemic cardiomyopathy, alcoholic cardiomyopathy, valvular cardiomyopathy, and hypertrophic cardiomyopathy) and from \"normal\" organ donors whose hearts cannot be used for transplants. The transcriptional profile of the mRNA in these samples will be measured with gene array technology. Changes in transcriptional profiles can be correlated with the physiologic profile of heart-failure hearts acquired at the time of transplantation..\nDate GSE1145 Last Updated: May 29 2005\n\nContributers:  Paul Allen Seigo Izumo Masako Tsubakihara Jozef Bartunek Jeffrey Brown Lauren E Riggi Martina Schinke Allen Paul\n\nIncludes GDS651.\n Update date: Aug 02 2004.\n Dataset description GDS651: Examination of left ventricle in idiopathic dilated and ischemic cardiomyopathy.  Diseased ventricles examined in reference to normal..\n"
#&gt; 
#&gt; $GSE1145$source
#&gt; NULL
#&gt; 
#&gt; $GSE1145$name
#&gt; [1] "changes in cardiac transcription profiles brought about by heart failure"
#&gt; 
#&gt; $GSE1145$id
#&gt; [1] 149
#&gt; 
#&gt; $GSE1145$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$lastUpdated
#&gt; [1] 1.519893e+12
#&gt; 
#&gt; $GSE1145$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE1145$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE1145$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE1145$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE1145$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1145$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE3075
#&gt; $GSE3075$shortName
#&gt; [1] "GSE3075"
#&gt; 
#&gt; $GSE3075$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE3075$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$accession
#&gt; [1] "GSE3075"
#&gt; 
#&gt; $GSE3075$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE3075$taxon
#&gt; [1] "mouse"
#&gt; 
#&gt; $GSE3075$experimentalDesign
#&gt; [1] 220
#&gt; 
#&gt; $GSE3075$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE3075$batchEffect
#&gt; [1] "This data set may have a batch artifact (PCnull); p=0.0"
#&gt; 
#&gt; $GSE3075$bioMaterialCount
#&gt; [1] 9
#&gt; 
#&gt; $GSE3075$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE3075$taxonId
#&gt; [1] 2
#&gt; 
#&gt; $GSE3075$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE3075$geeq
#&gt; $GSE3075$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 4.050078e-06
#&gt; 
#&gt; $GSE3075$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE3075$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE3075$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE3075$geeq$sScoreSampleSize
#&gt; [1] -1
#&gt; 
#&gt; $GSE3075$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE3075$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9948119
#&gt; 
#&gt; $GSE3075$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9952261
#&gt; 
#&gt; $GSE3075$geeq$publicQualityScore
#&gt; [1] 0.4278894
#&gt; 
#&gt; $GSE3075$geeq$publicSuitabilityScore
#&gt; [1] 0.625
#&gt; 
#&gt; $GSE3075$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE3075$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$geeq$id
#&gt; [1] 901
#&gt; 
#&gt; 
#&gt; $GSE3075$processedExpressionVectorCount
#&gt; [1] 22690
#&gt; 
#&gt; $GSE3075$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE3075$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE3075$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE3075$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$bioAssayCount
#&gt; [1] 9
#&gt; 
#&gt; $GSE3075$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE3075$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE3075$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$description
#&gt; [1] " SMARD1 is an infantile autosomal recessive motor neuron (MN) disease, caused by mutations in the Immunoglobulin mu binding protein 2 (IGHMBP2). We investigated the potential of a spinal cord neural stem cell population isolated on the basis of aldehyde dehydrogenase activity (ALDH) to modify disease progression of nmd mice, an animal model of SMARD1. ALDHhiSSClo stem cells are self-renewing and multipotent and when intratechally transplanted in nmd mice generate motor neurons properly localized in the spinal cord ventral horns. Transplanted nmd animals presented delayed disease progression, sparing of motor neurons and ventral root axons and increased life-span. To further investigate the molecular events responsible for these differences, microarray and Real time RT-PCR analysis of wild-type, mutated and transplanted nmd spinal cord were undertaken. We demonstrated a down-regulation of genes involved in excitatory amino acid toxicity and oxidative stress handling, as well as an up-regulation of genes related to the chromatin organization in nmd compared to wild-type mice, suggesting that they may play a role in SMARD1 pathogenesis. Spinal cord of nmd transplanted mice expressed high transcript levels for genes related to neurogenesis such as Doublecortin (DCX), LIS1 and drebrin. The presence of DCX-expressing cells in adult nmd spinal cord suggests that both exogenous and endogenous neurogenesis may contribute to the observed nmd phenotype amelioration.\n\nLast Updated (by provider): Dec 21 2005\n\nContributers:  Roberto Del Bo Andreina Bordoni Sabrina Salani Sandra Strazzer Stefania Corti Federica Locatelli Dimitra Papadimitriou Giacomo P Comi Nereo Bresolin Marco Crimi Chiara Donadoni Francesco  Fortunato Giorgia Menozzi\n\nIncludes GDS1468.\n Update date: Aug 23 2006.\n Dataset description GDS1468: Analysis of neuromuscular degeneration mutant (nmd) spinal cords transplanted with ALDHhiSSClo neural stem cells. nmd is a model for spinal muscular atrophy with respiratory distress type 1. Results provide insight into the mechanisms underlying neuromuscular improvements in stem cell recipients..\n"
#&gt; 
#&gt; $GSE3075$source
#&gt; NULL
#&gt; 
#&gt; $GSE3075$name
#&gt; [1] "Transplantation of ALDHhiSSClo Neural Stem Cells in nmd Mice, an Animal Model of SMARD1"
#&gt; 
#&gt; $GSE3075$id
#&gt; [1] 244
#&gt; 
#&gt; $GSE3075$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$lastUpdated
#&gt; [1] 1.519865e+12
#&gt; 
#&gt; $GSE3075$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE3075$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE3075$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE3075$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE3075$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3075$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE36059
#&gt; $GSE36059$shortName
#&gt; [1] "GSE36059"
#&gt; 
#&gt; $GSE36059$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE36059$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$accession
#&gt; [1] "GSE36059"
#&gt; 
#&gt; $GSE36059$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE36059$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE36059$experimentalDesign
#&gt; [1] 6850
#&gt; 
#&gt; $GSE36059$batchConfound
#&gt; [1] "Factor 'phenotype' may be confounded with batches; p=1.6e-06"
#&gt; 
#&gt; $GSE36059$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC5); p=2.4e-23"
#&gt; 
#&gt; $GSE36059$bioMaterialCount
#&gt; [1] 411
#&gt; 
#&gt; $GSE36059$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE36059$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE36059$geeq
#&gt; $GSE36059$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.001357295
#&gt; 
#&gt; $GSE36059$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE36059$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE36059$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE36059$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE36059$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9360849
#&gt; 
#&gt; $GSE36059$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9467456
#&gt; 
#&gt; $GSE36059$geeq$publicQualityScore
#&gt; [1] 0.2781065
#&gt; 
#&gt; $GSE36059$geeq$publicSuitabilityScore
#&gt; [1] 0.75
#&gt; 
#&gt; $GSE36059$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE36059$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE36059$geeq$id
#&gt; [1] 6298
#&gt; 
#&gt; 
#&gt; $GSE36059$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE36059$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE36059$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE36059$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE36059$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$bioAssayCount
#&gt; [1] 411
#&gt; 
#&gt; $GSE36059$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE36059$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE36059$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$description
#&gt; [1] " Histologic diagnosis of T cell-mediated rejection in kidney transplant biopsies has limited reproducibility because it is based on non-specific lesions using arbitrary rules that are subject to differing interpretations. We used microarray results from 403 indication biopsies previously given histologic diagnoses to develop a molecular classifier that assigned a molecular T cell-mediated rejection score to each biopsy. Independent assessment of the biopsies by multiple pathologists confirmed considerable disagreement on the presence of TCMR features: 79-88% accuracy and 35-69% sensitivity. The agreement of the molecular T cell-mediated rejection score with the histology diagnosis was similar to agreement among individual pathologists: accuracy 89%, sensitivity 51%. However, the score also predicted the consensus among pathologists, being highest when all agreed. Many discrepancies between the scores and the histologic diagnoses were in situations where histology is unreliable e.g. scarred biopsies. The score correlated with histologic lesions and gene sets associated with T cell-mediated rejection. The transcripts most often selected by the classifier were expressed in effector T cells, dendritic cells, or macrophages or inducible by interferon-gamma. Thus the T cell-mediated rejection score offers an objective assessment of kidney transplant biopsies, predicting the consensus opinion among multiple pathologists, and offering insights into underlying disease mechanisms.  Antibody-mediated rejection is a major cause of kidney transplant failure, but the current diagnostic system misses most cases due to dependency on subjective non-standardized tests. We hypothesized that molecular features could provide a test to address this problem. We classified 403 biopsies by a reference standard based on microcirculation lesions and donor-specific HLA antibody, and used microarray analysis to develop a classifier that assigned each biopsy a score reflecting the probability of antibody-mediated rejection. The scores correlated with donor-specific antibody and histologic lesions: 42/45 biopsies with antibody-mediated rejection scores &gt;0.5 had both donor-specific antibody and microcirculation lesions. Intermediate scores (0.2-0.5) were more ambiguous, but became more specific combined with donor-specific antibody. Compared to diagnoses based on histology-plus-donor-specific antibody, the scores had sensitivity 0.67; specificity 0.90. Donor-specific antibody improved the specificity to 0.97. The score correlated not only with diagnoses of individual pathologists but with the consensus among multiple pathologists. The classifier used transcripts expressed in endothelial cells (e.g. CDH13, DARC, ROBO4) and NK cells (e.g. CX3CR1, FGFBP2), as well as IFNG-inducible transcripts e.g. CXCL11. Thus the molecular phenotype of antibody-mediated rejection provides not only an objective test that predicts microcirculation lesions and donor-specific HLA antibody, but also offers mechanistic insights.\nLast Updated (by provider): Nov 27 2013\n\nContributers:  Philip F Halloran Jeff Reeve Joana Selleres\n"
#&gt; 
#&gt; $GSE36059$source
#&gt; NULL
#&gt; 
#&gt; $GSE36059$name
#&gt; [1] "Molecular diagnosis of T cell-mediated rejection in human kidney transplant biopsies; Molecular diagnosis of antibody-mediated rejection in human kidney transplants"
#&gt; 
#&gt; $GSE36059$id
#&gt; [1] 7660
#&gt; 
#&gt; $GSE36059$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$lastUpdated
#&gt; [1] 1.519918e+12
#&gt; 
#&gt; $GSE36059$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE36059$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE36059$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE36059$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE36059$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36059$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE5104
#&gt; $GSE5104$shortName
#&gt; [1] "GSE5104"
#&gt; 
#&gt; $GSE5104$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE5104$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$accession
#&gt; [1] "GSE5104"
#&gt; 
#&gt; $GSE5104$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE5104$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE5104$experimentalDesign
#&gt; [1] 1018
#&gt; 
#&gt; $GSE5104$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE5104$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE5104$bioMaterialCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE5104$technologyType
#&gt; [1] "TWOCOLOR"
#&gt; 
#&gt; $GSE5104$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE5104$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE5104$geeq
#&gt; $GSE5104$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.002653078
#&gt; 
#&gt; $GSE5104$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE5104$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE5104$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE5104$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE5104$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE5104$geeq$sScoreAvgPlatformPopularity
#&gt; [1] -0.5
#&gt; 
#&gt; $GSE5104$geeq$sScoreAvgPlatformSize
#&gt; [1] -0.5
#&gt; 
#&gt; $GSE5104$geeq$sScoreSampleSize
#&gt; [1] -0.3
#&gt; 
#&gt; $GSE5104$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE5104$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE5104$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE5104$geeq$qScorePlatformsTech
#&gt; [1] -1
#&gt; 
#&gt; $GSE5104$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE5104$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE5104$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.8364882
#&gt; 
#&gt; $GSE5104$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.8468945
#&gt; 
#&gt; $GSE5104$geeq$publicQualityScore
#&gt; [1] -0.1647294
#&gt; 
#&gt; $GSE5104$geeq$publicSuitabilityScore
#&gt; [1] 0.2125
#&gt; 
#&gt; $GSE5104$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE5104$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE5104$geeq$id
#&gt; [1] 2628
#&gt; 
#&gt; 
#&gt; $GSE5104$processedExpressionVectorCount
#&gt; [1] 21575
#&gt; 
#&gt; $GSE5104$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE5104$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE5104$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE5104$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE5104$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE5104$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE5104$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$description
#&gt; [1] " The clinical results obtained with organ allografts from marginal or extended donors are less satisfactory over both the short-term and long-term than the outcome of grafting from living donors. Early postoperative graft function depends on several pretransplant factors, with the major donor influences being brain death (BD) and ischemia/reperfusion(I/R) injury. The effects of BD and I/R injury not only reduce the number of functioning nephrons, but also trigger a host immune response to the grafted kidney. It has been hypothesized that allografts from marginal sources may not be biologically inert at the time of surgery, but may already be programmed to initiate or amplify a host response. To exclude the effects of allogenicity, we established a rat renal isograft model using dead donors and compared the results with those obtained after grafting from living donors. In this study, we performed an analysis of the changes of gene expression in rat kidney isografts using samples obtained at 0(pre-transplant) and 1 hour time points since transplantation procedure. The results enhanced our insight into the pathways and cascades that are activated or down-regulated by BD and/or I/R injury. Better knowledge of the key components of BD or I/R injury will provide clues to the factors triggering progression that leads to a decline of organ viability, as well as ideas on how to overcome such graft failures. Such data may also allow us to identify novel biomarkers as predictors of adverse outcomes.\nLast Updated (by provider): Aug 15 2006\n\nIncludes GDS2263.\n Update date: Aug 15 2006.\n Dataset description GDS2263: Analysis of isografted kidneys from brain dead donors. Brain death in donors triggers nonspecific inflammatory events that increase the kinetics and intensity of the immunologic response in recipients after kidney transplantation. Results provide insight into the mechanisms underlying this response..\n"
#&gt; 
#&gt; $GSE5104$source
#&gt; NULL
#&gt; 
#&gt; $GSE5104$name
#&gt; [1] "Genome-wide expression profile in rat model of renal isografts from brain dead donors"
#&gt; 
#&gt; $GSE5104$id
#&gt; [1] 847
#&gt; 
#&gt; $GSE5104$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$lastUpdated
#&gt; [1] 1.519882e+12
#&gt; 
#&gt; $GSE5104$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE5104$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE5104$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE5104$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE5104$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE5104$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE16695
#&gt; $GSE16695$shortName
#&gt; [1] "GSE16695"
#&gt; 
#&gt; $GSE16695$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE16695$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$accession
#&gt; [1] "GSE16695"
#&gt; 
#&gt; $GSE16695$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE16695$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE16695$experimentalDesign
#&gt; [1] 3133
#&gt; 
#&gt; $GSE16695$batchConfound
#&gt; [1] "Factor  'SamplingTimePoint' may be confounded with batches; p=0.0091"
#&gt; 
#&gt; $GSE16695$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC3); p=0.0026"
#&gt; 
#&gt; $GSE16695$bioMaterialCount
#&gt; [1] 28
#&gt; 
#&gt; $GSE16695$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE16695$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE16695$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE16695$geeq
#&gt; $GSE16695$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 4.708471e-05
#&gt; 
#&gt; $GSE16695$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE16695$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE16695$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE16695$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE16695$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE16695$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.98128
#&gt; 
#&gt; $GSE16695$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9821124
#&gt; 
#&gt; $GSE16695$geeq$publicQualityScore
#&gt; [1] 0.2831589
#&gt; 
#&gt; $GSE16695$geeq$publicSuitabilityScore
#&gt; [1] 0.7875
#&gt; 
#&gt; $GSE16695$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE16695$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE16695$geeq$id
#&gt; [1] 4046
#&gt; 
#&gt; 
#&gt; $GSE16695$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE16695$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE16695$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE16695$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE16695$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$bioAssayCount
#&gt; [1] 28
#&gt; 
#&gt; $GSE16695$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE16695$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE16695$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$description
#&gt; [1] " Transcriptome analyses of organ transplants have until now usually focused on whole tissue samples containing activation profiles from different cell populations. Here, we enriched endothelial cells from rat cardiac allografts and isografts, establishing their activation profile at baseline and on day 2, 3 and 4 after transplantation. Modulated transcripts were assigned to three categories based on their regulation profile in allografts and isografts. Categories A and B contained the majority of transcripts and showed similar regulation in both graft types, appearing to represent responses to surgical trauma. By contrast, category C contained transcripts that were partly allograft-specific and to a large extent associated with interferon-?-responsiveness. Several transcripts were verified by immunohistochemical analysis of graft lesions, among them the matricellular protein periostin which was one of the most highly upregulated transcripts but has not been associated with transplantation previously. In conclusion, the majority of the differentially expressed genes in graft endothelial cells are affected by the transplantation procedure whereas relatively few are associated with allograft rejection.\nLast Updated (by provider): Jul 08 2011\n\nContributers:  Marlene Lundström Bjarte Fosby Pål-Dag Line Clara Hammarström Bjørg Mikalsen Hilde Bjærke Junbai Wang Guttorm Haraldsen Monika Kasprzycka Helge Scott\n"
#&gt; 
#&gt; $GSE16695$source
#&gt; NULL
#&gt; 
#&gt; $GSE16695$name
#&gt; [1] "Gene expression data from endothelial cells and leukocytes enriched from transplanted rat hearts"
#&gt; 
#&gt; $GSE16695$id
#&gt; [1] 2860
#&gt; 
#&gt; $GSE16695$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$lastUpdated
#&gt; [1] 1.519935e+12
#&gt; 
#&gt; $GSE16695$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE16695$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE16695$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE16695$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE16695$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE16695$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE582
#&gt; $GSE582$shortName
#&gt; [1] "GSE582"
#&gt; 
#&gt; $GSE582$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE582$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$accession
#&gt; [1] "GSE582"
#&gt; 
#&gt; $GSE582$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE582$taxon
#&gt; [1] "mouse"
#&gt; 
#&gt; $GSE582$experimentalDesign
#&gt; [1] 108
#&gt; 
#&gt; $GSE582$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE582$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE582$bioMaterialCount
#&gt; [1] 14
#&gt; 
#&gt; $GSE582$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE582$taxonId
#&gt; [1] 2
#&gt; 
#&gt; $GSE582$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE582$geeq
#&gt; $GSE582$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.00182531
#&gt; 
#&gt; $GSE582$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE582$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE582$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE582$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE582$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE582$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE582$geeq$sScoreAvgPlatformSize
#&gt; [1] -0.5
#&gt; 
#&gt; $GSE582$geeq$sScoreSampleSize
#&gt; [1] -0.3
#&gt; 
#&gt; $GSE582$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE582$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE582$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE582$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE582$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE582$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE582$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9242569
#&gt; 
#&gt; $GSE582$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9371073
#&gt; 
#&gt; $GSE582$geeq$publicQualityScore
#&gt; [1] 0.1338725
#&gt; 
#&gt; $GSE582$geeq$publicSuitabilityScore
#&gt; [1] 0.4
#&gt; 
#&gt; $GSE582$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE582$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE582$geeq$id
#&gt; [1] 649
#&gt; 
#&gt; 
#&gt; $GSE582$processedExpressionVectorCount
#&gt; [1] 12488
#&gt; 
#&gt; $GSE582$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE582$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE582$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE582$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$bioAssayCount
#&gt; [1] 14
#&gt; 
#&gt; $GSE582$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE582$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE582$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$description
#&gt; [1] " C57/BL6 to C57/BL6 Balb/c to C57/BL6 Balb/c to C57/BL6 with anti CD80/86 mAb  Identificaton of gene expression profile in tolerizing murine cardiac allograft by co-stimulatory blockade.  The induction of specific tolerance would be the ultimate achievement in transplant immunology, but the precise mechanisms of immunological tolerance remain largely unknown. Here, we investigated global gene expression analysis in tolerizing murine cardiac allografts by means of oligonucleotide microarrays. Tolerance induction was achieved in cardiac allografts from BALB/c to C57BL/6 mice by daily intraperitoneal injection of anti-CD80 and CD86 mAbs. Comparative analysis revealed 64 genes to be induced more extensively in the tolerizing than in the syngeneic isografts, and 16 genes than in the rejecting allografts. Two genes were specifically upregulated in the tolerizing allografts. In the tolerizing allografts there were induced marked expressions of a number of genes for proinflammatory factors, including interferon-gamma inducible cytokines and chemokines, as well as apoptosis-related genes which were also upregulated in the rejecting allografts. Moreover, these gene expression patterns continued to be upregulated more than 70 days post transplant. These results provide evidence that immunological tolerance can be induced and maintained in the presence of prominent proinflammatory gene expression in vivo..\nDate GSE582 Last Updated: Jun 15 2005\n\nContributers:  M Sata K Naruse T Kodama T Hamakubo S Ihara H Aburatani T Kohro A Saiura H Yagita Y Sugawara C Mataki T Yamaguchi A Izumi T Sakihama M Makuuchi Y Matsui T Minami\n\nIncludes GDS437.\n Update date: Nov 26 2003.\n Dataset description GDS437: Examination of immunologic tolerance induction achieved in cardiac allografts from BALB/c to C57BL/6 mice by daily intraperitoneal injection of anti-CD80 and anti-CD86 monoclonal antibodies (mAbs)..\n"
#&gt; 
#&gt; $GSE582$source
#&gt; NULL
#&gt; 
#&gt; $GSE582$name
#&gt; [1] "Gene expression of transplanted hearts"
#&gt; 
#&gt; $GSE582$id
#&gt; [1] 108
#&gt; 
#&gt; $GSE582$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$lastUpdated
#&gt; [1] 1.519862e+12
#&gt; 
#&gt; $GSE582$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE582$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE582$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE582$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE582$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE582$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE18995
#&gt; $GSE18995$shortName
#&gt; [1] "GSE18995"
#&gt; 
#&gt; $GSE18995$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE18995$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$accession
#&gt; [1] "GSE18995"
#&gt; 
#&gt; $GSE18995$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE18995$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE18995$experimentalDesign
#&gt; [1] 2521
#&gt; 
#&gt; $GSE18995$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE18995$batchEffect
#&gt; [1] "This data set may have a batch artifact (PCnull); p=0.0"
#&gt; 
#&gt; $GSE18995$bioMaterialCount
#&gt; [1] 35
#&gt; 
#&gt; $GSE18995$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE18995$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE18995$geeq
#&gt; $GSE18995$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 8.966796e-05
#&gt; 
#&gt; $GSE18995$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE18995$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE18995$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE18995$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE18995$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE18995$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9691723
#&gt; 
#&gt; $GSE18995$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9703223
#&gt; 
#&gt; $GSE18995$geeq$publicQualityScore
#&gt; [1] 0.5671889
#&gt; 
#&gt; $GSE18995$geeq$publicSuitabilityScore
#&gt; [1] 0.6625
#&gt; 
#&gt; $GSE18995$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE18995$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$geeq$id
#&gt; [1] 1909
#&gt; 
#&gt; 
#&gt; $GSE18995$processedExpressionVectorCount
#&gt; [1] 54675
#&gt; 
#&gt; $GSE18995$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE18995$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE18995$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE18995$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$bioAssayCount
#&gt; [1] 35
#&gt; 
#&gt; $GSE18995$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE18995$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE18995$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$description
#&gt; [1] " Lung donation after cardiac death (DCD), in contrast to donation after brain death (DBD), is a promising and increasingly common method to help relieve the shortage of donor organs. However, the pathogenetic consequences of retrieved lungs after DCD vs. DBD have not been clarified. We aimed to study the differential gene expression profiles in lungs of DCD and DBD patients.\nLast Updated (by provider): Dec 15 2010\n\nContributers:  Marc de Perrot Sina A Gharib Jonathan Yeung Mingyao Liu Tom W Waddell Marcelo Cypel Shaf Keshavjee Masaki Anraku Chang H Kang Masaaki Sato Andrew F Pierre\n"
#&gt; 
#&gt; $GSE18995$source
#&gt; NULL
#&gt; 
#&gt; $GSE18995$name
#&gt; [1] "Expression data from donor lungs of cardiac death and brain death donors"
#&gt; 
#&gt; $GSE18995$id
#&gt; [1] 2253
#&gt; 
#&gt; $GSE18995$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$lastUpdated
#&gt; [1] 1.519931e+12
#&gt; 
#&gt; $GSE18995$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE18995$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE18995$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE18995$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE18995$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE18995$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE65564
#&gt; $GSE65564$shortName
#&gt; [1] "GSE65564"
#&gt; 
#&gt; $GSE65564$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE65564$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$accession
#&gt; [1] "GSE65564"
#&gt; 
#&gt; $GSE65564$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE65564$taxon
#&gt; [1] "mouse"
#&gt; 
#&gt; $GSE65564$experimentalDesign
#&gt; [1] 9000
#&gt; 
#&gt; $GSE65564$batchConfound
#&gt; [1] "Factor 'treatment' may be confounded with batches; p=0.0038, Factor 'strain' may be confounded with batches; p=0.00082"
#&gt; 
#&gt; $GSE65564$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC1); p=6.2e-05"
#&gt; 
#&gt; $GSE65564$bioMaterialCount
#&gt; [1] 28
#&gt; 
#&gt; $GSE65564$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE65564$taxonId
#&gt; [1] 2
#&gt; 
#&gt; $GSE65564$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE65564$geeq
#&gt; $GSE65564$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.001155063
#&gt; 
#&gt; $GSE65564$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE65564$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE65564$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE65564$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE65564$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE65564$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.877547
#&gt; 
#&gt; $GSE65564$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.8795048
#&gt; 
#&gt; $GSE65564$geeq$publicQualityScore
#&gt; [1] 0.2685007
#&gt; 
#&gt; $GSE65564$geeq$publicSuitabilityScore
#&gt; [1] 0.6625
#&gt; 
#&gt; $GSE65564$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE65564$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE65564$geeq$id
#&gt; [1] 6770
#&gt; 
#&gt; 
#&gt; $GSE65564$processedExpressionVectorCount
#&gt; [1] 35512
#&gt; 
#&gt; $GSE65564$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE65564$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE65564$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE65564$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$bioAssayCount
#&gt; [1] 28
#&gt; 
#&gt; $GSE65564$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE65564$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE65564$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$description
#&gt; [1] " Renal dendritic cells play key roles in renal homeostasis and during kidney allograft rejection. Microarray analysis aims to evaluate whether dendritic cells modulate their gene expression profile in relation to their distribution in the different renal compartments (with varying biophysical characteristics), under homeostatic conditions and during acute renal allograft rejection (3 days post-transplantation).\nLast Updated (by provider): Dec 02 2016\n\nContributers:  Thomas Hielscher Federica Chessa\n"
#&gt; 
#&gt; $GSE65564$source
#&gt; NULL
#&gt; 
#&gt; $GSE65564$name
#&gt; [1] "Expression data from renal dendritic cells under homeostatic conditions and from donor and host dendritic cells from renal allografts (3 days post-transplantation), isolated from cortex and medulla."
#&gt; 
#&gt; $GSE65564$id
#&gt; [1] 10094
#&gt; 
#&gt; $GSE65564$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$lastUpdated
#&gt; [1] 1.519887e+12
#&gt; 
#&gt; $GSE65564$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE65564$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE65564$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE65564$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE65564$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE65564$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE36539
#&gt; $GSE36539$shortName
#&gt; [1] "GSE36539"
#&gt; 
#&gt; $GSE36539$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE36539$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$accession
#&gt; [1] "GSE36539"
#&gt; 
#&gt; $GSE36539$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE36539$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE36539$experimentalDesign
#&gt; [1] 5347
#&gt; 
#&gt; $GSE36539$batchConfound
#&gt; [1] "Factor  'organism part' may be confounded with batches; p=1.5e-08"
#&gt; 
#&gt; $GSE36539$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC1); p=2.3e-07"
#&gt; 
#&gt; $GSE36539$bioMaterialCount
#&gt; [1] 32
#&gt; 
#&gt; $GSE36539$technologyType
#&gt; [1] "DUALMODE"
#&gt; 
#&gt; $GSE36539$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE36539$geeq
#&gt; $GSE36539$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.001676599
#&gt; 
#&gt; $GSE36539$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE36539$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE36539$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE36539$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE36539$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$geeq$qScoreOutliers
#&gt; [1] -1
#&gt; 
#&gt; $GSE36539$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE36539$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9427566
#&gt; 
#&gt; $GSE36539$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9542505
#&gt; 
#&gt; $GSE36539$geeq$publicQualityScore
#&gt; [1] -0.006535649
#&gt; 
#&gt; $GSE36539$geeq$publicSuitabilityScore
#&gt; [1] 0.6625
#&gt; 
#&gt; $GSE36539$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE36539$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE36539$geeq$id
#&gt; [1] 925
#&gt; 
#&gt; 
#&gt; $GSE36539$processedExpressionVectorCount
#&gt; [1] 30727
#&gt; 
#&gt; $GSE36539$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE36539$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE36539$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE36539$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$bioAssayCount
#&gt; [1] 32
#&gt; 
#&gt; $GSE36539$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE36539$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE36539$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$description
#&gt; [1] " Background: Hepatitis E Virus (HEV) is a new causative agent of chronic hepatitis in solid organ transplant recipients in Europe. Factors associated with the occurrence and persistence of chronic HEV infection remain largely unknown but chronic evolution seems to be the consequence of host’s immunological factors rather than of viral factors. Method: In a prospective case-control study, we have determined in whole blood of chronically HEV-infected kidney-transplant recipients the host response using microarray technology. Results: Chronically HEV-infected kidney-transplant recipients exhibited a specific transcriptional program, in which interferon effectors were prominent. The intensity of expression of each signature’s gene was significantly lower in patients who were subsequently cleared of HEV than in patients who were not. Furthermore, in patients who were cleared of HEV, a higher expression of these genes was associated with a longer delay until HEV clearance. Conclusions: The specific transcriptional program determined in chronically HEV-infected kidney-transplant recipients suggests an activation of type I interferon response. Intensity of interferon-stimulated genes expression could be useful to forecast the outcome of infection. High expression of interferon-stimulated genes could signify a dysregulation in the interferon response that might favour the persistence of the HEV infection. Trial’s registration number: NCT01090232; Registry’s URL: http://clinicaltrials.gov/ct2/show/study/NCT01090232?term=kidney+transplant+recipients&amp;cntry1=EU%3AFR&amp;rank=2\nLast Updated (by provider): Jan 31 2013\n\nContributers:  Amira Ben Amara Jean-Louis Mege Philippe Colson Vikram Mehraj Julien Textoris Valérie Moal Yvon Berland\n"
#&gt; 
#&gt; $GSE36539$source
#&gt; NULL
#&gt; 
#&gt; $GSE36539$name
#&gt; [1] "Interferon-related transcriptional program is induced by chronic hepatitis E infection in kidney-transplant recipients"
#&gt; 
#&gt; $GSE36539$id
#&gt; [1] 6027
#&gt; 
#&gt; $GSE36539$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$lastUpdated
#&gt; [1] 1.519861e+12
#&gt; 
#&gt; $GSE36539$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE36539$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE36539$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE36539$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE36539$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE36539$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE30886
#&gt; $GSE30886$shortName
#&gt; [1] "GSE30886"
#&gt; 
#&gt; $GSE30886$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE30886$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$accession
#&gt; [1] "GSE30886"
#&gt; 
#&gt; $GSE30886$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE30886$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE30886$experimentalDesign
#&gt; [1] 4672
#&gt; 
#&gt; $GSE30886$batchConfound
#&gt; [1] "Factor  'DiseaseState' may be confounded with batches; p=2.4e-10"
#&gt; 
#&gt; $GSE30886$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC1); p=2.0e-07"
#&gt; 
#&gt; $GSE30886$bioMaterialCount
#&gt; [1] 41
#&gt; 
#&gt; $GSE30886$technologyType
#&gt; [1] "DUALMODE"
#&gt; 
#&gt; $GSE30886$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE30886$geeq
#&gt; $GSE30886$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.01647014
#&gt; 
#&gt; $GSE30886$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE30886$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE30886$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE30886$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE30886$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE30886$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.6944841
#&gt; 
#&gt; $GSE30886$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.7052238
#&gt; 
#&gt; $GSE30886$geeq$publicQualityScore
#&gt; [1] 0.2436034
#&gt; 
#&gt; $GSE30886$geeq$publicSuitabilityScore
#&gt; [1] 0.6625
#&gt; 
#&gt; $GSE30886$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE30886$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE30886$geeq$id
#&gt; [1] 3768
#&gt; 
#&gt; 
#&gt; $GSE30886$processedExpressionVectorCount
#&gt; [1] 41000
#&gt; 
#&gt; $GSE30886$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE30886$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE30886$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE30886$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$bioAssayCount
#&gt; [1] 41
#&gt; 
#&gt; $GSE30886$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE30886$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE30886$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$description
#&gt; [1] " The ability to identify a robust means for both pediatric and adult liver transplant recipients for minimization and weaning of immunosuppression is greatly needed.   We analyzed 298 samples from pediatric (n=83) and published adult (n=57) liver transplant recipients, and the public available cell or tissue specific samples (n=158). Biomarkers were first identified from the integration of the cross-platform analysis of Stanford pediatric and adult studies and validated in independent samples from UCLA on microarray. Furthermore, Q-PCR was performed for targets validation and delivers a robust diagnostic assay of predicting potential liver tolerance after liver transplantation.   Thirteen unique genes were identified (FDR&lt;5%) using nearest shrunken centroid classification methods as a minimum gene set to cross-validate and predict Stanford pediatric tolerance samples with 1misclassification, 5 misclassification for adult samples, and 100% sensitivity and 83% specificity for an independent UCLA pediatric samples in microarray platform. These subset of tolerant specific genes are highly expressed CD56+ natural killer cells (p=0.03). Furthermore, the subset can be narrowed down to 3 key genes with combined ROC=0.988 for the liver tolerance prediction from Q-PCR verification and 64% MIS and STA were predicted as tolerance with the &gt;90% prediction probability.   Specific peripheral transcriptional programs can be identified in operational tolerance in pediatric recipients of liver allografts, distinct from those previously identified in adult operationally tolerant liver recipients, and may provide a means to non-invasively monitor patients in a serial manner for immunosuppression minimization. These genes are highly expressed in specific peripheral blood lymphocyte subsets, and their coordinated may support the maintenance of operational tolerance in children, following liver transplantation.\nLast Updated (by provider): May 02 2012\n\nContributers:  Sue V McDiarmid Anita Talisetti Laura Wozniak Qian Wang Ken Cox Minnie Sarwal Carlos Esquivel Rong Chen Silke Rodder Li Li Sue Heish\n"
#&gt; 
#&gt; $GSE30886$source
#&gt; NULL
#&gt; 
#&gt; $GSE30886$name
#&gt; [1] "A Common Peripheral Blood Gene-Set for Diagnosis and Prediction of Operational Tolerance in Pediatric and Adult Liver Transplantation"
#&gt; 
#&gt; $GSE30886$id
#&gt; [1] 5317
#&gt; 
#&gt; $GSE30886$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$lastUpdated
#&gt; [1] 1.519915e+12
#&gt; 
#&gt; $GSE30886$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE30886$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE30886$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE30886$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE30886$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE30886$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE38262
#&gt; $GSE38262$shortName
#&gt; [1] "GSE38262"
#&gt; 
#&gt; $GSE38262$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE38262$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$accession
#&gt; [1] "GSE38262"
#&gt; 
#&gt; $GSE38262$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE38262$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE38262$experimentalDesign
#&gt; [1] 6968
#&gt; 
#&gt; $GSE38262$batchConfound
#&gt; [1] "Factor 'disease2' may be confounded with batches; p=9.7e-06, Factor 'disease' may be confounded with batches; p=3.5e-07, Factor 'treatment' may be confounded with batches; p=2.2e-05"
#&gt; 
#&gt; $GSE38262$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC3); p=1.1e-12"
#&gt; 
#&gt; $GSE38262$bioMaterialCount
#&gt; [1] 92
#&gt; 
#&gt; $GSE38262$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE38262$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE38262$geeq
#&gt; $GSE38262$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.001468914
#&gt; 
#&gt; $GSE38262$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE38262$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE38262$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$qScoreOutliers
#&gt; [1] -1
#&gt; 
#&gt; $GSE38262$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE38262$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9415963
#&gt; 
#&gt; $GSE38262$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9546333
#&gt; 
#&gt; $GSE38262$geeq$publicQualityScore
#&gt; [1] -0.006480964
#&gt; 
#&gt; $GSE38262$geeq$publicSuitabilityScore
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE38262$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE38262$geeq$id
#&gt; [1] 6581
#&gt; 
#&gt; 
#&gt; $GSE38262$processedExpressionVectorCount
#&gt; [1] 33252
#&gt; 
#&gt; $GSE38262$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE38262$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE38262$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE38262$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$bioAssayCount
#&gt; [1] 92
#&gt; 
#&gt; $GSE38262$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE38262$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE38262$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$description
#&gt; [1] " We investigated the clinical and molecular significance of minimal peritubular capillary (PTC) and isolated glomerular C4d+ staining using microarrays. Immunohistochemistry for C4d was performed on paraffin-embedded sections. Of the 255 biopsies analyzed, 51% were C4d negative, 4% were minimal, 15% focal or diffuse PTC C4d+, and 31% isolated glomerular C4d+. Minimal and focal/ diffuse PTC C4d+ staining were associated with a higher frequency of donor-specific anti-HLA antibodies (DSA) (67% vs. 82% vs. 25%), antibody mediated rejection (AMR) (66% vs. 89% vs. 19%) and mean glomerulitis (0.88 vs. 0.65 vs. 0.25, p=0.003), interstitial inflammation (1.25 vs. 1.41 vs. 0.79; p=0.003) and peritubular capillaritis scores (1.5 vs. 1.5 vs. 0.34; p &lt; 0.001), compared to the C4d negative group, respectively. There were no differences in the DSA frequency, AMR rate, and Banff scores between isolated glomerular C4d+ and negative patients. While both minimal and focal/diffuse C4d+ biopsies showed increased expression of genes related to the immune response, and interferon-gamma and rejection induced, cytotoxic T cell and constitutive macrophage-associated pathogenesis based transcripts, there was no activation of immune-response related genes in isolated glomerular C4d+ biopsies. In summary, minimal PTC C4d+ staining but not isolated glomerular C4d+ staining is associated with AMR.\nLast Updated (by provider): Sep 17 2013\n\nContributers:  Monica Chung Min Ling Nicole Hayde Enver Akalin Graciela de Boccardo Ahmed Alansari Bin Ye James Pullman Daniel Schwartz Yi Bao Brent Calder Daniel Glicklich\n"
#&gt; 
#&gt; $GSE38262$source
#&gt; NULL
#&gt; 
#&gt; $GSE38262$name
#&gt; [1] "The clinical and molecular significance of different types of C4d staining in renal allografts"
#&gt; 
#&gt; $GSE38262$id
#&gt; [1] 7782
#&gt; 
#&gt; $GSE38262$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$lastUpdated
#&gt; [1] 1.519925e+12
#&gt; 
#&gt; $GSE38262$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE38262$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE38262$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE38262$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE38262$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE38262$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE1743
#&gt; $GSE1743$shortName
#&gt; [1] "GSE1743"
#&gt; 
#&gt; $GSE1743$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE1743$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$accession
#&gt; [1] "GSE1743"
#&gt; 
#&gt; $GSE1743$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE1743$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE1743$experimentalDesign
#&gt; [1] 4454
#&gt; 
#&gt; $GSE1743$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE1743$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE1743$bioMaterialCount
#&gt; [1] 41
#&gt; 
#&gt; $GSE1743$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE1743$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE1743$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE1743$geeq
#&gt; $GSE1743$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0009237941
#&gt; 
#&gt; $GSE1743$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE1743$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE1743$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE1743$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE1743$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE1743$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE1743$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE1743$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE1743$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE1743$geeq$sScoreMissingValues
#&gt; [1] -1
#&gt; 
#&gt; $GSE1743$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE1743$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE1743$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE1743$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE1743$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9436023
#&gt; 
#&gt; $GSE1743$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9489213
#&gt; 
#&gt; $GSE1743$geeq$publicQualityScore
#&gt; [1] 0.4212745
#&gt; 
#&gt; $GSE1743$geeq$publicSuitabilityScore
#&gt; [1] 0.2875
#&gt; 
#&gt; $GSE1743$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE1743$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE1743$geeq$id
#&gt; [1] 2870
#&gt; 
#&gt; 
#&gt; $GSE1743$processedExpressionVectorCount
#&gt; [1] 22283
#&gt; 
#&gt; $GSE1743$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE1743$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE1743$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE1743$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$bioAssayCount
#&gt; [1] 41
#&gt; 
#&gt; $GSE1743$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE1743$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE1743$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$description
#&gt; [1] " Background: Despite significant improvements in short-term kidney transplant survival, comparable increases in 5 and 10-year outcomes have not been achieved. Chronic allograft nephropathy (CAN) is a major cause of late graft loss. Toxic nephropathy and inadequate long-term immunosuppression are possible factors. We performed a randomized prospective trial comparing calcineurin inhibitor (CNI)-free to CNI-based immunosuppression to determine the impact on renal function, structure, and gene expression.  Methods: Sixty-one kidney recipients received mycophenolate mofetil (MMF), and prednisone (P). Randomized patients received concentration-controlled sirolimus or cyclosporine. Two years post-transplant 55 patients underwent renal function studies, 48 (87%) underwent transplant biopsies; all classified by Banff scoring and 41 by DNA microarrays.  Findings:  Comparing sirolimus/MMF/P to cyclosporine/MMF/P at two years, there was a significantly lower serum creatinine (1.35 vs. 1.81 mg/dl; p=0.008), significantly higher Cockroft-Gault glomerular filtration rate (GFR) (80.4 vs. 63.4 cc/min; p=0.008), iothalamate GFR (60.6 vs. 49.2 cc/min; p= 0.018), and Banff 0 (normal) biopsies (66.6 vs. 20.8%; p=0.013).  Regression analysis of calculated GFR’s from 1 to 36 months yielded a positive slope for sirolimus of 3.36 ml/min/year, and a negative slope for cyclosporine of –1.58 ml/min/year (p=0.008). Gene expression profiles of kidney biopsies with higher Banff CAN scores confirmed significant up regulation of genes responsible for immune/inflammation and fibrosis/tissue remodeling.  Interpretation:  At two years the sirolimus-treated patients have better renal transplant function, a diminished prevalence of CAN, and down regulated expression of genes responsible for the progression of CAN. All may provide for an alternative natural history with improved graft survival.\nLast Updated (by provider): May 16 2012\n\nContributers:  Jennifer A Hammond Caroline M Lanigan Hank Rollin Daniel J Cook Steven R Head Sunil M Kurian Daniel R Salomon Thomas C Whisenant Stuart M Flechner James T Burke Kim Solez\n\nIncludes GDS987.\n Update date: Apr 18 2005.\n Dataset description GDS987: Analysis of kidneys from adult renal transplant recipients subjected to calcineurin inhibitor-free immunosuppression using sirolimus. Patients treated with sirolimus have a lower prevalence of chronic allograft nephropathy compared to those treated with cyclosporine, a calcineurin inhibitor..\n"
#&gt; 
#&gt; $GSE1743$source
#&gt; NULL
#&gt; 
#&gt; $GSE1743$name
#&gt; [1] "De novo kidney transplantation without use of calcineurin inhibitors preserves renal structure and function"
#&gt; 
#&gt; $GSE1743$id
#&gt; [1] 4902
#&gt; 
#&gt; $GSE1743$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$lastUpdated
#&gt; [1] 1.519907e+12
#&gt; 
#&gt; $GSE1743$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE1743$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE1743$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE1743$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE1743$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE1743$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE25098
#&gt; $GSE25098$shortName
#&gt; [1] "GSE25098"
#&gt; 
#&gt; $GSE25098$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE25098$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$accession
#&gt; [1] "GSE25098"
#&gt; 
#&gt; $GSE25098$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE25098$taxon
#&gt; [1] "mouse"
#&gt; 
#&gt; $GSE25098$experimentalDesign
#&gt; [1] 3497
#&gt; 
#&gt; $GSE25098$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE25098$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE25098$bioMaterialCount
#&gt; [1] 28
#&gt; 
#&gt; $GSE25098$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE25098$taxonId
#&gt; [1] 2
#&gt; 
#&gt; $GSE25098$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE25098$geeq
#&gt; $GSE25098$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0008439206
#&gt; 
#&gt; $GSE25098$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE25098$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE25098$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE25098$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE25098$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE25098$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE25098$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE25098$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE25098$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE25098$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE25098$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE25098$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE25098$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE25098$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE25098$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9389228
#&gt; 
#&gt; $GSE25098$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9461532
#&gt; 
#&gt; $GSE25098$geeq$publicQualityScore
#&gt; [1] 0.2780219
#&gt; 
#&gt; $GSE25098$geeq$publicSuitabilityScore
#&gt; [1] 0.225
#&gt; 
#&gt; $GSE25098$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE25098$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE25098$geeq$id
#&gt; [1] 5649
#&gt; 
#&gt; 
#&gt; $GSE25098$processedExpressionVectorCount
#&gt; [1] 22690
#&gt; 
#&gt; $GSE25098$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE25098$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE25098$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE25098$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$bioAssayCount
#&gt; [1] 28
#&gt; 
#&gt; $GSE25098$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE25098$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE25098$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$description
#&gt; [1] " Microarray analysis of rhabdomyosarcomas generated using mice with conditional mutations in Kras and p53 using both in vivo and in vitro approaches to identify a cell of origin for rhabdomyosarcoma.\nLast Updated (by provider): Jan 31 2011\n\nContributers:  Jordan M Blum David G Kirsch\n"
#&gt; 
#&gt; $GSE25098$source
#&gt; NULL
#&gt; 
#&gt; $GSE25098$name
#&gt; [1] "Rhabdomyosarcoma can be initiated in activated Pax7+ Muscle Progenitor Cells"
#&gt; 
#&gt; $GSE25098$id
#&gt; [1] 3234
#&gt; 
#&gt; $GSE25098$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$lastUpdated
#&gt; [1] 1.519871e+12
#&gt; 
#&gt; $GSE25098$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE25098$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE25098$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE25098$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE25098$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE25098$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; 
#&gt; $GSE2018
#&gt; $GSE2018$shortName
#&gt; [1] "GSE2018"
#&gt; 
#&gt; $GSE2018$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2018$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$accession
#&gt; [1] "GSE2018"
#&gt; 
#&gt; $GSE2018$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2018$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE2018$experimentalDesign
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2018$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC2); p=0.0071"
#&gt; 
#&gt; $GSE2018$bioMaterialCount
#&gt; [1] 34
#&gt; 
#&gt; $GSE2018$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2018$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2018$geeq
#&gt; $GSE2018$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0004038271
#&gt; 
#&gt; $GSE2018$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2018$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2018$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2018$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE2018$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$geeq$qScoreOutliers
#&gt; [1] -1
#&gt; 
#&gt; $GSE2018$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2018$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9557703
#&gt; 
#&gt; $GSE2018$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9598845
#&gt; 
#&gt; $GSE2018$geeq$publicQualityScore
#&gt; [1] 0.4228406
#&gt; 
#&gt; $GSE2018$geeq$publicSuitabilityScore
#&gt; [1] 0.7875
#&gt; 
#&gt; $GSE2018$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE2018$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$geeq$id
#&gt; [1] 557
#&gt; 
#&gt; 
#&gt; $GSE2018$processedExpressionVectorCount
#&gt; [1] 22283
#&gt; 
#&gt; $GSE2018$parentTaxonId
#&gt; NULL
#&gt; 
#&gt; $GSE2018$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2018$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2018$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$bioAssayCount
#&gt; [1] 34
#&gt; 
#&gt; $GSE2018$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2018$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$description
#&gt; [1] " Bronchoalveolar lavage samples collected from lung transplant recipients.  Numeric portion of sample name is an arbitrary patient ID and AxBx number indicates the perivascular (A) and bronchiolar (B) scores from biopsies collected on the same day as the BAL fluid was collected.  Several patients have more than one sample in this series and can be determined by patient number followed by a lower case letter.  Acute rejection state is determined by the combined A and B score - specifically, a combined AB score of 2 or greater is considered an acute rejection.."
#&gt; 
#&gt; $GSE2018$source
#&gt; [1] ""
#&gt; 
#&gt; $GSE2018$name
#&gt; [1] "Human Lung Transplant - BAL"
#&gt; 
#&gt; $GSE2018$id
#&gt; [1] 1
#&gt; 
#&gt; $GSE2018$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$lastUpdated
#&gt; [1] 1.51986e+12
#&gt; 
#&gt; $GSE2018$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2018$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2018$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2018$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2018$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2018$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; </div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
    <h2>Contents</h2>
    <ul class="nav nav-pills nav-stacked">
      <li><a href="#arguments">Arguments</a></li>
      
      <li><a href="#value">Value</a></li>
      
      <li><a href="#examples">Examples</a></li>
    </ul>

  </div>
</div>

      <footer>
      <div class="copyright">
  <p>Developed by First Last.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="http://pkgdown.r-lib.org/">pkgdown</a>.</p>
</div>

      </footer>
   </div>

  </body>
</html>

