<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>datasetInfo — datasetInfo • gemmaAPI</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="datasetInfo — datasetInfo" />
<meta property="og:description" content="Retrieves information about a single dataset based on the given dataset 
identifier. Combines several API calls" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">gemmaAPI</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">2.0.3.9000</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>datasetInfo</h1>
    
    <div class="hidden name"><code>datasetInfo.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Retrieves information about a single dataset based on the given dataset 
identifier. Combines several API calls</p>
    </div>

    <pre class="usage"><span class='fu'>datasetInfo</span>(
  <span class='no'>dataset</span>,
  <span class='kw'>request</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='no'>...</span>,
  <span class='kw'>file</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>return</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>,
  <span class='kw'>overwrite</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>,
  <span class='kw'>memoised</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>
)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>dataset</th>
      <td><p>Character. Can either be the dataset ID or its short name 
(e.g. GSE1234). If a vector of length&gt;1 is provided return all matching dataset
 objects similar to <code><a href='allDatasets.html'>allDatasets</a></code> but without access to additional 
 parameters. <code>request</code> parameter cannot be specified for inputs of length&gt;1
 unless specifiet otherwise in request description.</p></td>
    </tr>
    <tr>
      <th>request</th>
      <td><p>Character. If NULL retrieves the dataset object. Otherwise</p><ul>
<li><p><code>platforms</code>: Retrieves platforms for the given dataset</p></li>
<li><p><code>samples</code>: Retrieves samples for the given dataset</p></li>
<li><p><code>annotations</code>: Retrieves the annotations for the given dataset</p></li>
<li><p><code>design</code>: Retrieves the design for the given dataset</p></li>
<li><p><code>data</code>: Retrieves the data for the given dataset. Parameters:</p><ul>
<li><p><code>filter</code>: Optional, defaults to FALSE. If TRUE, call
              returns filtered expression data.</p></li>
<li><p><code>IdColnames</code>: Optional. defaults to FALSE.
               If true shortens data column names to only include the bioAssayId
               which is unique to samples in Gemma. Makes it easier to match to
               samples acqured from the "samples" request.</p></li>
</ul></li>
<li><p><code>differential</code>: Retrieves available differential expression
     tests for the given dataset.</p><ul>
<li><p><code>offset</code>: Optional, defaults to 0. Skips the 
             specified amount of objects when retrieving them from the
              database.</p></li>
<li><p><code>limit</code>: Optional, defaults to 20. Limits the result 
             to specified amount of objects. Use 0 for no limit.</p></li>
</ul></li>
<li><p><code>degs</code>: Retrieves the differential expression results for the given dataset.</p><ul>
<li><p><code>differential</code>: Differential id of the differential expression. Can be acquired
             from the <code>differential</code> endpoint.</p></li>
</ul></li>
<li><p><code>diffExExpr</code>: Retrieves expression values for differential expression subsets for the given datasets.
     Parameters:</p><ul>
<li><p><code>diffExSet</code>: Result set id of the differential expression. Can be
             acquired from the <code>differential</code> endpoint:</p>             
<p><code>datasetInfo('GSE43364',request = 'differential')$resultSets[[1]]$id</code></p></li>
<li><p><code>keepNonSpecific</code>: Optional, defaults to FALSE.</p>             
<p>If set to false, the response will only include elements that map exclusively to each gene</p>             
<p>If set to true, the response will include all elements that map to each gene, even if they also map to other genes.</p></li>
<li><p><code>threshold</code>: Optional, defaults to 100. 
             The threshold that the differential expression has to meet to be included in the response.</p></li>
<li><p><code>limit</code>: Optional, defaults to 100. Maximum amount of returned gene-probe expression level pairs to include in the response.</p></li>
<li><p><code>consolidate</code>: Optional. Defaults no NULL.</p>             
<p>Whether genes with multiple elements should consolidate the information. If the 'keepNonSpecific' parameter is set to true, then all gene non-specific vectors are excluded from the chosen procedure.</p>             
<p>The options are:</p><ul>
<li><p><code>NULL</code>:list all vectors separately.</p></li>
<li><p><code>"pickmax"</code>: only return the vector that has the highest expression (mean over all its bioAssays).</p></li>
<li><p><code>"pickvar"</code>: only return the vector with highest variance of expression across its bioAssays</p></li>
<li><p><code>"average"</code>: create a new vector that will average the bioAssay values from all vectors</p></li>
</ul></li>
</ul></li>
<li><p><code>geneExpression</code>: Retrieves the expression levels of given genes for 
     given datasets. Can be used with multiple datasets. Parameters:</p><ul>
<li><p><code>genes</code>: Required. A list of identifiers, separated by commas (e.g: 1859, 5728).</p>             
<p>Can either be the NCBI ID (1859), Ensembl ID (ENSG00000157540) or official symbol (DYRK1A) of the gene.</p>             
<p>NCBI ID is the most efficient (and guaranteed to be unique) identifier.</p>             
<p>Official symbol represents a gene homologue for a random taxon, unless used in a specific taxon (see the Taxa Endpoints).</p>             
<p>If the gene taxon does not match the taxon of the given datasets, expression levels for that gene will be missing from the response</p>             
<p>You can combine various identifiers in one query, but any invalid identifier will cause the call to yield an error. Duplicate identifiers of the same gene will result in duplicates in the response.</p></li>
<li><p><code>keepNonSpecific</code>: Optional. Defaults to FALSE.</p>             
<p>If set to false, the response will only include elements that map exclusively to each queried gene</p>             
<p>If set to true, the response will include all elements that map to each queried gene, even if they also map to other genes.</p></li>
<li><p><code>consolidate</code>: Optional. Defaults no NULL.</p>             
<p>Whether genes with multiple elements should consolidate the information. If the 'keepNonSpecific' parameter is set to true, then all gene non-specific vectors are excluded from the chosen procedure.</p>             
<p>The options are:</p><ul>
<li><p><code>NULL</code>:list all vectors separately.</p></li>
<li><p><code>"pickmax"</code>: only return the vector that has the highest expression (mean over all its bioAssays).</p></li>
<li><p><code>"pickvar"</code>: only return the vector with highest variance of expression across its bioAssays</p></li>
<li><p><code>"average"</code>: create a new vector that will average the bioAssay values from all vectors</p></li>
</ul></li>
</ul></li>
</ul></td>
    </tr>
    <tr>
      <th>...</th>
      <td><p>Use if the specified request has additional parameters.</p></td>
    </tr>
    <tr>
      <th>file</th>
      <td><p>Character. File path. If provided, response will be saved to file</p></td>
    </tr>
    <tr>
      <th>return</th>
      <td><p>Logical. If the response should be returned. Set to false when
you only want to save a file</p></td>
    </tr>
    <tr>
      <th>overwrite</th>
      <td><p>Logical. If TRUE, existing files will be overwritten. If FALSE
a warning will be thrown and no action is taken.</p></td>
    </tr>
    <tr>
      <th>memoised</th>
      <td><p>Logical. If TRUE a memoised version of the function will be
used which is faster for repeated requests. Use <code><a href='forgetGemmaMemoised.html'>forgetGemmaMemoised</a></code>
to clear memory.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A data.frame or a list depending on the request</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='fu'>datasetInfo</span>(<span class='st'>'GSE81454'</span>)</div><div class='output co'>#&gt; $GSE81454
#&gt; $GSE81454$shortName
#&gt; [1] "GSE81454"
#&gt; 
#&gt; $GSE81454$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE81454$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE81454$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE81454$processedExpressionVectorCount
#&gt; [1] 49395
#&gt; 
#&gt; $GSE81454$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE81454$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE81454$accession
#&gt; [1] "GSE81454"
#&gt; 
#&gt; $GSE81454$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE81454$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GSE81454$taxonId
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$experimentalDesign
#&gt; [1] 9321
#&gt; 
#&gt; $GSE81454$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE81454$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE81454$geeq
#&gt; $GSE81454$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$geeq$sScoreAvgPlatformPopularity
#&gt; [1] -0.5
#&gt; 
#&gt; $GSE81454$geeq$sScoreAvgPlatformSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE81454$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE81454$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9267995
#&gt; 
#&gt; $GSE81454$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9372003
#&gt; 
#&gt; $GSE81454$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.001940028
#&gt; 
#&gt; $GSE81454$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE81454$geeq$corrMatIssues
#&gt; [1] 2
#&gt; 
#&gt; $GSE81454$geeq$replicatesIssues
#&gt; [1] 2
#&gt; 
#&gt; $GSE81454$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE81454$geeq$publicQualityScore
#&gt; [1] 0.2767429
#&gt; 
#&gt; $GSE81454$geeq$publicSuitabilityScore
#&gt; [1] 0.8125
#&gt; 
#&gt; $GSE81454$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE81454$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE81454$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE81454$geeq$id
#&gt; [1] 7697
#&gt; 
#&gt; 
#&gt; $GSE81454$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE81454$bioAssayCount
#&gt; [1] 31
#&gt; 
#&gt; $GSE81454$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE81454$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE81454$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE81454$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE81454$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE81454$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE81454$description
#&gt; [1] "Immune profiles were performed retrospectively in highly sensitized kidney transplant candidates Our hypothesis was that baseline differences in immune profiles could help identify candidates that respond to desensitization therapy.\nLast Updated (by provider): Apr 20 2017\nContributors:  Julie Yabu"
#&gt; 
#&gt; $GSE81454$source
#&gt; NULL
#&gt; 
#&gt; $GSE81454$name
#&gt; [1] "HLA sensitization study"
#&gt; 
#&gt; $GSE81454$id
#&gt; [1] 10634
#&gt; 
#&gt; $GSE81454$lastUpdated
#&gt; [1] 1.585531e+12
#&gt; 
#&gt; $GSE81454$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE81454$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE81454$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE81454$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE81454$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE81454$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE81454$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE81454$`_totalInQuery`
#&gt; [1] 1
#&gt; 
#&gt; </div><div class='input'><span class='fu'>datasetInfo</span>(<span class='st'>'GSE81454'</span>, <span class='kw'>request</span> <span class='kw'>=</span> <span class='st'>'platforms'</span>)</div><div class='output co'>#&gt; $GPL15207
#&gt; $GPL15207$shortName
#&gt; [1] "GPL15207"
#&gt; 
#&gt; $GPL15207$dateCached
#&gt; NULL
#&gt; 
#&gt; $GPL15207$numProbeSequences
#&gt; NULL
#&gt; 
#&gt; $GPL15207$numProbeAlignments
#&gt; NULL
#&gt; 
#&gt; $GPL15207$numProbesToGenes
#&gt; NULL
#&gt; 
#&gt; $GPL15207$numGenes
#&gt; NULL
#&gt; 
#&gt; $GPL15207$designElementCount
#&gt; NULL
#&gt; 
#&gt; $GPL15207$taxon
#&gt; [1] "human"
#&gt; 
#&gt; $GPL15207$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GPL15207$lastGeneMapping
#&gt; NULL
#&gt; 
#&gt; $GPL15207$lastSequenceAnalysis
#&gt; NULL
#&gt; 
#&gt; $GPL15207$lastSequenceUpdate
#&gt; NULL
#&gt; 
#&gt; $GPL15207$blackListed
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL15207$taxonID
#&gt; NULL
#&gt; 
#&gt; $GPL15207$color
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GPL15207$expressionExperimentCount
#&gt; [1] 20
#&gt; 
#&gt; $GPL15207$hasBlatAssociations
#&gt; NULL
#&gt; 
#&gt; $GPL15207$hasGeneAssociations
#&gt; NULL
#&gt; 
#&gt; $GPL15207$hasSequenceAssociations
#&gt; NULL
#&gt; 
#&gt; $GPL15207$isAffymetrixAltCdf
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL15207$isMerged
#&gt; NULL
#&gt; 
#&gt; $GPL15207$isMergee
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL15207$isSubsumed
#&gt; NULL
#&gt; 
#&gt; $GPL15207$isSubsumer
#&gt; NULL
#&gt; 
#&gt; $GPL15207$lastRepeatMask
#&gt; NULL
#&gt; 
#&gt; $GPL15207$description
#&gt; [1] ""
#&gt; 
#&gt; $GPL15207$name
#&gt; [1] "[PrimeView] Affymetrix Human Gene Expression Array"
#&gt; 
#&gt; $GPL15207$lastUpdated
#&gt; [1] 1.575781e+12
#&gt; 
#&gt; $GPL15207$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL15207$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GPL15207$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL15207$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GPL15207$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GPL15207$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GPL15207$curationNote
#&gt; NULL
#&gt; 
#&gt; $GPL15207$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GPL15207$id
#&gt; [1] 783
#&gt; 
#&gt; </div><div class='input'><span class='fu'>datasetInfo</span>(<span class='st'>'GSE81454'</span>, <span class='kw'>request</span><span class='kw'>=</span><span class='st'>'data'</span>,<span class='kw'>filter</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>)</div><div class='output co'>#&gt; <span style='color: #555555;'># A tibble: 49,395 x 37</span><span>
#&gt;    Probe Sequence GeneSymbol GeneName GemmaId NCBIid `GSE81454_Bioma~
#&gt;    </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span> </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>    </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>      </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>    </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>   </span><span style='color: #555555;font-style: italic;'>&lt;chr&gt;</span><span>             </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span>
#&gt; </span><span style='color: #555555;'> 1</span><span> 1171~ 1171510~ </span><span style='color: #BB0000;'>NA</span><span>         </span><span style='color: #BB0000;'>NA</span><span>       </span><span style='color: #BB0000;'>NA</span><span>      </span><span style='color: #BB0000;'>NA</span><span>                 5.57
#&gt; </span><span style='color: #555555;'> 2</span><span> 1171~ 1171510~ </span><span style='color: #BB0000;'>NA</span><span>         </span><span style='color: #BB0000;'>NA</span><span>       </span><span style='color: #BB0000;'>NA</span><span>      </span><span style='color: #BB0000;'>NA</span><span>                 5.49
#&gt; </span><span style='color: #555555;'> 3</span><span> 1171~ 1171510~ </span><span style='color: #BB0000;'>NA</span><span>         </span><span style='color: #BB0000;'>NA</span><span>       </span><span style='color: #BB0000;'>NA</span><span>      </span><span style='color: #BB0000;'>NA</span><span>                 5.26
#&gt; </span><span style='color: #555555;'> 4</span><span> 1171~ 1171510~ TNFAIP8L1  TNF alp~ 383466  126282             6.95
#&gt; </span><span style='color: #555555;'> 5</span><span> 1171~ 1171510~ OTOP2      otopetr~ 371285  92736              4.90
#&gt; </span><span style='color: #555555;'> 6</span><span> 1171~ 1171510~ C17orf78   chromos~ 418711  284099             3.85
#&gt; </span><span style='color: #555555;'> 7</span><span> 1171~ 1171510~ CTAGE6|CT~ CTAGE f~ 425383~ 34030~             5.08
#&gt; </span><span style='color: #555555;'> 8</span><span> 1171~ 1171510~ F8A1|F8A3~ coagula~ 176266~ 8263|~             5.22
#&gt; </span><span style='color: #555555;'> 9</span><span> 1171~ 1171510~ LINC01098  long in~ 421233  285501             3.67
#&gt; </span><span style='color: #555555;'>10</span><span> 1171~ 1171510~ SAMD7      sterile~ 426827  344658             4.54
#&gt; </span><span style='color: #555555;'># ... with 49,385 more rows, and 30 more variables:</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_17___BioAssayId=414332Name=patient1258.030[38792]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_19___BioAssayId=414331Name=patient1258.030[38848]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_3___BioAssayId=414330Name=patient1258.030[39011]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_1___BioAssayId=414329Name=patient1258.030[39105]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_29___BioAssayId=414345Name=patient1258.343[39400]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_5___BioAssayId=414344Name=patient1258.343[39428]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_2___BioAssayId=414343Name=patient1258.343[39456]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_9___BioAssayId=414342Name=patient1258.343[39540]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_7___BioAssayId=414341Name=patient1258.343[39576]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_18___BioAssayId=414340Name=patient1258.343[39581]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_20___BioAssayId=414339Name=patient1258.343[39602]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_21___BioAssayId=414338Name=patient1258.343[39624]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_22___BioAssayId=414337Name=patient1258.587[39450]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_24___BioAssayId=414335Name=patient1258.587[39506]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_23___BioAssayId=414336Name=patient1258.587[39513]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_25___BioAssayId=414334Name=patient1258.587[39590]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_4___BioAssayId=414359Name=patient1258.896[39226]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_6___BioAssayId=414358Name=patient1258.896[39254]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_8___BioAssayId=414357Name=patient1258.896[39288]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_10___BioAssayId=414356Name=patient1258.896[39367]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_11___BioAssayId=414355Name=patient1258.896[39548]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_12___BioAssayId=414354Name=patient1258.896[39562]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_13___BioAssayId=414353Name=patient1258.896[39591]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_14___BioAssayId=414352Name=patient1258.896[39623]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_15___BioAssayId=414351Name=patient1258.914[39345]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_26___BioAssayId=414350Name=patient1258.914[39373]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_27___BioAssayId=414349Name=patient1258.914[39457]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_30___BioAssayId=414348Name=patient1258.914[39547]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_31___BioAssayId=414347Name=patient1258.914[39571]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span style='color: #555555;'>,</span><span>
#&gt; </span><span style='color: #555555;'>#   `GSE81454_Biomat_28___BioAssayId=414346Name=patient1258.914[39583]` </span><span style='color: #555555;font-style: italic;'>&lt;dbl&gt;</span><span></div></span></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Ogan Mancarci.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


