<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html>
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>platformInfo — platformInfo • gemmaAPI</title>

<!-- jquery -->
<script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha384-nrOSfDHtoPMzJHjVTdCopGqIqeYETSXhZDFyniQ8ZHcVy08QesyHcnOUpMpqnmWq" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>

<!-- Font Awesome icons -->
<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js" integrity="sha384-cV+rhyOuRHc9Ub/91rihWcGmMmCXDeksTtCihMupQHSsi8GIIRDG0ThDc3HGQFJ3" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../jquery.sticky-kit.min.js"></script>
<script src="../pkgdown.js"></script>
  
  
<meta property="og:title" content="platformInfo — platformInfo" />

<meta property="og:description" content="Retrieves information about a single platform. Combines several API calls." />
<meta name="twitter:card" content="summary" />
<!-- mathjax -->
<script src='https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->


  </head>

  <body>
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="../index.html">gemmaAPI</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
      
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      
      </header>

      <div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>platformInfo</h1>
    </div>

    
    <p>Retrieves information about a single platform. Combines several API calls.</p>
    

    <pre class="usage"><span class='fu'>platformInfo</span>(<span class='no'>platform</span>, <span class='kw'>request</span> <span class='kw'>=</span> <span class='kw'>NULL</span>, <span class='no'>...</span>, <span class='kw'>file</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>return</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>, <span class='kw'>overwrite</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>, <span class='kw'>memoised</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>)</pre>
    
    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a> Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>platform</th>
      <td><p>Can either be the platform ID or its short name (e.g: GPL1355). 
Retrieval by ID is more efficient. Only platforms that user has access to will be available.</p>
<p>If a vector of length&gt;1 is provided return all matching platform
objects similar to <code><a href='allPlatforms.html'>allPlatforms</a></code> but without access to additional 
parameters. <code>request</code> parameter cannot be specified for vector inputs</p></td>
    </tr>
    <tr>
      <th>request</th>
      <td><p>Character. If NULL retrieves the platform object. Otherwise</p><ul>
<li><p><code>datasets</code>: Retrieves experiments in the given platform. Parameters:</p><ul>
<li><p><code>offset</code>: Optional, defaults to 0. Skips the 
             specified amount of objects when retrieving them from the
              database.</p></li>
<li><p><code>limit</code>: Optional, defaults to 20. Limits the result 
             to specified amount of objects. Use 0 for no limit.</p></li>
</ul></li>
<li><p><code>annotations</code>: Retrieves the annotation file for the given platform. If you set a file path, the downloaded file will be a .gz file.</p></li>
<li><p><code>elements</code>: Retrieves the composite sequences (elements/probes) for the given platform. Parameters:</p><ul>
<li><p><code>offset</code>: As in datasets</p></li>
<li><p><code>limit</code>: As in datasets</p></li>
</ul></li>
<li><p><code>specificElement</code> Retrieves a specific composite sequence (element) for the given platform. Paramaters:</p><ul>
<li><p><code>probe</code>: Required. Can either be the probe name or ID.</p></li>
</ul></li>
<li><p><code>genes</code> Retrieves the genes on the given platform element. Parameters:</p><ul>
<li><p><code>probe</code>: Required. Can either be the probe name or ID.</p></li>
<li><p><code>offset</code>: As in datasets</p></li>
<li><p><code>limit</code>: As in datasets</p></li>
</ul></li>
</ul></td>
    </tr>
    <tr>
      <th>...</th>
      <td><p>Use if the specified request has additional parameters.</p></td>
    </tr>
    <tr>
      <th>file</th>
      <td><p>Character. File path. If provided, response will be saved to file</p></td>
    </tr>
    <tr>
      <th>return</th>
      <td><p>Logical. If the response should be returned. Set to false when
you only want to save a file</p></td>
    </tr>
    <tr>
      <th>overwrite</th>
      <td><p>Logical. If TRUE, existing files will be overwritten. If FALSE
a warning will be thrown and no action is taken.</p></td>
    </tr>
    <tr>
      <th>memoised</th>
      <td><p>Logical. If TRUE a memoised version of the function will be
used which is faster for repeated requests. Use <code><a href='forgetGemmaMemoised.html'>forgetGemmaMemoised</a></code>
to clear memory.</p></td>
    </tr>
    </table>
    
    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A data.frame or a list depending on the request</p>
    

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='fu'>platformInfo</span>(<span class='st'>'GPL1355'</span>)</div><div class='output co'>#&gt; $GPL1355
#&gt; $GPL1355$shortName
#&gt; [1] "GPL1355"
#&gt; 
#&gt; $GPL1355$dateCached
#&gt; NULL
#&gt; 
#&gt; $GPL1355$numProbeSequences
#&gt; NULL
#&gt; 
#&gt; $GPL1355$numProbeAlignments
#&gt; NULL
#&gt; 
#&gt; $GPL1355$numProbesToGenes
#&gt; NULL
#&gt; 
#&gt; $GPL1355$numGenes
#&gt; NULL
#&gt; 
#&gt; $GPL1355$designElementCount
#&gt; NULL
#&gt; 
#&gt; $GPL1355$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GPL1355$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GPL1355$lastGeneMapping
#&gt; NULL
#&gt; 
#&gt; $GPL1355$lastSequenceAnalysis
#&gt; NULL
#&gt; 
#&gt; $GPL1355$lastSequenceUpdate
#&gt; NULL
#&gt; 
#&gt; $GPL1355$color
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GPL1355$isMerged
#&gt; NULL
#&gt; 
#&gt; $GPL1355$isMergee
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL1355$isSubsumed
#&gt; NULL
#&gt; 
#&gt; $GPL1355$isSubsumer
#&gt; NULL
#&gt; 
#&gt; $GPL1355$lastRepeatMask
#&gt; NULL
#&gt; 
#&gt; $GPL1355$expressionExperimentCount
#&gt; [1] 259
#&gt; 
#&gt; $GPL1355$hasBlatAssociations
#&gt; NULL
#&gt; 
#&gt; $GPL1355$hasGeneAssociations
#&gt; NULL
#&gt; 
#&gt; $GPL1355$hasSequenceAssociations
#&gt; NULL
#&gt; 
#&gt; $GPL1355$description
#&gt; [1] " The GeneChip Rat Genome 230 2.0 Array is a powerful tool for toxicology, neurobiology, and other applications using rat as a model organism.  - Provides comprehensive coverage of the transcribed rat genome on a single array - Comprised of more than 31,000 probe sets, analyzing over 30,000 transcripts and variants from over 28,000 well-substantiated rat genes - The publicly available draft of the rat genome and leading public rat databases were used to refine sequences and provide a higher quality of data output  All probe sets represented on the GeneChip Rat Expression Set 230 are included on the GeneChip Rat Genome 230 2.0 Array. Sequences used in the design of the GeneChip Rat Genome 230 2.0 Array were selected from GenBank, dbEST, and RefSeq. The sequence clusters were created from the UniGene database (Build 99, June 2002) and then refined by analysis and comparison with the publicly available draft assembly of the rat genome from the Baylor College of Medicine Human Genome Sequencing Center (June 2002).  The GeneChip Rat Genome 230 2.0 Array includes representation of the RefSeq database sequences and probe sets related to sequences and refined EST clusters previously represented on the GeneChip Rat Genome U34 Set.  Oligonucleotide probes complementary to each corresponding sequence are synthesized in situ on the arrays. Eleven pairs of oligonucleotide probes are used to measure the level of transcription of each sequence represented on the GeneChip Rat Genome 230 2.0 Array.  Annotations derived from Affymetrix CSV file dated 6/23/2004\nFrom GPL1355\nLast Updated: May 31 2005"
#&gt; 
#&gt; $GPL1355$name
#&gt; [1] "Affymetrix GeneChip Rat Genome 230 2.0 Array"
#&gt; 
#&gt; $GPL1355$lastUpdated
#&gt; [1] 1.523149e+12
#&gt; 
#&gt; $GPL1355$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL1355$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GPL1355$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL1355$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GPL1355$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GPL1355$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GPL1355$curationNote
#&gt; NULL
#&gt; 
#&gt; $GPL1355$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GPL1355$id
#&gt; [1] 2
#&gt; 
#&gt; </div><div class='input'><span class='fu'>platformInfo</span>(<span class='st'>'GPL1355'</span>,<span class='kw'>request</span> <span class='kw'>=</span> <span class='st'>'datasets'</span>,<span class='kw'>limit</span> <span class='kw'>=</span> <span class='fl'>10</span>)</div><div class='output co'>#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.5677968
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.4625
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] -0.3
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] -0.08686478
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] -0.02542229
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.08230379
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] " Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA..\nDate GSE2872 Last Updated: Jul 06 2005\n\nContributers:  S R D'Mello\n\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074..\n"
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.537208e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; 
#&gt; $GSE2870
#&gt; $GSE2870$shortName
#&gt; [1] "GSE2870"
#&gt; 
#&gt; $GSE2870$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2870$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$bioMaterialCount
#&gt; [1] 18
#&gt; 
#&gt; $GSE2870$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2870$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2870$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE2870$experimentalDesign
#&gt; [1] 205
#&gt; 
#&gt; $GSE2870$accession
#&gt; [1] "GSE2870"
#&gt; 
#&gt; $GSE2870$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2870$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2870$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2870$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2870$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2870$geeq
#&gt; $GSE2870$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$geeq$publicQualityScore
#&gt; [1] 0.5686016
#&gt; 
#&gt; $GSE2870$geeq$publicSuitabilityScore
#&gt; [1] 0.4625
#&gt; 
#&gt; $GSE2870$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2870$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2870$geeq$sScoreSampleSize
#&gt; [1] -0.3
#&gt; 
#&gt; $GSE2870$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE2870$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$geeq$qScoreSampleMeanCorrelation
#&gt; [1] -0.05144912
#&gt; 
#&gt; $GSE2870$geeq$qScoreSampleMedianCorrelation
#&gt; [1] -0.01978897
#&gt; 
#&gt; $GSE2870$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.04760352
#&gt; 
#&gt; $GSE2870$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2870$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2870$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2870$geeq$id
#&gt; [1] 770
#&gt; 
#&gt; 
#&gt; $GSE2870$bioAssayCount
#&gt; [1] 18
#&gt; 
#&gt; $GSE2870$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2870$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2870$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2870$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2870$description
#&gt; [1] " Prolonged stress has adverse consequences for neurons in the hippocampus (HIPP). The amygdala (AMY) is a brain region that is similar to HIPP across many measures, suggesting that chronic stress might modulate AMY and HIPP function in similar ways. However, studies addressing this issue have produced surprising results. For example, a regimen of chronic stress shown to produce atrophy in HIPP neurons caused dendritic branching in AMY neurons. These and other data have revealed that excessive stress induces fundamentally opposing processes in the AMY and HIPP, such that AMY function is facilitated by levels of stress that produce deleterious effects in HIPP. The mechanisms that contribute to the opposing responses of these brain regions to chronic stress are unknown. Such knowledge may suggest novel directions for pharmacological interventions to protect the HIPP from stress-mediated damage. We will determine gene expression patterns in the hippocampus and amygdala under basal and stressful conditions, with a particular interest in those genes that are differentially regulated across the two regions. We hypothesize that genes that are oppositely regulated in the amygdala and hippocampus after stress mediate the opposing functional consequences of chronic stress in these two brain regions.  Moreover, genes which are differentially expressed in these regions under basal conditions may underlie the distinct responses of these regions to stress. Rats will receive either 30sec of handling (Control group; n=15) or 3hr of immobilization stress (Stress group; n=15) each day for 14 consecutive days.  This stress paradigm has previously been shown to produce opposing effects on dendritic morphology and region-dependent behaviors for  the hippocampus versus amygdala.  Twenty-four hours after the last stress or handling session, the rats will be overanaesthetized, and the brain will be removed and placed in ice-cold buffer.  Tissue will be rapidly dissected from the basolateral complex of the amygdala and the CA3 region of the hippocampus of both hemispheres, flash frozen in liquid nitrogen, and stored at -80C until further processing.  The tissue from each brain region will be pooled across groups, yielding four samples (Control-Hippocampus, Control-Amygdala, Stress-Hippocampus, and Stress-Amygdala).  Total RNA will be extracted using standard methods, and sent to the centers.  Each sample will be run in triplicate, utilizing a total of 12 Affymetrix Rat Genome U34A gene chips.  A triplicate array assay of pooled samples has previously been shown to both substantially increase the sensitivity of detecting genes of interest and reduce the variability associated with individual microarrays.  In addition, the large number of rats used in each group will reduce the variability in gene expression associated with individual responses to chronic stress, as well as variability due to slight anatomical differences in the tissue extracted from each rat.\n\nLast Updated (by provider): Jul 06 2005\n\nContributers:  W Ogle\n\nIncludes GDS1794.\n Update date: Jun 08 2006.\n Dataset description GDS1794: Analysis of the amygdalae (Amy) and hippocampi (Hipp) of animals subjected to chronic stress induced by daily handling sessions for 14 days. Studies indicate that chronic stress produce a response in Hipp that is opposite of that in Amy..\n"
#&gt; 
#&gt; $GSE2870$source
#&gt; NULL
#&gt; 
#&gt; $GSE2870$name
#&gt; [1] "Ogle-5P01NS037520-05"
#&gt; 
#&gt; $GSE2870$id
#&gt; [1] 199
#&gt; 
#&gt; $GSE2870$lastUpdated
#&gt; [1] 1.537209e+12
#&gt; 
#&gt; $GSE2870$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2870$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2870$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent
#&gt; $GSE2870$lastNeedsAttentionEvent$performer
#&gt; [1] "hi1hi1"
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent$date
#&gt; [1] 1.525908e+12
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent$eventType
#&gt; $GSE2870$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216715
#&gt; 
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2870$lastNeedsAttentionEvent$id
#&gt; [1] 25065138
#&gt; 
#&gt; 
#&gt; $GSE2870$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2870$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2870$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; 
#&gt; $GSE4037
#&gt; $GSE4037$shortName
#&gt; [1] "GSE4037"
#&gt; 
#&gt; $GSE4037$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE4037$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$bioMaterialCount
#&gt; [1] 18
#&gt; 
#&gt; $GSE4037$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE4037$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE4037$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE4037$experimentalDesign
#&gt; [1] 270
#&gt; 
#&gt; $GSE4037$accession
#&gt; [1] "GSE4037"
#&gt; 
#&gt; $GSE4037$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE4037$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE4037$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE4037$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE4037$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE4037$geeq
#&gt; $GSE4037$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$geeq$publicQualityScore
#&gt; [1] 0.6943326
#&gt; 
#&gt; $GSE4037$geeq$publicSuitabilityScore
#&gt; [1] 0.4625
#&gt; 
#&gt; $GSE4037$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE4037$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE4037$geeq$sScoreSampleSize
#&gt; [1] -0.3
#&gt; 
#&gt; $GSE4037$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE4037$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$geeq$qScoreSampleMeanCorrelation
#&gt; [1] -0.05754252
#&gt; 
#&gt; $GSE4037$geeq$qScoreSampleMedianCorrelation
#&gt; [1] -0.1396716
#&gt; 
#&gt; $GSE4037$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.1194644
#&gt; 
#&gt; $GSE4037$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE4037$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE4037$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE4037$geeq$id
#&gt; [1] 1022
#&gt; 
#&gt; 
#&gt; $GSE4037$bioAssayCount
#&gt; [1] 18
#&gt; 
#&gt; $GSE4037$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE4037$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE4037$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE4037$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE4037$description
#&gt; [1] " The goal of the research proposed herein is to identify (i) gene products that allow identification of specific precursor populations in vivo and (ii) genetic programs that regulate the generation of lineaege-restricted precursor (LRP) cells from neuroepithelial stem cells (NSC).  As these cells play a major role in development and tissue repair, this effort will enhance both basic and clinically relevant research. Note: Funding for this micro-array analysis was granted through NINDS as supplement 3R01NS042820-02S1. Our ability to generate essentially unlimited numbers of purified rodent NSCs, thier restricted decendants leading the generation of macroglia - and also to control the generation of each of these cell types from its immediate ancestor - provides us with a unique system for studying the multi-step nature of differentiation. Using defined conditions, samples will be isolated from each type of purified cell population. Expression profiles will be compared to reveal cell type specific markers and regulatory factors. The generation of differentiated cell types from neuroepithelial stem cells appears to require progression through a sequence of lineaege-restricted intermediate cell stages.  In the case of at least one of the pathways involved in oliigodendrocyte generation, it now appears that all of the stages of this sequence have been defined and all of the intermediate cells have been isolated (as detailed in the Noble, M. et al. (2004)  Dev. Biol. 265, 33-52).  Studies in vitro, and consequent to cell transplantation, have enabled us to gain an increasingly detailed understanding of the differentiation potential of these cells. The microarray analysis proposed herein will provide the specific markers that will enable the LRP cells to be unambiguously identified in vivo, thereby setting the stage to develop a detailed mapping of the role of these cells during development. RNA will be isolated from purified cell populations.  All populations will initially be derived from spinal cord.  Preparation of purified populations of each of these cell types has been described in multiple of our previous publications (Mayer, M. et al. (1994) Development 120, 142-153; Mayer-Pröschel, M. et al. (1997) Neuron 19, 773-785; Rao, M. and Mayer-Pröschel, M. (1997) Dev. Biology 188, 48-63). Total RNA from purified cell cultures will be extracted in a two step process. RNA will first be extracted by dounce homogenization in phenol/guanidine isothiocyonate solution (Trizol, Invitrogen, California) followed by isopropanol precipitation. Contaminating traces of DNA and protein will then be removed by silica-based affinity chromatography and Dnase treatment (NucleoSpin RNA Purification system, Clontech, California). Depending on the cell type, 5x105 cells will typically yield 5-15µg of high quality RNA (OD260/280=2.1 at pH7.5). In order to determine purity and molecular weight distribution, samples will be routinely tested on a Agilent Bioanalyzer. cDNA will be generated from total RNA and then be submitted to one round of amplification and labeling using the messageAmp system (Ambion). Probes are then hybridized to the Affymerix Rat 230 (A/B or 2.0) GeneChip expression array. Array data will be processed using the Micro Array Suite 5.0 (MAS, Affymetrix). Cel files containing normalized, relative expression data for all genes tested will be used for downstream statistical analysis.\n\nLast Updated (by provider): Jan 13 2006\n\nContributers:  Christoph Proschel\n"
#&gt; 
#&gt; $GSE4037$source
#&gt; NULL
#&gt; 
#&gt; $GSE4037$name
#&gt; [1] "prosc-affy-rat-185420"
#&gt; 
#&gt; $GSE4037$id
#&gt; [1] 289
#&gt; 
#&gt; $GSE4037$lastUpdated
#&gt; [1] 1.537209e+12
#&gt; 
#&gt; $GSE4037$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE4037$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4037$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent
#&gt; $GSE4037$lastNeedsAttentionEvent$performer
#&gt; [1] "hi1hi1"
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent$date
#&gt; [1] 1.525971e+12
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent$eventType
#&gt; $GSE4037$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216833
#&gt; 
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE4037$lastNeedsAttentionEvent$id
#&gt; [1] 25065376
#&gt; 
#&gt; 
#&gt; $GSE4037$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE4037$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE4037$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; 
#&gt; $GSE3110
#&gt; $GSE3110$shortName
#&gt; [1] "GSE3110"
#&gt; 
#&gt; $GSE3110$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE3110$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$bioMaterialCount
#&gt; [1] 10
#&gt; 
#&gt; $GSE3110$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE3110$batchConfound
#&gt; [1] "Factor 'Treatment' may be confounded with batches; p=0.0098"
#&gt; 
#&gt; $GSE3110$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC 1), p=0.0098"
#&gt; 
#&gt; $GSE3110$experimentalDesign
#&gt; [1] 275
#&gt; 
#&gt; $GSE3110$accession
#&gt; [1] "GSE3110"
#&gt; 
#&gt; $GSE3110$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE3110$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE3110$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE3110$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE3110$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE3110$geeq
#&gt; $GSE3110$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$geeq$publicQualityScore
#&gt; [1] 0.2724288
#&gt; 
#&gt; $GSE3110$geeq$publicSuitabilityScore
#&gt; [1] 0.7125
#&gt; 
#&gt; $GSE3110$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE3110$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE3110$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE3110$geeq$sScoreSampleSize
#&gt; [1] -0.3
#&gt; 
#&gt; $GSE3110$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE3110$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$geeq$qScoreSampleMeanCorrelation
#&gt; [1] -0.1074191
#&gt; 
#&gt; $GSE3110$geeq$qScoreSampleMedianCorrelation
#&gt; [1] -0.09299869
#&gt; 
#&gt; $GSE3110$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.06537356
#&gt; 
#&gt; $GSE3110$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE3110$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE3110$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE3110$geeq$id
#&gt; [1] 1028
#&gt; 
#&gt; 
#&gt; $GSE3110$bioAssayCount
#&gt; [1] 10
#&gt; 
#&gt; $GSE3110$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE3110$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE3110$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE3110$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE3110$description
#&gt; [1] " The hypothalamic paraventricular (PVN) and supraoptic (SON) nuclei  are important integrative structures that regulate co-ordinated  responses to perturbations in cardiovascular homeostasis. Through  descending projections from parvocellular neurons to the brainstem,  the PVN acts as an autonomic 'premotor nucleus', regulating reflex  changes in sympathetic nerve activity that are involved in blood  pressure and blood volume regulation. Endocrine responses are  mediated through axonal projections from SON and PVN magnocellular  neurons (MCNs) to the capillaries of the posterior pituitary neural  lobe. In response to dehydration, a massive release of the  antidiuretic peptide hormone vasopressin (VP) into the circulation is  accompanied by a dramatic functional remodelling of the HNS. We have  used microarrays to comprehensively catalogue the genes expressed in  the PVN, the SON and the neurointermediate lobe (NIL) of the  pituitary gland. Further, we have identified transcripts that are  regulated as a consequence of dehydration, as well as RNAs that are  enriched in either the PVN or the SON. We suggest that these  differentially expressed genes represent candidate regulators and  effectors of HNS activity and remodelling.\n\nLast Updated (by provider): Jan 31 2006\n\nContributers:  GEMMA BEIGHTON JULIAN PATON SONG YAO CHARLES C HINDMARCH\n"
#&gt; 
#&gt; $GSE3110$source
#&gt; NULL
#&gt; 
#&gt; $GSE3110$name
#&gt; [1] "Comprehensive description of the transcriptome of hypothalamo-neurohypophyseal system in euhydrated and dehydrated rat"
#&gt; 
#&gt; $GSE3110$id
#&gt; [1] 294
#&gt; 
#&gt; $GSE3110$lastUpdated
#&gt; [1] 1.537209e+12
#&gt; 
#&gt; $GSE3110$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE3110$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3110$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent
#&gt; $GSE3110$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent$date
#&gt; [1] 1.525731e+12
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent$eventType
#&gt; $GSE3110$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216156
#&gt; 
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE3110$lastNeedsAttentionEvent$id
#&gt; [1] 25064085
#&gt; 
#&gt; 
#&gt; $GSE3110$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE3110$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE3110$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; 
#&gt; $GSE3125
#&gt; $GSE3125$shortName
#&gt; [1] "GSE3125"
#&gt; 
#&gt; $GSE3125$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE3125$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$bioMaterialCount
#&gt; [1] 10
#&gt; 
#&gt; $GSE3125$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE3125$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE3125$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE3125$experimentalDesign
#&gt; [1] 276
#&gt; 
#&gt; $GSE3125$accession
#&gt; [1] "GSE3125"
#&gt; 
#&gt; $GSE3125$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE3125$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE3125$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE3125$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE3125$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE3125$geeq
#&gt; $GSE3125$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$geeq$publicQualityScore
#&gt; [1] 0.2838638
#&gt; 
#&gt; $GSE3125$geeq$publicSuitabilityScore
#&gt; [1] 0.4625
#&gt; 
#&gt; $GSE3125$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE3125$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE3125$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE3125$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE3125$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE3125$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE3125$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE3125$geeq$sScoreSampleSize
#&gt; [1] -0.3
#&gt; 
#&gt; $GSE3125$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE3125$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE3125$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE3125$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE3125$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE3125$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE3125$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.984347
#&gt; 
#&gt; $GSE3125$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9870464
#&gt; 
#&gt; $GSE3125$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 6.439892e-05
#&gt; 
#&gt; $GSE3125$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE3125$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE3125$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE3125$geeq$id
#&gt; [1] 1032
#&gt; 
#&gt; 
#&gt; $GSE3125$bioAssayCount
#&gt; [1] 10
#&gt; 
#&gt; $GSE3125$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE3125$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE3125$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE3125$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE3125$description
#&gt; [1] " The hypothalamic paraventricular (PVN) and supraoptic (SON) nuclei  are important integrative structures that regulate co-ordinated  responses to perturbations in cardiovascular homeostasis. Through  descending projections from parvocellular neurons to the brainstem,  the PVN acts as an autonomic 'premotor nucleus', regulating reflex  changes in sympathetic nerve activity that are involved in blood  pressure and blood volume regulation. Endocrine responses are  mediated through axonal projections from SON and PVN magnocellular  neurons (MCNs) to the capillaries of the posterior pituitary neural  lobe. In response to dehydration, a massive release of the  antidiuretic peptide hormone vasopressin (VP) into the circulation is  accompanied by a dramatic functional remodelling of the HNS. We have  used microarrays to comprehensively catalogue the genes expressed in  the PVN, the SON and the neurointermediate lobe (NIL) of the  pituitary gland. Further, we have identified transcripts that are  regulated as a consequence of dehydration, as well as RNAs that are  enriched in either the PVN or the SON. We suggest that these  differentially expressed genes represent candidate regulators and  effectors of HNS activity and remodelling.\n\nLast Updated (by provider): Jan 31 2006\n\nContributers:  GEMMA BEIGHTON JULIAN PATON SONG YAO CHARLES C HINDMARCH\n"
#&gt; 
#&gt; $GSE3125$source
#&gt; NULL
#&gt; 
#&gt; $GSE3125$name
#&gt; [1] "Comprehensive description of the transcriptome of neurointermediate lobe in euhydrated and dehydrated rat"
#&gt; 
#&gt; $GSE3125$id
#&gt; [1] 295
#&gt; 
#&gt; $GSE3125$lastUpdated
#&gt; [1] 1.537209e+12
#&gt; 
#&gt; $GSE3125$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE3125$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3125$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent
#&gt; $GSE3125$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent$date
#&gt; [1] 1.525732e+12
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent$eventType
#&gt; $GSE3125$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216161
#&gt; 
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE3125$lastNeedsAttentionEvent$id
#&gt; [1] 25064096
#&gt; 
#&gt; 
#&gt; $GSE3125$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE3125$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE3125$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; 
#&gt; $GSE4776
#&gt; $GSE4776$shortName
#&gt; [1] "GSE4776"
#&gt; 
#&gt; $GSE4776$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE4776$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$bioMaterialCount
#&gt; [1] 83
#&gt; 
#&gt; $GSE4776$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE4776$batchConfound
#&gt; [1] "Factor 'OrganismPart' may be confounded with batches; p=8.9e-16"
#&gt; 
#&gt; $GSE4776$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC 4), p=1.7737e-06"
#&gt; 
#&gt; $GSE4776$experimentalDesign
#&gt; [1] 304
#&gt; 
#&gt; $GSE4776$accession
#&gt; [1] "GSE4776"
#&gt; 
#&gt; $GSE4776$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE4776$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE4776$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE4776$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE4776$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE4776$geeq
#&gt; $GSE4776$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$geeq$publicQualityScore
#&gt; [1] -0.0007930966
#&gt; 
#&gt; $GSE4776$geeq$publicSuitabilityScore
#&gt; [1] 0.375
#&gt; 
#&gt; $GSE4776$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE4776$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE4776$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE4776$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE4776$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE4776$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE4776$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE4776$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE4776$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE4776$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE4776$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE4776$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE4776$geeq$qScoreReplicates
#&gt; [1] -1
#&gt; 
#&gt; $GSE4776$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE4776$geeq$qScoreSampleMeanCorrelation
#&gt; [1] -0.01030632
#&gt; 
#&gt; $GSE4776$geeq$qScoreSampleMedianCorrelation
#&gt; [1] -0.005551676
#&gt; 
#&gt; $GSE4776$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.128428
#&gt; 
#&gt; $GSE4776$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE4776$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE4776$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE4776$geeq$id
#&gt; [1] 1129
#&gt; 
#&gt; 
#&gt; $GSE4776$bioAssayCount
#&gt; [1] 83
#&gt; 
#&gt; $GSE4776$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE4776$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE4776$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE4776$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE4776$description
#&gt; [1] " It is suggested that the stress induced activation of the HPA axis and associated increases in plasma ACTH and glucocorticoids (corticosterone, B in the rat) are contributing factors in the development of many psychopathologies.  However, the circadian fluctuation of the HPA axis activity (characterized by a zenith in the plasma ACTH and B, before the beginning of the active awake state and the nadir during the inactive sleeping period) occurs throughout the life and is important for normal physiological and behavioral functioning. A blunting or enhancement of this rhythm through changes in the trough or in the peak is a characteristic feature of many pathological states. At the CNS level, the circadian fluctuation of the HPA axis activity is associated with changes in the expression profile of structural, functional, and immediate early genes. However, little is known about the specific role of B in the modulation of the circadian pattern of gene regulation in the CNS. Thus, during the circadian cycle B is a major factor, which could influence the expression profile of a large family of genes. Accordingly, the microarray technology, because of its characteristic feature that facilitates the identification of the expression profiles of a wide range of genes simultaneously, is an ideal technique. Because the B status of animals can be easily manipulated by adrenalectomy (ADX) surgery with or without B replacement, this model will be used in these studies. Thus, the proposed studies are intended to characterize, by the use of the microarray technique, the effects of the B milieu on diurnal gene expression profiles in selected rat brain regions strongly influenced by: a.) B status: amygdala, hippocampus, hypothalamus, and septum; and  b.)  Circadian Activity: dorsal raphe, frontal Cortex and locus coeruleus. The gradient of the B response during the circadian cycle implies that at the beginning of the sleep cycle MR are predominantly occupied while at the end of the sleep cycle, both MR and GR are occupied. Thus, we propose that differing ratios of activation of these nuclear receptors could differentially modulate gene expression and impart temporal characteristics in the expression profile of a large family of genes. Furthermore, we propose that these effects may be region specific, reflecting regional differences in GR and MR distribution, B uptake, and neural activity. B may exert these circadian effects on regional gene expression profiles either directly or indirectly via interneurons. However, little is known about the circadian effect of B on gene expression profiles in different brain regions. Such knowledge is important to identify the specific role of B in circadian differences in CNS function and, potentially, the development of psychopathologies. After arrival at the facility the rats (male Long Evans) will be maintained on 12:12 h LD cycle ('lights-on\" at 0700 h) with food and water ad libitum. One week after, they will undergo bilateral ADX under survival anesthetics (day-0).  The study will include three groups (n=10); (a) ADX: rats with no adrenals, (b) pADX: ADX rats with subcutaneous 40 percent B pellet (fused cholesterol and B), and (c) SHAM: rats undergoing flank incisions without the removal of the adrenals. After surgery, rats will be housed individually. The ADX rats will receive 0.5 percent saline in addition to regular tap water. On post-surgical day-7, 15 rats (five/group) will be sacrificed by decapitation 2h after 'lights-on\", and the remaining 15, 2h after \"lights-off\". The brains will be removed immediately, frozen on dry ice, and stored at -80 C. The trunk blood will be collected with EDTA and plasma frozen for ACTH and B RIA. The brains from rats with confirmed ADX will be used for free hand dissection of brain regions: frontal cortex, septum, hypothalamus, amygdala, hippocampus, dorsal raphe, and locus coeruleus. Tissue will be stored at -80 C. Total RNA will be extracted from individual brain regions using Qiagen RNeasy Lipid Tissue kit. Initial evaluation of quality and quantity of the RNA will be done by the absorbance method. Frozen samples containing 0.5 ul total RNA or higher will be sent to the NINDS-NIMH Microarray Consortium in dry ice for evaluation of the integrity of 28S and 18S RNA on an Agilent Bioanalyzer. If the integrity of RNA is not good, the experiment will be repeated and new RNA samples will be provided. Replicates of each brain region with 28S/18S ratio of 1.7 and higher will be selected for assay using Affymetrix Rat U34A array. The RNA samples from one region from all the brains will be processed together. A total of 84 chips may be utilized in these studies, however the final number will depend upon integrity of the total RNA.\nNote: [] samples from this series, which appear in other Expression Experiments in Gemma, were not imported from the GEO source. The following samples were removed: \nLast Updated (by provider): Jun 06 2006\n\nContributers:  K. V Thrivikraman\n"
#&gt; 
#&gt; $GSE4776$source
#&gt; NULL
#&gt; 
#&gt; $GSE4776$name
#&gt; [1] "Thrivikraman-5P50MH058922-050002"
#&gt; 
#&gt; $GSE4776$id
#&gt; [1] 322
#&gt; 
#&gt; $GSE4776$lastUpdated
#&gt; [1] 1.537209e+12
#&gt; 
#&gt; $GSE4776$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE4776$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4776$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent
#&gt; $GSE4776$lastNeedsAttentionEvent$performer
#&gt; [1] "ellie27h"
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent$date
#&gt; [1] 1.52667e+12
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent$eventType
#&gt; $GSE4776$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 219790
#&gt; 
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE4776$lastNeedsAttentionEvent$id
#&gt; [1] 25071194
#&gt; 
#&gt; 
#&gt; $GSE4776$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE4776$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE4776$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; 
#&gt; $GSE4753
#&gt; $GSE4753$shortName
#&gt; [1] "GSE4753"
#&gt; 
#&gt; $GSE4753$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE4753$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$bioMaterialCount
#&gt; [1] 40
#&gt; 
#&gt; $GSE4753$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE4753$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE4753$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE4753$experimentalDesign
#&gt; [1] 324
#&gt; 
#&gt; $GSE4753$accession
#&gt; [1] "GSE4753"
#&gt; 
#&gt; $GSE4753$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE4753$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE4753$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE4753$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE4753$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE4753$geeq
#&gt; $GSE4753$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$geeq$publicQualityScore
#&gt; [1] 0.7107919
#&gt; 
#&gt; $GSE4753$geeq$publicSuitabilityScore
#&gt; [1] 0.5375
#&gt; 
#&gt; $GSE4753$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE4753$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE4753$geeq$sScoreSampleSize
#&gt; [1] 0.3
#&gt; 
#&gt; $GSE4753$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE4753$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$geeq$qScoreSampleMeanCorrelation
#&gt; [1] -0.02518688
#&gt; 
#&gt; $GSE4753$geeq$qScoreSampleMedianCorrelation
#&gt; [1] -0.02445654
#&gt; 
#&gt; $GSE4753$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.02279474
#&gt; 
#&gt; $GSE4753$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE4753$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE4753$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE4753$geeq$id
#&gt; [1] 1173
#&gt; 
#&gt; 
#&gt; $GSE4753$bioAssayCount
#&gt; [1] 40
#&gt; 
#&gt; $GSE4753$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE4753$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE4753$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE4753$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE4753$description
#&gt; [1] " Hypoxic-ischemic (HI) injury in the developing brain is a common cause of disability in children, and there are no effective treatments at this time.  Exposure to sublethal hypoxic conditions (hypoxic preconditioning) 24 hours prior to hypoxic-ischemic insult is protective in the developing rat model.  We have observed protective effects on brain histopathology and on long-term sensory-motor behavioral tasks.  Changes in gene expression are thought to underlie this protective effect.  By comparing gene expression in rats subjected to hypoxic preconditioning or sham conditioning at several time points from 0 to 24 hrs after preconditioning, we should gain insight into the mechanisms underlying these neuroprotective effects and may identify targets for therapeutic intervention. The aim of this study is to determine the effect of hypoxic preconditioning on global gene expression, and, in littermates, to examine the effect of hypoxic preconditioning 24 h prior to hypoxic-ischemic insult on brain histopathology. We hypothesize that changes in gene expression underlie the protective effect of hypoxic preconditioning against subsequent hypoxic-ischemic insult. Gene expression will be examined in two groups, 1) preconditioned and 2) sham controls, at 4 time points.  On postnatal day 6, preconditioned animals are exposed to normothermic hypoxia for 3 hrs (8.0% oxygen, 36 degrees C), and sham animals are simultaneouosly exposed to normoxia at 36 degrees C. Animals are then returned to their dams until euthanized at 4 time points (0h, 2h, 8h, and 24h later).  Five brains/group/timepoint will be used, with an equal number of males and females in each group.  Brains are removed and dissected on ice.  Cerebral cortex is dissected from both hemispheres and rapidly frozen on dry ice.  Total RNA is isolated using the QIAGEN RNeasy Protect Maxi Kit.  Littermates of these animals will be exposed to hypoxic preconditioning or sham preconditioning and subjected to hypoxic-ischemic injury 24 h later.  These animals are euthanized at postnatal day 14 for histopathologic evaluation of injury.\nNote: [] samples from this series, which appear in other Expression Experiments in Gemma, were not imported from the GEO source. The following samples were removed: \nLast Updated (by provider): May 01 2006\n\nContributers:  Mary-Ann Wilson\n"
#&gt; 
#&gt; $GSE4753$source
#&gt; NULL
#&gt; 
#&gt; $GSE4753$name
#&gt; [1] "wilso-affy-rat-132990"
#&gt; 
#&gt; $GSE4753$id
#&gt; [1] 342
#&gt; 
#&gt; $GSE4753$lastUpdated
#&gt; [1] 1.53721e+12
#&gt; 
#&gt; $GSE4753$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE4753$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4753$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent
#&gt; $GSE4753$lastNeedsAttentionEvent$performer
#&gt; [1] "hi1hi1"
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent$date
#&gt; [1] 1.526494e+12
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent$eventType
#&gt; $GSE4753$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 218389
#&gt; 
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE4753$lastNeedsAttentionEvent$id
#&gt; [1] 25068435
#&gt; 
#&gt; 
#&gt; $GSE4753$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE4753$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE4753$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; 
#&gt; $GSE4031
#&gt; $GSE4031$shortName
#&gt; [1] "GSE4031"
#&gt; 
#&gt; $GSE4031$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE4031$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4031$bioMaterialCount
#&gt; [1] 105
#&gt; 
#&gt; $GSE4031$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE4031$batchConfound
#&gt; [1] "Factor  'Treatment' may be confounded with batches; p=0.0"
#&gt; 
#&gt; $GSE4031$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC2); p=8.9e-14"
#&gt; 
#&gt; $GSE4031$experimentalDesign
#&gt; [1] 329
#&gt; 
#&gt; $GSE4031$accession
#&gt; [1] "GSE4031"
#&gt; 
#&gt; $GSE4031$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE4031$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE4031$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE4031$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE4031$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE4031$geeq
#&gt; $GSE4031$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4031$geeq$publicQualityScore
#&gt; [1] 0.2837996
#&gt; 
#&gt; $GSE4031$geeq$publicSuitabilityScore
#&gt; [1] 0.625
#&gt; 
#&gt; $GSE4031$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE4031$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE4031$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE4031$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE4031$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$qScoreReplicates
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$geeq$qScoreSampleMeanCorrelation
#&gt; [1] -0.008246287
#&gt; 
#&gt; $GSE4031$geeq$qScoreSampleMedianCorrelation
#&gt; [1] -0.01340309
#&gt; 
#&gt; $GSE4031$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.0395783
#&gt; 
#&gt; $GSE4031$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4031$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE4031$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE4031$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE4031$geeq$id
#&gt; [1] 1188
#&gt; 
#&gt; 
#&gt; $GSE4031$bioAssayCount
#&gt; [1] 105
#&gt; 
#&gt; $GSE4031$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE4031$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4031$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4031$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE4031$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4031$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4031$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE4031$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4031$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE4031$description
#&gt; [1] " In pevious research we have shown that the disruption of the normal development of the ventral hippocampus in rodents leads to cellular abnormalities in the frontal cortex and behavioral deficits related to schizophenia (Neurotox Res. 2002, 4(5-6):469-475). We propose the use of gene expression analysis to investigate the molecular underpinnings of these processes which may shed light on the molecular processes relevant to human schizophrenia. In addition, we seek to characterize expression differences induced by chronic administration of antipsychotic medications, which may give insight into the molecular processes involved in ameliorating psychotic symptoms. Using both surgical and drug interventions, we aim to examine experimentally induced expression differences in the rodent  brain that are relevant to human neuropsychiatric disorders. Disruption of the normal development of the vental hippocampus or chronic neuroleptic administration,  will result in gene expression changes in the frontal cortex of rats. Elucidation of the molecular cascades underlying these treatments will shed light on both the pathoetiology (from the lesion experiments) and theurapeutic processes (from the antipsychotic treatment experiments) involved in human schizophrenia. We have developed 3 groups of samples: 1] Ventral hippocampal lesion in neonatal rats (treatment and controls, total N=22) 2] Tetrodotoxin disruption of the ventral hippocampus (treatment and controls, total N=20) 3] Chronic administration of neuroleptics (multiple drugs, multiple doses, and controls, total N=63) Grand Total N=105"
#&gt; 
#&gt; $GSE4031$source
#&gt; [1] "See also: \thttp://arrayconsortium.tgen.org/np2/viewProject.do?action=viewProject&amp;projectId=89421"
#&gt; 
#&gt; $GSE4031$name
#&gt; [1] "colan-affy-rat-89421"
#&gt; 
#&gt; $GSE4031$id
#&gt; [1] 347
#&gt; 
#&gt; $GSE4031$lastUpdated
#&gt; [1] 1.53721e+12
#&gt; 
#&gt; $GSE4031$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE4031$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE4031$needsAttention
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE4031$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE4031$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GSE4031$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE4031$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE4031$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; 
#&gt; $GSE2880
#&gt; $GSE2880$shortName
#&gt; [1] "GSE2880"
#&gt; 
#&gt; $GSE2880$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2880$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$bioMaterialCount
#&gt; [1] 62
#&gt; 
#&gt; $GSE2880$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2880$batchConfound
#&gt; [1] "Factor 'organism part' may be confounded with batches; p=3.6e-13, Factor 'treatment' may be confounded with batches; p=0.0013"
#&gt; 
#&gt; $GSE2880$batchEffect
#&gt; [1] "This data set may have a batch artifact (PC 2), p=1.0332e-09"
#&gt; 
#&gt; $GSE2880$experimentalDesign
#&gt; [1] 400
#&gt; 
#&gt; $GSE2880$accession
#&gt; [1] "GSE2880"
#&gt; 
#&gt; $GSE2880$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2880$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2880$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2880$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2880$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2880$geeq
#&gt; $GSE2880$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$geeq$publicQualityScore
#&gt; [1] 0.1412196
#&gt; 
#&gt; $GSE2880$geeq$publicSuitabilityScore
#&gt; [1] 0.875
#&gt; 
#&gt; $GSE2880$geeq$qScorePublicBatchEffect
#&gt; [1] -1
#&gt; 
#&gt; $GSE2880$geeq$qScorePublicBatchConfound
#&gt; [1] -1
#&gt; 
#&gt; $GSE2880$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2880$geeq$sScoreSampleSize
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2880$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$geeq$qScoreSampleMeanCorrelation
#&gt; [1] -0.01335429
#&gt; 
#&gt; $GSE2880$geeq$qScoreSampleMedianCorrelation
#&gt; [1] -0.01146285
#&gt; 
#&gt; $GSE2880$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 0.03285908
#&gt; 
#&gt; $GSE2880$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2880$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2880$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2880$geeq$id
#&gt; [1] 1328
#&gt; 
#&gt; 
#&gt; $GSE2880$bioAssayCount
#&gt; [1] 62
#&gt; 
#&gt; $GSE2880$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2880$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2880$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2880$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2880$description
#&gt; [1] " Epilepsy is a major neurological disorder that affects approximately 1% of the population. The processes that lead to the development of epilepsy (epileptogenesis) are largely unknown. Levetiracetam is a novel antiepileptic drug (AED) that in the kindling model inhibits epileptogenesis in addition to being effective in controlling established epilepsy. The mechanisms of action of levetiracetam as an AED and an antiepileptogenic drug are unknown. By identifying the effect of chronic levetiracetam therapy on gene expression in the brain we hope to be able to identify genes that are involved in epileptogenesis. By comparing the gene expression profiles of levetiracetam and phenytoin treatments, we hope to be able to distinguish between genes that are important for the antiepileptic (anti-seizure) effect and genes that are important for the antiepileptogenic effect of levetiracetam. Phenytoin is a well-established AED; its mechanism of action involves inhibition of sodium channels. In contrast to levetiracetam, available data suggest that phenytoin in certain situations may enhance rather than inhibit the development of epilepsy. We will identify changes in gene expression in the hippocampus, frontal cortex, and brain stem caused by chronic treatment (3 months) with levetiracetam or phenytoin. We will search for differences in the gene expression profiles after chronic treatment with the two drugs, to explain the different clinical effects of the two drugs. We hypothesize that chronic treatment with levetiracetam leads to changes in the expression of genes that are involved in both the antiepileptogenetic and antiepileptic effect of the drug, whereas chronic treatment with phenytoin affects genes that may be related to pro-epileptogenetic as well as antiepileptic effects of that drug. We further hypothesize that i) regional differences in the effects of levetiracetam and phenytoin may reflect important functional differences in the actions of these two drugs, and ii) there may be different mechanisms involved in their anti-seizure effect. Male Wistar rats receive levetiracetam (150 mg/kg x 2), phenytoin (75 mg/kg x 2) or control solution (N=7 in each group) through a gastric tube twice daily for 90 days. Experimental animals and controls are housed together in the same cages in groups of three, facilitating paired comparisons among the three groups in animals that have experienced the same environment throughout the 3 month experimental period. Four hrs after the last dose animals are anesthetized, decapitated, and total RNA is extracted from brain stem, frontal cortex, and hippocampus. RNA will be sent to the TGEN facility for hybridization to the rat U230A chips. The number of chips needed is (2 drugs + 1 control) * (3 brain regions) * (7 animals per group) = 63 chips. We will initially select those genes for analysis that are called ?present? by the Affymetrix software in 4 or more animals for at least one of the 6 experimental groups. Then t-tests with Benjamini &amp; Hochberg correction for protection against multiple comparisons will be used to identify genes that are differentially expressed at a false discovery rate of .05 among any of the six groups. We will be primarily interested in the following expression profiles, analyzed as paired comparisons by the Affymetrix software:  1. phenytoin vs control in hippocampus 2. phenytoin vs control in brain stem 3. levetiracetam vs control in hippocampus 4. levetiracetam vs control in brain stem  We will then compare the profiles of both drugs in each brain region (e.g., phenytoin vs levetiracetam in hippocampus) to select genes that are drug-specific. We will also take a Venn diagram approach to identify genes that are drug-specific across both brain regions tested.\nLast Updated (by provider): Oct 28 2005\n\nContributers:  Ray Dingledine\n\nIncludes GDS1864.\n Update date: Dec 05 2006.\n Dataset description GDS1864: Analysis of the brainstem, frontal cortex, and hippocampus of animals chronically treated for 90 days with levetiracetam or phenytoin. Levetiracetam inhibits epileptogenesis in the kindling model of epilepsy. Phenytoin's antiepileptic activity involves the inhibition of sodium channels..\n"
#&gt; 
#&gt; $GSE2880$source
#&gt; NULL
#&gt; 
#&gt; $GSE2880$name
#&gt; [1] "dingl-affy-rat-50847"
#&gt; 
#&gt; $GSE2880$id
#&gt; [1] 400
#&gt; 
#&gt; $GSE2880$lastUpdated
#&gt; [1] 1.53721e+12
#&gt; 
#&gt; $GSE2880$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2880$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2880$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent
#&gt; $GSE2880$lastNeedsAttentionEvent$performer
#&gt; [1] "sophialy"
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent$date
#&gt; [1] 1.526532e+12
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent$eventType
#&gt; $GSE2880$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 218945
#&gt; 
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2880$lastNeedsAttentionEvent$id
#&gt; [1] 25069431
#&gt; 
#&gt; 
#&gt; $GSE2880$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2880$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2880$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; 
#&gt; $GSE3111
#&gt; $GSE3111$shortName
#&gt; [1] "GSE3111"
#&gt; 
#&gt; $GSE3111$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE3111$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$bioMaterialCount
#&gt; [1] 10
#&gt; 
#&gt; $GSE3111$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE3111$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE3111$batchEffect
#&gt; NULL
#&gt; 
#&gt; $GSE3111$experimentalDesign
#&gt; [1] 401
#&gt; 
#&gt; $GSE3111$accession
#&gt; [1] "GSE3111"
#&gt; 
#&gt; $GSE3111$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE3111$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE3111$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE3111$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE3111$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE3111$geeq
#&gt; $GSE3111$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$geeq$publicQualityScore
#&gt; [1] 0.2842847
#&gt; 
#&gt; $GSE3111$geeq$publicSuitabilityScore
#&gt; [1] 0.2125
#&gt; 
#&gt; $GSE3111$geeq$qScorePublicBatchEffect
#&gt; [1] 0
#&gt; 
#&gt; $GSE3111$geeq$qScorePublicBatchConfound
#&gt; [1] 0
#&gt; 
#&gt; $GSE3111$geeq$sScorePublication
#&gt; [1] 1
#&gt; 
#&gt; $GSE3111$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE3111$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE3111$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE3111$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE3111$geeq$sScoreSampleSize
#&gt; [1] -0.3
#&gt; 
#&gt; $GSE3111$geeq$sScoreRawData
#&gt; [1] -1
#&gt; 
#&gt; $GSE3111$geeq$sScoreMissingValues
#&gt; [1] -1
#&gt; 
#&gt; $GSE3111$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE3111$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE3111$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE3111$geeq$qScoreBatchInfo
#&gt; [1] -1
#&gt; 
#&gt; $GSE3111$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9903391
#&gt; 
#&gt; $GSE3111$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9899931
#&gt; 
#&gt; $GSE3111$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 1.200538e-05
#&gt; 
#&gt; $GSE3111$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE3111$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE3111$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE3111$geeq$id
#&gt; [1] 1330
#&gt; 
#&gt; 
#&gt; $GSE3111$bioAssayCount
#&gt; [1] 10
#&gt; 
#&gt; $GSE3111$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE3111$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$public
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE3111$shared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE3111$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE3111$description
#&gt; [1] " The hypothalamic paraventricular (PVN) and supraoptic (SON) nuclei  are important integrative structures that regulate co-ordinated  responses to perturbations in cardiovascular homeostasis. Through  descending projections from parvocellular neurons to the brainstem,  the PVN acts as an autonomic 'premotor nucleus', regulating reflex  changes in sympathetic nerve activity that are involved in blood  pressure and blood volume regulation. Endocrine responses are  mediated through axonal projections from SON and PVN magnocellular  neurons (MCNs) to the capillaries of the posterior pituitary neural  lobe. In response to dehydration, a massive release of the  antidiuretic peptide hormone vasopressin (VP) into the circulation is  accompanied by a dramatic functional remodelling of the HNS. We have  used microarrays to comprehensively catalogue the genes expressed in  the PVN, the SON and the neurointermediate lobe (NIL) of the  pituitary gland. Further, we have identified transcripts that are  regulated as a consequence of dehydration, as well as RNAs that are  enriched in either the PVN or the SON. We suggest that these  differentially expressed genes represent candidate regulators and  effectors of HNS activity and remodelling.\nLast Updated (by provider): Jan 31 2006\n\nContributers:  GEMMA BEIGHTON JULIAN PATON CHARLES C HINDMARCH SONG YAO\n"
#&gt; 
#&gt; $GSE3111$source
#&gt; NULL
#&gt; 
#&gt; $GSE3111$name
#&gt; [1] "Comprehensive description of the transcriptome of PVN in euhydrated and dehydrated rat"
#&gt; 
#&gt; $GSE3111$id
#&gt; [1] 401
#&gt; 
#&gt; $GSE3111$lastUpdated
#&gt; [1] 1.53721e+12
#&gt; 
#&gt; $GSE3111$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE3111$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE3111$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent
#&gt; $GSE3111$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent$date
#&gt; [1] 1.525734e+12
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent$eventType
#&gt; $GSE3111$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216172
#&gt; 
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE3111$lastNeedsAttentionEvent$id
#&gt; [1] 25064113
#&gt; 
#&gt; 
#&gt; $GSE3111$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE3111$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE3111$`_totalInQuery`
#&gt; [1] 249
#&gt; 
#&gt; </div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
    <h2>Contents</h2>
    <ul class="nav nav-pills nav-stacked">
      <li><a href="#arguments">Arguments</a></li>
      
      <li><a href="#value">Value</a></li>
      
      <li><a href="#examples">Examples</a></li>
    </ul>

  </div>
</div>

      <footer>
      <div class="copyright">
  <p>Developed by Ogan Mancarci.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="http://pkgdown.r-lib.org/">pkgdown</a>.</p>
</div>

      </footer>
   </div>

  </body>
</html>

