<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>platformInfo — platformInfo • gemmaAPI</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="platformInfo — platformInfo" />
<meta property="og:description" content="Retrieves information about a single platform. Combines several API calls." />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">gemmaAPI</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">2.0.3.9000</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>platformInfo</h1>
    
    <div class="hidden name"><code>platformInfo.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Retrieves information about a single platform. Combines several API calls.</p>
    </div>

    <pre class="usage"><span class='fu'>platformInfo</span>(
  <span class='no'>platform</span>,
  <span class='kw'>request</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='no'>...</span>,
  <span class='kw'>file</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>return</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>,
  <span class='kw'>overwrite</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>,
  <span class='kw'>memoised</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>
)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>platform</th>
      <td><p>Can either be the platform ID or its short name (e.g: GPL1355). 
Retrieval by ID is more efficient. Only platforms that user has access to will be available.</p>
<p>If a vector of length&gt;1 is provided return all matching platform
objects similar to <code><a href='allPlatforms.html'>allPlatforms</a></code> but without access to additional 
parameters. <code>request</code> parameter cannot be specified for vector inputs</p></td>
    </tr>
    <tr>
      <th>request</th>
      <td><p>Character. If NULL retrieves the platform object. Otherwise</p><ul>
<li><p><code>datasets</code>: Retrieves experiments in the given platform. Parameters:</p><ul>
<li><p><code>offset</code>: Optional, defaults to 0. Skips the 
             specified amount of objects when retrieving them from the
              database.</p></li>
<li><p><code>limit</code>: Optional, defaults to 20. Limits the result 
             to specified amount of objects. Use 0 for no limit.</p></li>
</ul></li>
<li><p><code>annotations</code>: Retrieves the annotation file for the given platform. If you set a file path, the downloaded file will be a .gz file.</p></li>
<li><p><code>elements</code>: Retrieves the composite sequences (elements/probes) for the given platform. Parameters:</p><ul>
<li><p><code>offset</code>: As in datasets</p></li>
<li><p><code>limit</code>: As in datasets</p></li>
</ul></li>
<li><p><code>specificElement</code> Retrieves a specific composite sequence (element) for the given platform. Paramaters:</p><ul>
<li><p><code>probe</code>: Required. Can either be the probe name or ID.</p></li>
</ul></li>
<li><p><code>genes</code> Retrieves the genes on the given platform element. Parameters:</p><ul>
<li><p><code>probe</code>: Required. Can either be the probe name or ID.</p></li>
<li><p><code>offset</code>: As in datasets</p></li>
<li><p><code>limit</code>: As in datasets</p></li>
</ul></li>
</ul></td>
    </tr>
    <tr>
      <th>...</th>
      <td><p>Use if the specified request has additional parameters.</p></td>
    </tr>
    <tr>
      <th>file</th>
      <td><p>Character. File path. If provided, response will be saved to file</p></td>
    </tr>
    <tr>
      <th>return</th>
      <td><p>Logical. If the response should be returned. Set to false when
you only want to save a file</p></td>
    </tr>
    <tr>
      <th>overwrite</th>
      <td><p>Logical. If TRUE, existing files will be overwritten. If FALSE
a warning will be thrown and no action is taken.</p></td>
    </tr>
    <tr>
      <th>memoised</th>
      <td><p>Logical. If TRUE a memoised version of the function will be
used which is faster for repeated requests. Use <code><a href='forgetGemmaMemoised.html'>forgetGemmaMemoised</a></code>
to clear memory.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A data.frame or a list depending on the request</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='fu'>platformInfo</span>(<span class='st'>'GPL1355'</span>)</div><div class='output co'>#&gt; $GPL1355
#&gt; $GPL1355$shortName
#&gt; [1] "GPL1355"
#&gt; 
#&gt; $GPL1355$dateCached
#&gt; NULL
#&gt; 
#&gt; $GPL1355$numProbeSequences
#&gt; NULL
#&gt; 
#&gt; $GPL1355$numProbeAlignments
#&gt; NULL
#&gt; 
#&gt; $GPL1355$numProbesToGenes
#&gt; NULL
#&gt; 
#&gt; $GPL1355$numGenes
#&gt; NULL
#&gt; 
#&gt; $GPL1355$designElementCount
#&gt; NULL
#&gt; 
#&gt; $GPL1355$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GPL1355$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GPL1355$lastGeneMapping
#&gt; NULL
#&gt; 
#&gt; $GPL1355$lastSequenceAnalysis
#&gt; NULL
#&gt; 
#&gt; $GPL1355$lastSequenceUpdate
#&gt; NULL
#&gt; 
#&gt; $GPL1355$blackListed
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL1355$taxonID
#&gt; NULL
#&gt; 
#&gt; $GPL1355$color
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GPL1355$expressionExperimentCount
#&gt; [1] 276
#&gt; 
#&gt; $GPL1355$hasBlatAssociations
#&gt; NULL
#&gt; 
#&gt; $GPL1355$hasGeneAssociations
#&gt; NULL
#&gt; 
#&gt; $GPL1355$hasSequenceAssociations
#&gt; NULL
#&gt; 
#&gt; $GPL1355$isAffymetrixAltCdf
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL1355$isMerged
#&gt; NULL
#&gt; 
#&gt; $GPL1355$isMergee
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL1355$isSubsumed
#&gt; NULL
#&gt; 
#&gt; $GPL1355$isSubsumer
#&gt; NULL
#&gt; 
#&gt; $GPL1355$lastRepeatMask
#&gt; NULL
#&gt; 
#&gt; $GPL1355$description
#&gt; [1] " The GeneChip Rat Genome 230 2.0 Array is a powerful tool for toxicology, neurobiology, and other applications using rat as a model organism.  - Provides comprehensive coverage of the transcribed rat genome on a single array - Comprised of more than 31,000 probe sets, analyzing over 30,000 transcripts and variants from over 28,000 well-substantiated rat genes - The publicly available draft of the rat genome and leading public rat databases were used to refine sequences and provide a higher quality of data output  All probe sets represented on the GeneChip Rat Expression Set 230 are included on the GeneChip Rat Genome 230 2.0 Array. Sequences used in the design of the GeneChip Rat Genome 230 2.0 Array were selected from GenBank, dbEST, and RefSeq. The sequence clusters were created from the UniGene database (Build 99, June 2002) and then refined by analysis and comparison with the publicly available draft assembly of the rat genome from the Baylor College of Medicine Human Genome Sequencing Center (June 2002).  The GeneChip Rat Genome 230 2.0 Array includes representation of the RefSeq database sequences and probe sets related to sequences and refined EST clusters previously represented on the GeneChip Rat Genome U34 Set.  Oligonucleotide probes complementary to each corresponding sequence are synthesized in situ on the arrays. Eleven pairs of oligonucleotide probes are used to measure the level of transcription of each sequence represented on the GeneChip Rat Genome 230 2.0 Array.  Annotations derived from Affymetrix CSV file dated 6/23/2004\nFrom GPL1355\nLast Updated: May 31 2005"
#&gt; 
#&gt; $GPL1355$name
#&gt; [1] "Affymetrix GeneChip Rat Genome 230 2.0 Array"
#&gt; 
#&gt; $GPL1355$lastUpdated
#&gt; [1] 1.575053e+12
#&gt; 
#&gt; $GPL1355$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL1355$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GPL1355$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GPL1355$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GPL1355$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GPL1355$lastNeedsAttentionEvent
#&gt; NULL
#&gt; 
#&gt; $GPL1355$curationNote
#&gt; NULL
#&gt; 
#&gt; $GPL1355$`_totalInQuery`
#&gt; [1] 1
#&gt; 
#&gt; $GPL1355$id
#&gt; [1] 2
#&gt; 
#&gt; </div><div class='input'><span class='fu'>platformInfo</span>(<span class='st'>'GPL1355'</span>,<span class='kw'>request</span> <span class='kw'>=</span> <span class='st'>'datasets'</span>,<span class='kw'>limit</span> <span class='kw'>=</span> <span class='fl'>10</span>)</div><div class='output co'>#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; 
#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; 
#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; 
#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; 
#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; 
#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; 
#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; 
#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; 
#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; 
#&gt; $GSE2872
#&gt; $GSE2872$shortName
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$metadata
#&gt; NULL
#&gt; 
#&gt; $GSE2872$userOwned
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$bioMaterialCount
#&gt; NULL
#&gt; 
#&gt; $GSE2872$processedExpressionVectorCount
#&gt; [1] 31099
#&gt; 
#&gt; $GSE2872$batchConfound
#&gt; NULL
#&gt; 
#&gt; $GSE2872$batchEffect
#&gt; [1] "No batch effect was detected"
#&gt; 
#&gt; $GSE2872$accession
#&gt; [1] "GSE2872"
#&gt; 
#&gt; $GSE2872$externalDatabase
#&gt; [1] "GEO"
#&gt; 
#&gt; $GSE2872$taxon
#&gt; [1] "rat"
#&gt; 
#&gt; $GSE2872$taxonId
#&gt; [1] 3
#&gt; 
#&gt; $GSE2872$experimentalDesign
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$technologyType
#&gt; [1] "ONECOLOR"
#&gt; 
#&gt; $GSE2872$isPublic
#&gt; [1] TRUE
#&gt; 
#&gt; $GSE2872$geeq
#&gt; $GSE2872$geeq$sScorePublication
#&gt; [1] -1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformAmount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScorePlatformsTechMulti
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformPopularity
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreAvgPlatformSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreSampleSize
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$sScoreRawData
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$sScoreMissingValues
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreOutliers
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePlatformsTech
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreReplicates
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$qScoreBatchInfo
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMeanCorrelation
#&gt; [1] 0.9855889
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleMedianCorrelation
#&gt; [1] 0.9866121
#&gt; 
#&gt; $GSE2872$geeq$qScoreSampleCorrelationVariance
#&gt; [1] 3.412146e-05
#&gt; 
#&gt; $GSE2872$geeq$noVectors
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$corrMatIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$replicatesIssues
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$batchCorrected
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$geeq$publicQualityScore
#&gt; [1] 0.8552303
#&gt; 
#&gt; $GSE2872$geeq$publicSuitabilityScore
#&gt; [1] 0.5
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchEffect
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$qScorePublicBatchConfound
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$geeq$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$geeq$id
#&gt; [1] 558
#&gt; 
#&gt; 
#&gt; $GSE2872$arrayDesignCount
#&gt; [1] 1
#&gt; 
#&gt; $GSE2872$bioAssayCount
#&gt; [1] 12
#&gt; 
#&gt; $GSE2872$currentUserHasWritePermission
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$currentUserIsOwner
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$externalUri
#&gt; [1] "http://www.ncbi.nlm.nih.gov/geo/"
#&gt; 
#&gt; $GSE2872$isShared
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$securableClass
#&gt; [1] "ubic.gemma.model.expression.experiment.ExpressionExperiment"
#&gt; 
#&gt; $GSE2872$userCanWrite
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$description
#&gt; [1] "Neurological diseases disrupt the quality of the lives of patients and often lead to their death prematurely.  A common feature of most neurological diseases is the degeneration of neurons, which results from an inappropriate activation of apoptosis. Drugs that inhibit neuronal apoptosis could thus be candidates for therapeutic intervention in neurodegenerative disorders.  We have identified (and recently reported) a chemical called GW5074 that inhibits apoptosis in a variety of cell culture paradigms of neuronal apoptosis.  Additionally, we have found that GW5074 reduces neurodegeneration and improves behavioral outcome in a mouse model of Huntington's disease.  Although GW5074 is a c-Raf inhibitor, we know very little about the molecular mechanisms underlying its neuroprotective action.  Identifying genes that are regulated by GW5074 in neurons will shed insight into this issue. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited.  The specific aim is to identify genes that are differentially expressed in neurons treated with GW5074. We believe that neuroprotection by GW5074 involves the regulation of several genes.  Some of these genes are likely to be induced whereas the expression of other genes might be inhibited. Cultures of cerebellar granule neurons undergo apoptosis when switched from medium containing levated levels of potassium (high K+ or HK) to medium containing low potassium (LK).  Although cell death begins at about 18 h, we have found that the by 6 h after LK treatment these neurons are irreversibly committed to cell death.  We will treat cerebellar granule neuron cultures with LK medium (which induces them to undergo apoptosis) or with GW5074 (1 uM).  We will extract RNA at two time-points after treatment: 3 h and 6 h. Analysis at the two different time-points will show us whether changes in expression of specific genes is transient or sustained or whether the changes occurs early or relatively late in the process.  Neuronal cultures will be prepared from 1 week old Wistar rats.  The cultures will be maintained in culture for 1 week before treatment.  Following treatment the cells will be lysed and total RNA isolated.  The RNA will be stored at -80oC and shipped to the microarray facility for analysis.  The experiment will be done in triplicate.  Thus, for each time-point (3 or 6h treatment) we will be hope to provide 3 sets of samples (each set coming from a different culture preparation and containing lysates from cells treated with LK or GW5074).  Having samples from 3 independent cultures will mitigate any expression differences resulting from subtle variations in culture quality or in the preparation or quality of RNA.\nDate GSE2872 Last Updated: Jul 06 2005\nContributors:  S R D'Mello\nIncludes GDS1837.\n Update date: Jun 09 2006.\n Dataset description GDS1837: Analysis of cultured cerebellar granule neurons in low potassium medium 3 and 6 hours following treatment with the c-Raf inhibitor GW5074. GW5074 blocks low potassium induced cell death. Results provide insight into the neuroprotective action of GW5074."
#&gt; 
#&gt; $GSE2872$source
#&gt; NULL
#&gt; 
#&gt; $GSE2872$name
#&gt; [1] "d'mel-affy-rat-168311"
#&gt; 
#&gt; $GSE2872$id
#&gt; [1] 2
#&gt; 
#&gt; $GSE2872$lastUpdated
#&gt; [1] 1.58528e+12
#&gt; 
#&gt; $GSE2872$troubled
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$lastTroubledEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$needsAttention
#&gt; [1] FALSE
#&gt; 
#&gt; $GSE2872$troubleDetails
#&gt; [1] "No trouble details provided."
#&gt; 
#&gt; $GSE2872$lastNoteUpdateEvent
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent
#&gt; $GSE2872$lastNeedsAttentionEvent$performer
#&gt; [1] "johnphan"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$detail
#&gt; NULL
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$actionName
#&gt; [1] "Update"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventTypeName
#&gt; [1] "DoesNotNeedAttentionEvent"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$action
#&gt; [1] "U"
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$date
#&gt; [1] 1.525801e+12
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$note
#&gt; [1] "Does not need attention."
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType
#&gt; $GSE2872$lastNeedsAttentionEvent$eventType$id
#&gt; [1] 216270
#&gt; 
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$`_totalInQuery`
#&gt; [1] 0
#&gt; 
#&gt; $GSE2872$lastNeedsAttentionEvent$id
#&gt; [1] 25064283
#&gt; 
#&gt; 
#&gt; $GSE2872$curationNote
#&gt; NULL
#&gt; 
#&gt; $GSE2872$`_totalInQuery`
#&gt; [1] 9447
#&gt; 
#&gt; </div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Ogan Mancarci.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


